Notes:

**Date: 5/25/2020**

James Thorburn et. al. (2018) Aquatic Conserv: Mar Freshw Ecosyst. 2018;28:1167–1180 ; Data for n (population) and He includes FS relationship. Data for n and He excluding FS relationship vary by very little from the data I plotted so I chose not to include the data. I also did not include data that were not in HWE. This goes as following: locus DFU285 for the Wales site. If I were to remove the Fs relationships, only SACA 6396 in Celtic Sea and Southern North Sea + SACA 3853 in Southern North Sea would be out of HWE.

**Date: 5/26/2020**

Ingo J. Timm et. al (2012) Conserv Genet 13:693–706 ; Had to obtain coordinates using Google maps since only specific site names were mentioned in the paper. For multiple sites, I picked a point between the sites and used them for my coordinates. The sites are (Samalona, Barranglompo, Barrangcaddi) & (Kapoposang, Suranti).
The rest of the sites required me to pick a point since it varied between multiple island clusters or an entire town. For both cases, I picked coordinates located in the middle of the clusters of islands or town. For the Batam site, instead of locus 45, it is a monomorph causing there to be no data obtained for this specific locus in this location.

M. A. Tripp-Valdez et. al. (2012) J. Appl. Ichthyol. 28 , 516–523; There were no problems with extracting data from this paper

Cemal Turan (2015) Biochemical Systematics and Ecology 63 174e182 ; Had to negate some data due to the some being out of Hardy-Winberg Equilibrium. The following He’s were not recorded: (Bulgarian Varna Coast- Tth4, Tth06, Tth8), (Black Sea Igneada- Tth4, Tth06, Tth34), (Black Sea Duzce- Tth4, Tth07), (the Black Sea Samsun- Tth4), (the Black Sea Trabzon- Tth4), (Istanbul Bosporus- Tth4), (Marmara Sea Bandırma- Tth4), (Aegean Sea Izmir- Tth07, Tth34), (North-eastern Mediterranean Sea Antalya Bay- Tth4, Tth06), (North-eastern Mediterranean Sea Iskenderun Bay- Tth4, Tth06), & (Adriatic Sea [Croatian Coast]- Tth06, Tth07)

**Date: 5/27/2020**

Jim N. Underwood et. al. (2012) Ecology and Evolution, 2: 666-679; Used Google Maps and map of site locations provided in the paper to find the coordinates. Excluded the following due to significant heterozygote deficits: site RS1 (Cm_B007, Cm_A115, Cm_D114), RS2 (Cm_B007), RS3 (Cm_B007, Cm_A115, Cm_D114), RS4 (Cm_B007, Cm_A115, Cm_D114), RS5 (Cm_B007, Cm_A115, Cm_D114), RS6 (Cm_B007, Cm_A115, Cm_D114), SL1 (Cm_B007, Cm_A115, Cm_D114), SL3 (Cm_B007, Cm_A115, Cm_D114), SL4 (Cm_A115, Cm_D114), SS1 (Cm_B007, Cm_A115, Cm_D114), SS2 (Cm_B007, Cm_A115, Cm_D114), SS3 (Cm_B007, Cm_A115, Cm_D114), SS4 (Cm_B007, Cm_A115, Cm_D114). 

Alexandra E. Valentin (2014) Fisheries Research 154 102–119; Used Google Maps and map of site locations provided in the paper to find the coordinates for site Bonbay & site sag. Excluded the following due to significant departure from HWE: for S. Mentella- 2J42 (SEB37, SPI10II), 3L29 (SEB37), 1F1050 (SEB37), 3PN1 (SEB37), 4S35 (SEB37), sag (SEB37).

Fausto Valenzuela-Quin ̃onez  (2014) ICES Journal of Marine Science 71(7), 1619–162; Data provided in the paper was the combination of data collected from multiple sites. Therefore, the site name is called Upper Gulf of California. The four collection sites had <1degree difference with longitude and about 2.5 degree difference with latitude.

**Date: 5/28/2020**

Fausto Valenzuela-Quin ̃onez (2014) ICES Journal of Marine Science 71(7), 1619–162 (continued); Had to exclude the following loci due to them being out of HWE: Tmac74, Soc442, Soc430

A.P. Van de Putte (2012) Deep-Sea Research II 59–60 199–207; Had to exclude the following loci due to them being out of HWE even after the Bonferroni corrections: Site EI (Elan114), SA2 (Elan025 and Elan114), SA3 (Elan001 and Elan025), LS062 (Elan025 and Elan114), BW3 (Elan025 and Elan114)

Anton P. Van de Putte (2012) Marine Genomics 8 23–34; The sites Cape Armitage and Cape Roberts may have been mislabelled in the papers map. Therefore, I used the coordinates I found on google maps for each site rather than using the paper’s labelled sites to pick my coordinates. The rest of the sites were labelled correctly when comparing the paper’s labelled sites and site locations in google maps. Due to there being data that was not in HWE, I have refrained from inputing the following data in the spreadsheet: For Trematomus newnesi- Site SO (Trne035 and Trne066), Site TA1 (Trne035). Due to possible mislabelling in the table, it is possible that the following are monomorphs due to the data provided and was also excluded: For Trematomus bernacchii-  Site DI (Trne055), Site EI (Trne055), Site JI (Trne055) and Site SO (Trne055). Due to possible mislabelling in the paper’s map, site Casey (CS) could vary by 4 degrees longitudinal wise. Therefore, I decided to not include the data for that site entirely. 

**Date: 5/29/2020**

Martin H. van der Meer (2012) Conservation Genet Resour 5:363–366; Had to exclude 8 out of the 17 loci due to them being out of HWE: Cb4 1B, Cb5 4A, Cb6 1C, Cb7 4B, C10 3A, Cb13 4C, Cb23 3C, Cb30 5B

Martin H. van der Meer et. al. (2012) Conservation Genet Resour 5:363–366; Had to exclude the following due to the data being out of HWE: Site ER/Elizabeth Reef (Am4, Am7, Am11, Am18, Am19), Site MR/Middleton Reef (Am4, Am9), Site LHI/Lord Howe Island (Am9), Site LHIL/Lord Howe Island Lagoon (Am1, Am18)

Martin H. van der Meer et. al. Ecology and Evolution 2013; 3(6): 1653– 1666; Had to exclude the following due to the data being out of HWE: Site ER/Elizabeth Reef (Ct4. Ct23, Ct24), Site MR/Middleton Reef (Ct4, Ct16), Site LHI/Lord How

Kanako Ueno et. al. (2013) Conservation Genet Resour 5:659–662; Excluded the data for locus Silj14 as it shows that there was significant departure from HWE after Bonferroni correction for Ho. For Sillago parvisquamis, there was no information for the following loci: Silj04, Silj08, Silj14, Silj22, Silj39, Silj43 and Silj69. I also excluded Silj38 and Silj50 from Sillago parvisquamis due to significant departures from HWE

Carolina Vargas-Caro et. al. (2017) PLoS ONE 12(2): e0172255; Due to the markers original tested in Sillago japonica, the CrossSsp for S. Japonica is recorded as 0 while the CrossSsp for Z. chilensis is 1.

**Date: 6/1/2020**

A. Vella et al. (2016) J. Appl. Ichthyol. 32, 523–531; Used Google Maps , site name, and map provided by the paper to find coordinates

Thomas Vignaud et al. (2013) PLoS ONE 8(4): e61067; Used Google Maps , site name, and map provided by the paper to find coordinates. For the Fakahina site, Cli-2 is monomorphic and is excluded from the data. For the Maria site, Cli-2 only has values of 1 for both He and Ho and was also excluded.

Román Vilas et al. (2015) Marine Genomics 23 (2015) 77–86; Only used Table 1 (contained 58 micro satellite, only 28 were recorded as it was anonymous. The EST-linked were not included. Due to significance level, loci Sma168, Sma147, Sma142, Sma184, Sma185, and Sma100 were not included. Can’t find CrossSpp or Repeat sequence 

Jordi Vinas et al. (2011) Rev Fish Biol Fisheries (2011) 21:527–541; Used Google Maps and map provided by the paper to find coordinates

Le Wang et al. (2012) Int. J. Mol. Sci., 13, 5584-5597; Used populations YCW, QDW, & ZJW only since it is wild. Populations DQ, MY, NB, XP, & SD are all cultured stocks. For population ZJW, locus H54 was excluded as it was out of HWE. Used Google Maps and map provided by the paper to find coordinates. Unable to find CrossSpp or repeat sequences.

**Date: 6/2/2020**

Thomas Vignaud et al. (2013) PLoS ONE 8(4): e61067; excluded the paper since the He’s were whole numbers. The He data in the paper’s table didn’t seem to reflect what the paper talked about. Therefore, to avoid confusion, the paper was excluded. 

QING WANG (2016) Journal of Genetics, Vol. 95, No. 2; Used populations DYW and HNW since they are wild populations. Populations FJC, DYC, and HNC, were excluded since it was a cultured stock. Used Google Maps and map provided by the paper to find coordinates

Robert E. Wilson (2019)ICES Journal of Marine Science, 76(6), 1713–1721; Did not include the following due to it being out of HWE: Southern Chukchi (Gmo34, PGmo32), Eastern Chukchi (Bsa6-1, Bsa7-1, Bsa14, Bsa60-1, Bsa101, Gmo8, Gmo34, PGmo32), Western Beaufort (PGmo127-1), Central Beaufort (PGmo127-1), Southern Beaufort: Southern Shelf (PGmo32)

**Date: 6/3/2020**

L. C. Woodall (2015) Conserv Genet 16:1139–1153; Excluded the following because it had a significance level of p<0.01: TPOa (Hcaμ25), MSP (Hcaμ11), VBU (Hcaμ11, Hcaμ27). Excluded Hgu12 for VBU because it is possibly monomorphic.

XU Dongdong et al. (2012) Chinese Journal of Oceanology and Limnology Vol. 30 No. 4, P. 604-610; excluded Locus Po35 for Zhoushan due to P being 0.000

Dongdong Xu (2017) MARINE BIOLOGY RESEARCH, VOL. 13, NO. 4, 456–462; Excluded the following due to it being out of HWE: QD (Nibea03, Nibea04, Nibea08, Nibea09), ZS (Nibea08, Nibea10, Nibea11, Nial28), ND (Nibea03, Nibea06, Nibea08), ZH (Nibea03, Nibea08, Nibea09)

Emma F. Young (2015) Evol Appl, 8: 486-509; Excluded the following due to it being out of HWE: N. rossii= SS (Nro66), SG (Nro66). Locus Cgu70 and Cgu69 comes from unpublished data. Therefore, to avoid confusion, I left the CrossSpp empty for these loci

**Date: 6/4/2020**

Thomas A. White (2011)Mar Ecol Prog Ser 434: 155–164; He data collected was the avgs of all the loci together. Therefore, no marker names were given. I took the avgs of the CrossSpp and repeats and recorded those in the respective cells. Also, sampling sites NE1, NE2, and NE3 were excluded as they had significant departure from HWE

TA White (2011) Heredity 106, 690–699; He data collected was the avgs of all the loci together. Therefore, no marker names were given. I took the avgs of the CrossSpp and repeats and recorded those in the respective cells. Also, sampling sites MAR5, MAR6, Porcupine, and Greenland were excluded as they had significant departure from HWE

Janne Timm et al. (2017) Front. Mar. Sci. 4:309; The He data comes from the avg of all loci together at the specific location. Avgs of crossSpp and repeats of loci in paper have been recorded. Used Google Maps, site name, and map provided by the paper to find coordinates.

Tetsuya Umino (2013) Conservation Genet Resour 5:771–773; Used Google Maps and site name to find coordinates. For Sillago japonica, there was no information for the following loci: Silp2w, Silp8w, Silp27w, SIL8. I also excluded Silp41w from Sillago japonica due to significant departures from HWE

**Date: 6/5/2020**

Niklas Tysklind et al. (2013) Evol Appl, 6: 891-909; Got sample sizes and He from the following years due to largest sample size: NS1, NS2, NS3, NS4, EC1, EC2, IS3, IS4, IS5, and IS6. Got sample sizes and He from the following years due to that year being the only year to have samples for the locus: IS1, IS2, IS7, AT1, and AT2. The He data comes from the avg of all loci together at the specific location. Avgs of crossSpp and repeats of loci in paper have been recorded. Country/Sea basin name found from plugging the paper’s coordinates into Google maps.

Vaini et al. (2019) Genet Mol Biol. Jan-Mar; 42(1): 74–79; excluded the paper since the longitude and latitudes for the samples vary by more than >3 degrees for both. The samples varied that much because some were collected in South Western Atlantic Ocean and others in the North Eastern Atlantic Ocean.

Fausto Valenzuela-Quiñonez et al. (2016) Journal Of Heredity, 107(6), 496–503; Excluded trite Roca Consag as the He’s were significant after the Bonferroni correction. The He data comes from the avg of all loci together at the specific location. Avgs of crossSpp and repeats of loci in paper have been recorded. Used Google Maps, site name, and map provided by the paper to find coordinates

S. G. VANDAMME et al. (2014) Molecular Ecology 23, 618–636; Excluded this paper as the He was an avg of all loci used at a specific site. However, the loci was a combination of microsatellites and EST-derived microsatellites. Since there was no He per-locus to exclude the EST-derived microsatellites, I had to exclude the paper all together. 

Varela, Ritchie, & Smith (2013) Fisheries Research 140 83–90; The He data comes from the avg of all loci together at the specific location. Avgs of crossSpp and repeats of loci in paper have been recorded. Used Google Maps, site name, and map provided by the paper to find coordinates 

**Date: 6/8/2020**

Venerus et al. (2013) Naturwissenschaften 100:645–658; chose to exclude this paper as the He’s taken were an avg from multiple sample sites that varied by more than 3 degrees for both longitude and latitude per locus. 

Verissimo, McDowell, & Graves (2012) Marine and Freshwater Research, 63, 505–512; chose to exclude this paper as it had the avg He of multiple sample sites per locus. The sample sites ranged from eastern North Atlantic (Ireland, Portugal, and mid-Atlantic ridge north of Azores) to western Indian off South Africa, to western South Pacific off north-west New Zealand. Because of the ranges of latitudes and longitudes, I excluded the paper. Table 1 recorded Ho and Fis but since no equation with specific corrections weren’t included in the paper, I cannot confidently solve for He.

Vignaud et al. (2014) Molecular Ecology 23, 2590–2601; The He data comes from the avg of all loci together at the specific location. Avgs of crossSpp and repeats of loci in paper have been recorded. Used Google Maps, site name, and map provided by the paper to find coordinates 

Viret et al. (2018) PLoS ONE 13 (9): e0203866; The He data comes from the avg of all loci together at the specific location. Avgs of crossSpp and repeats of loci in paper have been recorded. Used Google Maps, site name, and map provided by the paper to find coordinates 

Waldrop et al. (2016) Journal of Biogeography (J. Biogeogr.) 43, 1116–1129; The He data comes from the avg of all loci together at the specific location. Avgs of crossSpp and repeats of loci in paper have been recorded. Used Google Maps, site name, and map provided by the paper to find coordinates. Excluded data for the Northwestern Hawaiian Islands (NWHI) because unsure where the sample site is and the possible range of longitudes for the northwestern Hawaiian Islands are more than 3 degrees.

Wallace (2015) Environ Biol Fish 98:2173–2186; Excluded this paper as no He data was given in the paper. There was also no supplementary materials that could have had He data. Therefore, it was excluded. 

Wallace (2016) Mar Biol 163:142; Excluded this paper because He was given but per locus. The He data values were an avg of multiple sites that ranged from all over the Caribbean. Since there is no data found in the paper or supplementary material that has He of a locus at 1 specific location (rather than an avg of all the locations), it was excluded. 

**Date: 6/9/2020**

Wang et al. (2013) PLOS ONE 8(12): e83493; Unable to find repeat pattern. Excluded the following locus/loci at specific locations due to presence of deviation from equilibrium: site 24 (H84), site 29 (H22,H84, P450), site 16 (P450), site 25 (H22, H84), site 27 (H22, H84, P450), site 28 (H22, H84, P450), site 30 (H22, H84, HSP27, P450), site 1 (P450), site 2 (P450), site 15 (H44, H84, HSP27, P450), site 18 (H71, P450), site 19 (H84), site 31 (H22, H84), site 33 (H22, H84), site 4 (P450), site 5 (H22, H84), site 6 (H84, P450), site 9 (H22, H84, HSP27, P450), site 21 (P450), site 7 (H84, P450), site 10 (P450), site 12 (P450), site 14 (H22, P450), & site 22 (H22, H84, P450) . Excluded the following locus/loci at specific locations due to null alleles: site 17 (HSP27), site 27 (HSP27), site 28 (HSP27), site 30 (H22), site 15 (HSP27, P450), site 32 (H22), site 5 (H22), site 6 (P450), site 9 (P450), site 13 (H84), site 21 (H84, HSP27), site 7 (H84, P450), site 10 (P450), & site 14 (P450). No information was given for locus HSP27, P450, HCA127 for site 32. 

Wang et al. (2015) Fisheries Research, 161, 86–92; The He data comes from the avg of all loci together at the specific location. Avgs of crossSpp of loci in paper have been recorded. Unable to find repeat pattern. 

Weltz et al. (2018) Conservation Genetics 19:1505–1512; The paper is not a primer note. However, the microsatellites used were developed for the study. The He data comes from the avg of all locations combined per locus.

**Date: 6/11/2020**

Wennerstrom et al. (2013) Biodivers Conserv 22:3045–3065; excluded the following due to it being not in agreement with Hardy-Weinberg proportions: Clupea harengus (Outside Baltic), Pungitius pungitius (Outside Baltic, The Kvark, Bothnian Bay). Excluded the following species as they are not fish species: Mytilus trossulus & Fucus vesiculosus. For Clupea harengus, Esox lucius, & Coregonus lavaretus, the He data comes from the avg of all loci together at the specific location. Avgs of crossSpp and repeats of loci in paper have been recorded. For Gasterosteus aculeatus & Pungitius pungitius, The He data comes from the avg of all loci together at the specific location but I was unable to find crossSpp & repeat patterns.

Wildes et al. (2011) Fish. Bull. 109:382–393; excluded the following collections due to it being not the most recent year recorded and/or sample size is smaller: Berners Bay 07 & Berners Bay 08, Sitka Sound 07, Lynn Canal 07 & Lynn Canal 08a.

Xu et al. (2011) Genet. Mol. Res. 10 (3): 1701-1706; excluded this paper as the sampling sites varied by more than 3 degrees of latitude and longitude wise. He was also not shown explicitly. Instead, the data is presented as the decimal answer of Ho/He. 

**Date: 6/12/2020**

Verissimo et al. (2017) Ecology and Evolution, 7:4768–4781; excluded this paper. Although the paper mentions He (expected heterozygosity) in table 1, it does not plot the data. Therefore, I was unable to include this paper. 

Teacher et al. (2012) Evolutionary Applications 6:549–567; included He for the 59 loci since Her14 is seen as an outlier. Also excluded site DE-RUGEN since the paper stated that it appeared to be an outlier as well. Because the He’s are an avg for each specific location, the avg of CrossSpp and repeat pattern was also taken
