01.01.2020

Malin shared GitHub repository with me on Pinsky lab site that has all work currently done on this project by past undergraduates
* Cloned to local computer under name: Pinsky_Lab/Marine_Genetic_Diversity_Proj

01.03.2020

Browsing mtdna & microsat papers that passed screening by undergrads
	* Made lists of the unique papers (author, year, journal (when provided)) to get a sense of what authors and # of papers included
		* mtDNA list: mtDNA_included_papers_2013 --> 162 papers
		* microsat_list: msat_included_papers_2011 --> 389 papers

01.05.2020

Extending original search terms (as given by Malin in readme in GitHub repository) for following years:
	1. microsats: 2011-2020 (to get any 2011 papers not included in original search)
	2. mtDNA: 2013-2020 (to get any 2013 papers not included in original search)

Exporting lists of author, journal, date, abstract to endnote in following groups:
	1. microsat_fish_diversity_2011+ (1025 papers)
	2. mtdna_fish_diversity_2013+ (417 papers)

Google scholar search?Created excel spreadsheet titled mtdna_screening_notes_2013-2020/Marine_Genetic_Diversity_Proj with metadata information about papers from Web of Science search on 1.5.2020
	* Columns for authors, year, journal, marine fish (y/n), mtDNA (y/n), wild population (y/n), included (y/n) --> based on Malin's instructions on GitHub for undergraduates
		* Must meet all three criteria (marine, mtDNA & wild) in order to pass screen (screen in that order)
		* Papers that pass screen added to QuickList on EndNote and will eventually be added to new (refined) group called mtdna_abstract_2013+_pass
			* Those that don't pass will remain in original mtdna group
				* Screened 113 papers (Kept 56)

01.06.2020

Finished initial screening of mtdna abstracts (screened remaining 304 papers and kept 179)
	* In total parsed original pool of 417 papers down to 235 on first pass
		* Most lost bc weren't marine fish (either freshwater fish or marine non-fish)

Starting initial screening of msat abstracts
	* Created excel spreadsheet titled msat_screening_notes_2011-2020/Marine_Genetic_Diversity_Proj with metadata information about papers from Web of Science search on 1.5.2020
		* Columns for authors, year, journal, marine fish(y/n), msat (y/n), wild population (y/n), non-EST (y/n), included (y/n) --> based on Malin's instructions on GitHub
			* Must meet all four criteria in order to pass screen (screen in that order)
				* EST = expressed sequence tag, comes from mRNA (expressed gene in tissue) and more likely that that gene will be under selection (which we don't want as looking for diversity shaped by mostly neutral processes)
					* Work backwards from mRNA to create EST and target gene in DNA that interested in (can then sequence/play with the actual DNA)
						* Used for microsats bc easy-ish way to get primers for genes?
			* Papers that pass screen added to QuickList on EndNote and then added to new group called microsat_abstract_2011+_pass
				* Those that don't pass remain in original msat group
					* Screened 210 papers, kept 100

01.07.2020

Continuing first screen of msat papers
	* Screened 290 papers, kept 121

01.08.2020

Continuing first screen of msat papers (from 500 on, just about halfway through at start of day)
	* Screened 200 papers, kept 95

01.09.2020

Finished first screen of msat papers
	* Screened 325 papers, kept 156
		* In total parsed original pool of 1025 papers down to 473 on first pass
			* Most lost bc weren't marine fish
				* Although more non-wild pop papers in msat group than mtdna group

Added sheet to excel spreadsheet titled mtdna_screening_notes_2013-2020 (sheet titled mtdna_paper_screen) with only papers that passed abstract screen (changed original sheet name to mtdna_abstract_screen)
	* Columns for authors, year, journal, wild pop (y/n), hybrid (y/n), sample size (>= 4), mtDNA bp (>=200 bp), RFLPs (y/n), marine fish (y/n), # sample sites (>1), lat/long (y/n), lat/long accuracy (<=3 degrees), multiple time points, data already reported (y/n) --> based on Malin's instructions on GitHub for undergraduates
		* Must meet all criteria to pass screen (in that order)
			* If there is more than one time point, choose time point with largest sample size to report (if tie, use most recent)
			* If data reported in earlier paper, use that paper
		* Papers that pass screen added to QuickList on EndNote and will eventually be added to new (refined) group called mtdna_paper_2013+_pass
			* Those that don't pass will remain in mtdna_abstract_2013+_pass

Started screening papers
	* Hard to determine lat/long precision for some papers
		* Being conservative for now -- if exact coordinates provided give precision as 0
			* If detailed map provided, try to match accuracy to scale of map
				* If map isn't detailed, setting accuracy as 3 degrees for now to keep in pile (email authors/dig through supplemental material at later date)

Screened first 50 papers, kept 2601.10.2020Continuing second screening of mtdna papers
	* Screened next 69 papers, kept 40

01.12.2020

Continuing second screening of mtdna papers
	* Screened 23 papers, kept 9

01.13.2020

Continuing second screening of mtdna papers
	* Screened 69 papers kept 39

01.14.2020

Finished second screening of mtdna papers
	* Screened 23 papers kept 11

Kept a total of 125 papers (down from 234)
	* Kept 5 papers bc couldn't read/access them
	* Most papers again lost bc not a marine species (or doing a diet analysis)
		* Some bc didn't have large enough sample size
		* Some bc didn't have lat/long coordinates or were too broad (averaged across a wide area)
			* Although keeping ones may be able to dig up coordinates for (if give landing site, for ex)

Added sheet to excel spreadsheet titled msat_screening_notes_2011-2020 (sheet titled msat_paper_screen) with only papers that passed abstract screen (changed original sheet name to msat_abstract_screen)
	* Columns for authors, year, journal, marine fish (y/n), wild pop (y/n), hybrid (y/n), microsat (y/n), EST-linked (y/n), HWE (y/n), polymorphic (y/n), sample size (>= 4), # sample sites (>1), lat/long (y/n), lat/long accuracy (<=3 degrees), multiple time points, data already reported (y/n) --> based on Malin's instructions on GitHub for undergraduates
		* Malin didn't have criteria for sample size/# sample sites BUT keeping same as mtdna data
			* May go back and add if sites is less than 1 but def need decent sample size for diversity estimates
		* Must meet all criteria to pass screen (in that order)
			* If there is more than one time point, choose time point with largest sample size to report (if tie, use most recent)
			* If data reported in earlier paper, use that paper
		* Papers that pass screen added to QuickList on EndNote and will eventually be added to new (refined) group called microsat_paper_2011+_pass
			* Those that don't pass will remain in microsat_abstract_2011+_pass

Started second screening msat papers
	* Screened 52 papers kept 25

01.15.2020Creating spreadsheet with officially metadata information from mtdna papers (2013+) (mtdna_2013-2020_data)
	* Same set up as Malin's spreadsheet
		* Columns:
			* Spp --> scientific name (check Fishbase to make sure most up-to-date)
			* CommonName --> get from Fishbase (paper if not available)
			* Source --> "Adams et al. 2010 Molecular Ecology 8:1000-1015"
			* Country --> country where sample was taken (or ocean if far from shore)
			* Site --> site name as given by author
			* LatDeg --> degrees latitude (negative for Southern Hemisphere)
				* If not given in paper, use paper figures OR google maps of place names)
					* If range, okay for average but only if less than 3 degrees
			* LatMin --> only if no decimal degrees
			* LatSec --> only if no decimal degrees
			* LonDeg --> negative for Western Hemisphere
			* LonMin
			* LonSec
			* CollectionYear --> year in which samples were taken
				* Leave blank if not stated
				* Average if samples from multiple years were pooled
			* MarkerName --> name of mtDNA locus as listed in paper
			* n --> number of individuals sampled
			* bp --> length of mtDNA locus
			* He --> expected heterozygosity
				* Haplotype diversity
			* Hese --> standard error of He measurement (if SD then divide SD by square root of n)
			* Pi --> nucleotide diversity
			* Pise --> standard error of pi

NEEDS to have Spp, source, He or pi, and Lat (Deg or Min/Sec) for each line in order to keep
	* So each line is a unique combo of species/sample location/marker
		* Eg if two markers used for same sample location each get diff line

Pulling papers (first 50) from mtdna paper pass list and saving to computer so can do on plane tomorrow (saving under Marine Genetic Diversity Papers/mtDNA_Paper)
	* Go paper by paper

Continuing second screen of microsat papers
	* Screened 31 papers, kept 18

01.16.2020

Starting to pull data from mtdna papers (2011+)
	* Double check if 2011 paper that not already pulled
Notes:

	* Not recording Anderson et al. 2017 Journal of Fish Biology 91:558-573 bc study of invasive species
	* Not recording Andreotti et al. 2016 J. Biogeogr. 43:328-339 bc pooled all South African studies into a single pop for analysis (spans more than 3 deg latitude)
		* Also aggregations of white sharks which have large home ranges
	* Not recording Andrews et al. 2016 Molecular Phylogenetics and Evolution 100:361-271 bc already reported (and included) in Andrews et al. 2014
	* Barker et al. 2017 Journal of Fish Biology 91:664-668 not included bc sample size not large enough
		* Many individuals collected subsequently ID-d as diff species
	* Berkstrom, Jorgensen & Hellstrom 2013 Mar. Ecol. Prog. Ser. 477:201-215 not included bc primarily diet analysis of 2 species
		* mtDNA sequence just used to barcode for species ID
	* Bernal et al. 2017 Molecular Ecology 26:639-652 not used because hybridization study
	* Bonello et al. 2016: Journal of the Marine Biological Association of the United Kingdom 96:1585-1596 not used bc using NCBI/GenBank specimens (pre-recorded and low sample size)
	* Catarino et al. 2017 PLoS ONE 12:e0174988 MAR 1-3 sites averaged across window of <3 degrees
		* Showing largest range (min correspond to appropriate deg) but need to convert to decimal and take mean distance btwn samples
			* MED sampling sites dropped bc longitude window too big (>3 degrees)
		* ROC collected in 2 years BUT only 2011 samples used for mtDNA analyses
	* Chen et al. 2016 Zoological Studies 55:33 not included bc (1) most samples already included in previous studies and (2) samples included in current study don't have precise enough lat-long recorded
	* Coleman et al. 2016 Molecular Phylogenetics and Evolution 100:243-253 not included bc don't have precise enough lat-long
		* Don't provide detailed sampling locations to pull coordinates from (just general, i.e. "Philippines")
			* Map not very detailed either
	* Deng et al. 2019 Journal of the Marine Biological Association of the United Kingdom 1-8, splitting pi for some sites into two lineages (a & b)
	* Need to double check that showing se and not sd in some sites
		* If not sure highlight box

01.17.2020

Continuing to pull data from mtdna papers
	* First filling in missing countries/geographic coordinates for other papers and checking whether sd or se provided (when given)
	* Adding notes column to note if had to pull geographic coordinates based on place names
Notes:

	* Removing Ahti etal. 2016 J. Biogeogr. 43:1103-1115 bc don't actually provide latitude/longitude coordinates for sites
	* DiBattista et al. 2015 J. Biogeogr. 42:1919-1929 not included bc don't have lat/long coordinates provided and just given generalized place names
	* DiBattista et al. 2016 Heredity 647-653 not included bc don't have het/pi statistics
	* DiBattista et al. 2016 Molecular Phylogenetics and Evolution 101:203-215 not included bc hybridization study
	* DiBattista et al. 2017 Coral Reefs 36:625-638 not included bc het and pi statistics reported calculated from all sampling locations pooled together
	* Dimitriou et al. 2019 Diversity-Basel 11:149 not included bc only 1 sampling site is new, sampling site doesn't have coordinates provided and site name not specific enough to pull coordinates, He/pi not reported

01.18.2020

Continuing to pull data from mtdna papers
Notes:

	* Eiriksson & Arnason 2015 Environ. Biol. Fish 98:67-79 not included bc no information on He/pi and haplotypes not given
	* Eirkisson & Arnason 2014 Environ. Biol. Fish 97:103-110 not including North Sea Samples bc came from 7 diff locations and lat/long is averaged across those
	* Fernandez-Silva 2015 J. Biogeogr. 42:2402-2413 not including following sites bc sample size is less than 4: Saipan, Diego Garcia, Faafu Atoll, Muscat & Yanbu
	* Ferrari et al. 2018 Peerj 10.7717 haplotype diversity calculated by hand given haplotype counts at each site
		* Removing sites with less than 4 samples & removing Rockhall Plateau bc no latitude provided

01.19.2020Continuing to pull data from mtdna papersNotes: 
	* Freitas et al. 2017 Fisheries Research 191:87-94 some localities removed bc: sample size too small (Rio Grande de Sol & La Paloma), He/pi not calculated (Espirito Santo), or already reported in Marquez 2012 (Rio de Janiero, Sao Paulo & River Plate)
	* Gao et al. 2019 J. Ocean Univ. China 18:947-952 Rizhao has three separate populations so choosing one with largest sample size (RZ2)
	* Giles et al. 2014 Bull. Mar. Sci. 90:427-454 not included bc lat/long represent midpoints between landing sites and/or vessel transects (no idea how far apart those sampling sites truly are)
	* Habib et al. 2015 Ocean Sci. J. 50:143-154 only including Korean sites bc other two locations (Japanese sites)  previously reported in Kimura et al. 2007
	* Han et al. 2019 Mar. Biodiv. 49:263-275 not included bc not giving He/pi for sites and don't have raw sequences to calculate on own
	* Healey et al. 2017 Ecology & Evolution 10.1002/ece3.3775 not including two L. mahsena sites (Seychelles Mahe & Mozambique Pemba) bc sample size <4 individuals
	* Healey et al. 2018 ICES Journal of Marine Science 75:1465-1472 sampling sites were pooled so checked supplemental material and only including pooled sites that are less than 3 degrees apart

		* Presenting average lat long given google earth coordinates for pooled sites
		* For Mozambique and SA that split into two clades, presenting clade A & B combined line
	* Henriques et al. 2016 J. Biogeogr. 43:1820-1831 not included bc He/pi not given and raw sequences not provided to calculate on own
	* Henriques et al. 2016 Molecular Ecology 25:5843-5861 not included bc He/pi averaged across region >3 degrees latitude and specific samples/haplotypes not given to calculate on own
	* Henriques et al. 2017 Fisheries Research 187:86-97 reported several years at same site, picked year with largest sample size (site-by-site basis) or most recent year when there was a tie
	* Horne et al. 2013 Journal of Heredity 104:532-546 reported several sampling times at same site, picked year with largest sample size

		* Bc some sites in Guam reported (and treated) samples by months one year, kept those months separate as well
	* Hune et al. 2014 Polar Biology 38:607-617 not reported bc He/pi pooled across sites for each species and not given raw sequences to calculate on own
	* Huyghe & Kochzius 2018 PLoS ONE 13:e0193825 only reporting Mozambique sites as other sites previously reported in Huyghe & Kochzius 2016

		* CHECK THIS

01.20.2020Continuing to pull data from mtdna papersNotes: 
	* Jackson et al. 2014 PLoS ONE 9:e97508 not including British Virgin Islands site bc none of the individuals were included for mtDNA analysis
	* Kennington et al. 2017 Fisheries Research 193:242-249 some sites reported for multiple time points, picking time point with largest sample size (Imperieuse Reef) or most recent time period when there is a tie (Scott Reef)
	* Kuriiwa et al. 2014 Ichthyol. Res. 61:361-374 haplotype diversity calculated by hand given haplotype counts at each site (bc presented by lineage in paper tables)

		* Removing sites with less than 4 samples
	* Lait, Marshall & Carr 2018 Ecology and Evolution 8:6420-6437 including BUT should note that He/pi based on entire mitogenome (16575 bp) which seems unfair? --> haplotype diversity essentially 1 for each population

		* Removing Gilbert Bay sample bc <4 individuals
		* Removing Lake Qasigialiminiq pop sample bc in a lake

01.28.2020Continuing to pull data from mtdna papersNotes:
	* Lavina-Vincent et al. 2017 Mar. Biodiv. 47:413-419 excluded bc only one sampling location
	* Lewallen et al. 2016 PLoS ONE 11:e0163198 excluded bc, while exact lat/long coordinates provided, sample sizes too small (<4 individuals) for analyses

		* In paper, aggregated across ocean for comparisons
	* Li et al. 2015 Biochem Genet 53:8-22 not included bc coordinates not given for sampling sites, names not given for sampling sites & map resolution is too large
	* Liggens et al. 2015 Ecography 38:1212-1224 not included bc lat/long coordinates not given, sampling sites not named, and analyses done clade wide not per pop
	* Liu et al. 2019 PeerJ 7:e7384 some sites excluded bc: sample size <4 individuals (Nusa Lembongan & Saipan) or site name not specific enough to grab lat/long coordinates (Xisha, NPP III Inlet, Marshall Island, New Caledonia, Fiji) and lat/long coordinates not provided
	* Magallón-Gayón, Diaz-Jaimes & Uribe-Alcocer 2016 PeerJ 4:e2583 collected at multiple time points in each location BUT mtDNA only extracted for one time point at each
	* McKeown, Arkhipkin & Shaw 2015 Fisheries Research 164:1-7 not including bc no lat/long coordinates or specific site names to pull from
	* Milana, Ciampoli & Sola 2014 Biological Journal of the Linnean Society 113:635-641 haplotype diversity calculated by hand given haplotype counts at each site (bc presented by lineage in paper tables)

		* Removing sites with less than 4 samples
	* Misawa et al. 2018 Fishery Bulletin 117:24-36 only including sampling sites that were collected at one location

		* Some sites pooled from large lat-long range
	* Miya et al. 2016 Zootaxa 4061:381-396 not included bc can't access paper
	* Morales et al. 2018 Rev. Fish Biol. Fisheries 28:941-951 not included bc sample sites are pooled across large lat-long region (ocean basins) and midpoint coordinates are only coordinates provided
	* Moreira et al. 2019 J. Fish. Biol. 94:725-731 not included bc sampling sites either pooled from large lat/long range (and specific haplotype information not included) or specific location not given to pull coordinates off Google Maps
	* Mzingirwa et al. 2019 Molecular Biology Reports 46:5079-5088 not including sample sites with no individuals sampled for mtDNA (Comoros)
	* Nirchio et al. 2019 Genetica 147:47-56 haplotype diversity calculated by hand given haplotype counts at each site

		* Only using three cites that were collected in current paper
	* Nyegaard et al. 2018 Estuarine, Coastal and Shelf Science 207:1-15 not included because het/pi metrics not presented and haplotypes not given to calculate by hand

01.29.2020Continuing to pull data from mtdna papersNotes:
	* Ochoa-Munoz et al. 2018 Revista de Biologia Marina y Oceanografia 53:157-168 not included bc no He/pi metrics included and raw haplotypes not given to calculate by hand
	* Otwoma et. al. 2018 Journal of Fish Biology 93:597-608 only including sites caught for paper and not sites previously recorded
	* Pazmiño et al. 2017 Conserv. Genet. 18:1151-1163 Santa Cruz & San Cristobal sites were aggregated by island so reporting average lat-long coordinates provided for each specific site for any given island

		* Range is less than 1 degree for each island
		* Should note that not sure of exact sample sizes --- 85 total sampled but only 77 used in mtDNA analyses and unclear where those 77 came from
	* Podlesnykh & Moreva 2014 Russian Journal of Genetics 50:949-956 not included bc no He/pi metrics included and raw haplotypes not given to calculate by hand
	* Portnoy et al. 2016 ICES Journal of Marine Science 73:2322-2332 not included bc no He/pi metrics included and raw haplotypes not given to calculate by hand
	* Reid et al. 2016 Heredity 117:307-315 not included bc samples aggregated into regions >3 degrees lat/long and raw haplotypes not given to calculate by hand
	* Robertson et al. 2018 PeerJ 6:e4328 not included because on invasive species
	* Šegvić-Bubić et al. 2016 Fisheries Research 179:271-279 only including sites with >4 samples (based on supplemental material)
	* Sellas et al. 2015 Journal of Heredity 266-275 Mexican sites average lat/long reported (less than 3 degrees separating farthest sites)
	* Sepúlveda & González 2017 Journal of Fish Biology 90:249-264 sites sampled in multiple years, retaining metrics from year with largest sample size

01.31.2020Continuing to pull data from mtdna papersNotes:
	* Silva et al. 2017 Scientific Reports 7:2893 only including sites that haven't been included in Silva, Horne & Castilho 2014 paper
	* Silva et al. 2018 Fisheries Research 198:138-149 not including Colombia site bc entire country and no given coordinates

02.02.2020Continuing to pull data from mtdna papersNotes:
	* Sims et al. 2014 Coral Reefs 33:223-233 including samples gathered for study as well as unpublished mtDNA CR sequences from some sites from James Cook University

		* Not included published sequences from Fiji
		* Not including PNG (Kimbe Bay) bc some published sequences included and split into two clades for analysis (don't have raw sequences to calculate He on own)
		* Not including Indonesia because sample size <4
		* Not including Maldives & Japan (Morote Island doesn't exist?) because can't get accurate location
	* Stefanni et al. 2015 Heredity 115:527-537 including BUT Azores likely recent colonization event (according to paper) SO maybe don't want to include bc diversity going to be more impacted by recent colonization/bottleneck/founder events (other sampling sites presented likely fine)

		* Not including Canaries Island & Mauritania bc can't get specific location for collection
		* Portugal sites lat/long listed are averages of sites collected from (<3 degrees apart)
	* Tohkairin et al. 2016 Ichthyol. Res. 63:370-381 not reporting COI & Cytb concatenated diversity reports bc each sequence also reported individually

		* Not including Hokkaido pop bc it's only one Hokkaido
		* Some lat/long listed are averages of sites collected from (<3 degrees apart)

			* Middle/SW Honshu & Pacific not included bc sites >3 degrees apart
	* Turan et al. 2019 Frontiers in Marine Science 6:740 not included bc COI not included in genetic diversity analyses
	* van der Meer et al. 2013 Ecology and Evolution 3:1653-1666 stats in supplemental material
	* van der Meer et al. 2015 Coral Reefs 34:393-402 stats in supplemental material
	* Vella & Vella 2017 Marine Genomics 36:25-32 not included bc no diversity metrics and raw sequences not provided to calculate by hand
	* Venerus et al. 2013 Naturwissenschaften 100:645-658 some sites from other studies but Argentina samples mostly new, diversity stats for Argentina samples pooled from >3 degree range

		* Raw sequences may be provided to calculate He by hand, but need to dig more carefully through supplemental
	* Vignaud et al. 2014 Molecular Ecology 23:2590-2601 some sites also include samples from Castro et al. 2007 Molecular Ecology 16:5183-5192 so excluding sites that seem to have samples from both places (bc raw sequences not given to calculate without); also whale shark so is pop/diversity really tied to location?

		* Ningaloo Reef, Gulf of California, Philippines/Taiwan, Maldives
		* Red Sea & Mozambique not included bc no specific location provided to get coordinates for
	* Viñas et al. 2014 ICES Journal of Marine Science 71:391-297 some samples from Santas et al. 2008 ICES Journal of Marine Science 65:1635-1644 so only including regions that are novel to current paper

02.03.2020Continuing to pull data from mtdna papersNotes:
	* Waldrop et al. 2016 J. Biogeogr. 43:1116-1129 not including sites where sampling location is too broad to get specific geographic coordinates (American Samoa, Marshall Islands, MHI, NWHI, Indonesia, Oman, Jazirat Baraqan)
	* Wang et al. 2013 Biochemical Systematics and Ecology 46:137-144 not included bc genetic diversity for sites reported as clades and not given raw sequences to calculate by hand
	* Wilson et al. 2019 ICES Journal of Marine Science 76:1713-1721 excluding Chukchi site bc pooled across too wide a lat-long region
	* Wu et al. 2014 Fish. Sci. 80:415-426 excluding all sites except one because caught over too large a lat/long region (region presented, showing avg. coordinates)
	* Xiao et al. 2015 Mitochondrial DNA 26:409-419 not including Korean samples bc gotten off GenBank

02.05.2020Finished pulling data from mtdna papersNotes:
	* Zhang et al. 2014 PLoS ONE 9:e105494 sites pooled into three different regions for genetic analyses for one of the species (H. trimaculatus) only keeping regions that represent sites closer than 3 degrees lat/long

		* Lat/long presented in datasheet is average of sites included in given region
	* Zheng, Zou & Han 2015 Biochemical Systematics and Ecology 58:169-177 some locations sampled over more than one year so only keeping one with largest sample size or most recent one if sample size is the same

Still need to double check google scholar to make sure not missing anything but otherwise done with mtDNA data collection!! (Through 2019)02.07.2020Continuing second screen of microsat papers
	* Screened 26, kept 19

02.08.2020Continuing second screen of microsat papers
	* Screened 69, kept 44

		* Total to date screened 178, kept 106

02.09.2020Continuing second screen of microsat papers
	* Screened 43, kept 24

		* Total to date screened 221, kept 130

02.19.2020Continuing second screen of microsat papers
	* Screened 29, kept 20

		* Total to date screened 250, kept 150

02.20.2020Working on merging my mtDNA dataset with Malin's already compiled one
	* So have fully up-to-date map for prelims

		* Will just use his code to generate map (or modify slightly)
	* Rows are the same, just need to double check not duplicate studies (2013 studies might have been pulled twice)

		* Nothing duplicated
	* Merged datasets together and named: "data/mtDNA/mtDNA_merged.csv"

working on R script for cleaning and mapping mtDNA (primarily editing portion that generates data for map and creates map) 

		* Fixing some lat-long coordinates to be able to calculate decimal coordinates and map

			* Catarino et al. 2017 PLoS ONE 12:e0174988 throwing error
			* Had some typos where looks like lat/long coordinates were either switched or didn't put a decimal place in

02.22.2020Need to check msat and old mtdna papers bc concerned some of those species not truly marine (some anadromous)Also will need to double check coastal/pelagic classifications
	* And make sure including all species, seems like some got dropped in classifying ecology

03.12.2020Continuing second screen of microsat paprers
	* Screened 100, kept 72

		* Total to date screened 350, kept 222

03.13.2020Continuing second screen of microsat papers
	* Screened 70, kept 47

		* Total to date screened 420, kept 269

03.17.2020Finished second screen of microsat papers
	* Screened 53, kept 43

		* Total to date screened 473 kept 312

Need to go back and make sure ##s line up between endnote and excel spreadsheet
	* Double checked and everything looks good

Going back to add papers to mtdna with only one sampled site
	* At least pull data now: put in new excel spreadsheet: mtDNA_2013-2020_DNA_1site

		* Reviewed papers with only 1 sampling site in mtDNA data set --> 4 papers that look like pass criteria

			* Saved papers to mtDNA paper directory and pulling info to add to new spreadsheet

				* Notes:

					* Vella, Vella & Schembri (2017) Marine Genomics 36:17-23: some species with less than 4 samples at site so removing from consideration (Carcharinus plumbeus, Galeus melastomus, Alopias superciliosus, Carcharadon carcharias, Odontaspis feox, Oxynotus centrina, Dasyatis tortonesei, Myliobatis aquila, Leucoraja circularis, Rostroraja abla)

						* Squalus spp. excluded as well bc don't know what species these samples are
					* Yu et al. (2015) Ichthyological Research 62:439-449 not included because not seeing any het or pi stats
	* At end, added three additional papers:

		* Imsiridou et al. (2016) Cah. Biol. Mar. 57:9-15 (1 species)
		* Turanov, Lee & Kartavtsev (2019) Mitochondrial DNA Part A 30:264-272 (1 species)
		* Vella, Vella & Schembri (2017) Marine Genomics 36:17-23 (25 species)

03.18.2020Going back and double-checking studies undergrads pulled for mtDNA to make sure all species are truly marine
	* All look good

Creating spreadsheet with officially metadata information from msat papers (2011+) (msat_2011-2020_data)
	* Same set up as Malin's spreadsheet

		* Columns:

			* Spp --> scientific name (check Fishbase to make sure most up-to-date)
			* CommonName --> get from Fishbase (paper if not available)
			* Source --> "Adams et al. 2010 Molecular Ecology 8:1000-1015"
			* PrimerNote --> 1 for primer note, 0 for not

				* Primer Note is study that only exists to report the discovery of new microsats
			* Country --> country where sample was taken (or ocean if far from shore)
			* Site --> site name as given by author
			* LatDeg --> degrees latitude (negative for Southern Hemisphere)

				* If not given in paper, use paper figures OR google maps of place names)

					* If range, okay for average but only if less than 3 degrees
			* LatMin --> only if no decimal degrees
			* LatSec --> only if no decimal degrees
			* LonDeg --> negative for Western Hemisphere
			* LonMin
			* LonSec
			* CollectionYear --> year in which samples were taken

				* Leave blank if not stated
				* Average if samples from multiple years were pooled
			* NumMarkers --> # of microsat markers whose data entered on line (when can report single marker at a time do so)
			* MarkerName --> name of mtDNA locus as listed in paper
			* CrossSpp --> 1 if msat originally developed in different species

				* If reporting average of msat markers input proportion that were cross-species (e.g. 0.2 if 1 of 5 was cross-species)
			* n --> number of individuals sampled

				* If NumMarkers > 1 may need to be an average of all markers reported

					* If some markers not in all individuals for example
			* Repeat --> length of msat repeat (2-5)

				* If repeat is complex then use most common repeat motif length
				* If NumMarkers > 1 then use average across all repeat sizes of markers
			* He --> expected heterozygosity

				* Haplotype diversity
			* Hese --> standard error of He measurement (if SD then divide SD by square root of n)

NEEDS to have Spp, source, NumMarkers, n, He, and Lat (Deg or Min/Sec) for each line in order to keep
	* So each line is a unique combo of species/sample location/marker

		* Eg if two markers used for same sample location each get diff line

Do NOT enter loci not in H-W equilibrium
	* One loci out of HWE in several pops

		* Bc likely allelic dropout/null alleles OR under selection
	* Out of HWE because of inbreeding? make a note but keep for now

Pulling papers (first 100) from msat paper pass list and saving to computer (saving under Marine Genetic Diversity Papers/msat_Paper)
	* Go paper by paper

Starting to pull data from msat papersNotes:
	* Agostini et al. (2013) Polar Biol. 36:1509-1518: not included because pooled all sites into het analyses
	* Aguila et al. (2015) PLoS ONE 10:e0138292: not including Palawan site bc too broad and no lat/long coords provided; all Philippines sites coords are averaged across region; not including Obe652 locus bc out of HWE across sites
	* Almojil, Cliff & Spaet (2018) Ecology & Evolution 8:9536-9549: Oman, Yemen, South Africa not included bc sites >3 deg apart; some sites averaged across regions
	* An et al. (2012) Journal of the World Aquaculture Society 43:73-83: Buan population KSs8 locus not included, Yeosu KSs6 & KSs11B loci not included (het deficit)

03.19.2020Continuing to pull data from msat papersNotes:
	* An et al. (2013) Mol. Biol. Rep. 40:3445-3456: not including Tongyeong & Goeje hatchery strains; excluded KTm223 in Pohand & KTm234 in Taean bc not in HWE (het deficit)
	* An et al. (2013) Biochemical Systematics and Ecology 50:397-405: excluded KSC6 in Pohang & KSC9 in Sungsan bc not in HWE (het excess)
	* An et al. (2014) GMR 13:8901-8912: lat/long averaged across two wild sampling sites; KHn3 & KHn19 excluded bc evidence of null alleles
	* Anderson & Karel (2013) Fish. Bull. 112:71-81: excluding Gulf & Atlantic menhaden samples because analyzed as one large group; excluding AF* loci because already reported in Anderson 2007; excluding Bp301 in yellowfin menhaden bc not in HWE; excluding Bp155 in finescale menhaden & Bp003, Bp039, Bp155, Bp489 & Bp500 in yellowfin menhaden bc monomorphic (or didn't amplify) in respective pop
	* Andreotti et al. (2016) J. Biogeogr. 43:328-339: only including SA samples bc others don't have nucDNA info; excluding Ccar627.x locus bc no repeat/sequence provided; Cca1072 & Ccar9 excluded for Gansbaai pop bc not in HWE
	* Andrews et al. (2014) PLoS ONE 9:e91665: not including bc some loci show evidence of null alleles (out of HWE in all or almost all pops) but all loci included in average calculations
	* Araneda et al. (2017) Lar. Am. J. Aquat. Res. 45:188-192: Sample sites pooled into one location for analysis; not including cmrDe2 & cmrDe9 loci bc not in HWE (het deficit)

03.20.2020Continuing to pull data for msat papers03.22.2020Continuing to pull data for msat papersNotes:
	* Barton et al. (2018) Marine and Freshwater Research 69:487-501: Lel1012 excluded in all pops, Lel040 & Lel027 excluded at Locker Point, Lel039 at Cape Preston & Lel039 excluded at Camden Sound bc not in HWE (het deficiency); Moreton Bay not included bc no fish included in genetic analyses

03.23.2020Continuing to pull data for msat papersNotes:
	* Bekkevold et al. (2016) PLoS ONE 11:e0148488: Her114 excluded from analysis bc not in HWE (het deficit); Bothnian Bay, Gulf of Finland, Banks & Shetland sites not included bc no microsat data; Using data from most recent sample per site
	* Beldade et al. (2014) Mar. Ecol. Prog. Ser. 499:193-201: EAC_B07 (Puerto Libertad, Santo Tomas, San Luiz Gonzaga), EAC_A06 & EAC_A03 (Puerto Libertad & San Luiz Gonzaga), EAC_D02 (Santo Tomas) excluded bc not in HWE (het deficit)
	* Beltran et al. (2015) Conservation Genetics Resources 7:543-545: not included bc sampling sites pooled for het calculations (>3 lat/long range)
	* Bentley et al. (2014) Mar. Ecol. Prog. Ser. 506:267-277: Only keeping sites with 5 or more samples (only ones recorded msat data for)
	* Berry et al. (2012) Fish. Oceanogr. 21:243-254: Only keeping Metro & South-West site bc others span >3 deg lat/long range; GheA127, GheA129 & GheA114 removed from average het analysis w/in sites
	* Berry et al. (2012) Molecular Ecology 21:2145-2159: Only keeping North & South Ningaloo & South Gascoyne as only sites with data available; (using raw microsat scores from dryad site); RTE19, 58rte, RTE 23, 75rte, 44rte & RTE 68 excluded from N. Ningaloo, 44rte excluded from S. Ningaloo bc not in HWE (het deficit)

03.24.2020Continuing to pull data for msat papersNotes:
	* Bertolotti et al. (2015) PeerJ 3:e1253: Combining two sampling locations into 1 (<3 degrees lat/long apart); Paxalb_32, Paxalb_20, Paxalb_24 excluded bc not in HWE (het deficit)
	* Berumen et al. (2012) Ecology & Evolution 2:444-453 not included bc no het calcs and raw data not provided to calculate by hand
	* Bester-van der Merwe et al. (2017) PLoS ONE 9:e0184481: Chile & New Zealand sites not included bc previously reported in Hernandex et al. (2015) Fish. Res. 167:132-142; excluding Algulhaus Bank pop as has many loci out of HWE; not reporting repeats bc don't know loci names to pull from

03.25.2020Continuing to pull data for msat papersNotes:
	* Boissin et al. (2016) Molecular Ecology 25:2195-2209: Not including France/Tunisia sites bc don't have het calcs (and no raw scores to calculate by hand)

03.26.2020Continuing to pull data for msat papersNotes:
	* Boissin et al. (2019) Scientific Reports 9:15816: lat/long coordinates given are average of sampled sites for each location; Mozambique Channel, Tuamotu, Phoenix, Tuvalu excluded bc span >3 deg lat/long range
	* Bonin et al. (2015) Mar. Biodiv. 46:135-140: Amphiprion akindynos & Amphiprion omanensis not included bc all markers out of HWE (het deficit), CF27 & CF36 not included bc out of HWE (het deficit)
	* Borrell et al. (2012) ICES Journal of Marine Science 69:1357-1371: excluded bc msat het calculations pooled across >3 degree range and raw freqs not provided to calc by hand
	* Bozano et al. (2014) Biology and Environment: Proceedings of the Royal Irish Academy 115B:211-220: excluded bc can't get lat/long coordinates for samples
	* Broderick et al. (2011) Journal of Fish Biology 79:633-661: excluded bc msat het calculations pooled across >3 degree range and raw freqs not provided to calc by hand
	* Buchholz-Sørensen & Vella (2016) PLoS ONE 11:e0159864: only recording Malta & Hurd Bank pops bc only ones het provided for; Malta: GAG038, RHCA002, RHCA008, EM10, GAG007, SC06, RHCA007 & Hurd Bank: RHCA002, EM10, GAG049, SC06 not included bc out of HWE; RHCA003 not included for Hurd Bank bc no het reported
	* Byrne, Bernardi & Avise (2013) Journal of Heredity 104:521-531: excluded bc msat het calculations pooled across >3 degree range and raw freqs not provided to calc by hand
	* Calò et al. (2016) Fisheries Research 176:30-38: Omel02 at Les Embiez & Torrevieja, Omel61 at Cavalaire & Cap Creus MPA not included bc not in HWE


03.27.2020

Continuing to pull data for msat papers

Notes:
	* 

Canales-Aguirre et al. (2016) PLoS ONE 11:e0160670: Zone_A: Spfu_9, Spfu_29, Spfu_30, Spfu_44, Spfu_45, Spfu_48, Zone_B: Spfu_29, Spfu_44, Zone_D: Spfu_29, Spfu_30, Zone_E: Spfu_9, Spfu_29, Spfu_30, Spfu_44, Zone_H: Spfu_44, Zone_I: Spfu_9, Spfu_29, Spfu_44, Spfu_45, Spfu_48, Zone_J: Spfu_9, Spfu_29, Spfu_44, Zone_K: Spfu_6, Spfu_29, Spfu_44, Zone_L: Spfu_9, Spfu_29, Spfu_30, Spfu_44, Zone_N: Spfu_9, Spfu_29, Spfu_44, Spfu_45, Spfu_48 excluded bc not in HWE
	* 
Carlsson et al. (2011) 58:627-636: Mound 1 & Mound 2 using year with largest sample size or most recent year; Hat54 excluded in Flat 1, Mound 2, Mound 4, Mound 5 & Mound 6 bc not amplified; Hat16 (Mound 2 & Mound 6) Hat6 (Mound 3), Hat 2a, Hat 14, Hat25 (Mound 6) excluded bc not in HWE
	* 
Carson et al. (2011) Fishery Bulletin 109:416-428: Lan11 in St. Croix & Och4 in Florida Keys excluded bc not in HWE (null alleles)



03.30.2020

Continuing to pull data for msat papers

Notes:
	* 
Castillo-Olguín,Uribe-Alcocer & Díaz-Jaimes (2012) Ciencias Marinas 38:635-652: Cli-13 (Sinaloa, Nayarit, Chiapas) excluded bc monomorphic; Cli-12 (Sinaloa, Nayarit, Campeche, Tabasco), Cli-13 (Baja California Sur), Cli-106(Campeche, Tabasco), Cli-112 (Baja California, Baja California Sur, Sinaloa) excluded bc not in HWE; Cli-13 (Chiapas) excluded bc monomorphic
	* 
Catrino et al. (2017) PLoS ONE 12:e0174988: MAR1-3 lat/long coords average of those provided in paper; ROC11 shown and not ROC_12 bc ROC_11 has larger sample sizes; MED1-2 not include bc lat/long coords >3 deg range and raw genotype not provided to calculate by hand
	* 
Ceballos et al. (2016) Heredity 116:372-377: San Antonio Oeste not included bc previously reported in Ceballos et al. (2012) Mar. Biol. 159:499-505
	* 
Cerda et al. (2019) J. Appl. Ichthyol. 35:1147-1153: Las Cruces & Isla Negra combined into one location (Litoral Central) bc >3 lat/long deg apart; Gle-1 (Litoral Central & Coquimbo), Gle-9 (Litoral Central), Gle-5.2 (Coquimbo) excluded bc not in HWE;Gle-10 (Los Molles) excluded bc monomorphic
	* 
Chaoui et al. (2012) Molecular Ecology 21:5497-5511: Only reporting stats from anonymous SSRs
	* 
Cheng, Zhu & Chen (2014) Conserv. Genet. 15:1021-1035: SJT18 (DongGang, LiuHe, SanSha), SJT49 (DongGang), SJT218 (ShiDao), SJNT62 (ShiDao, LiuHe, SanSha, ZhuHai, SanYa), SJT36 (LianYunGang), SJT51 (LianYunGang, SanYa), SJT205 (LianYunGang, SanSha), SJNT66 (LianYunGang, BaiMaJing), SJNT142 (LianYunGang, DongShan, SanYa), SJT152 (BaiMaJing), SJT153 (BaiMaJing, SanYa) excluded bc not in HWE; SJNT66 (SanYa) excluded bc monomorphic
	* 
Córdova-Alarcón et al. (2019) Front. Mar. Sci. 6:748: excluded bc reported het stats across all sampling sites >3 lat/long deg range and raw allele freqs not provided to calc by hand
	* 
Coscia et al. (2012) Heredity 108:537-546: Using het stats for neutral ssrs provided in supp table (13 putatively neutral markers)
	* 
Coscia et al. (2018) Deep-Sea Research Part I 132:60-67: East Greenland, George Bank, Hatton Bank, Norway excluded bc can't pull specific coordinates (site names not exact enough)
	* 
Cuveliers et al. (2011) Molecular Ecology 20:3555-3568: Only using 2007 sites; excluding Sos(AC)30 & Sos(AC)40 bc out of HWE at all 3 sites



03.31.2020

Continuing to pull data for msat papers

Notes:
	* 
Cuveliers et al. (2012) Mar. Biol. 159:1239-1253: Keeping sites with largest sample size; excluding THA07, BEL07, NOR07 because those sites previously reported in Cuveliers et al. (2011) Molecular Ecology 20:3555-3568; SoS(AC)45 (Kattegat, Ger_North Sea, Celtic Sea, Irish Sea, Eng_English Channel, WHC_English Channel), SolGA12 (Kagerrak, WHC_English Channel, Bay of Biscay), SolCA13 (Ger_North Sea), SoS(AC)6 (Linc_North Sea, Irish Sea, Eng_English Channel, Bay of Biscay), F8-ICA9 (Celtic Sea), F8-ITG11 (Irish Sea), Ssegata26 (Irish Sea, Wadden Sea), Sseca28 (Wadden Sea) excluded bc not in HWE



04.01.2020

Continuing to pull data for msat papers

Notes:
	* 
Dahle et al. (2018) BMC Genetics 19:42: Only using dataset 1 (bc all sites included there); if site sampled more than once, using one with largest sample size; excluding site if more than half (3 or more) microsats out of HWE (Godøy, Byrknesøy Gulen); excluding some sites bc can't get good lat/long coordinates from google maps (Repparfjord, Fiskerfjord Tjeldsund, Saltenfjord Hopen, Skjerstadfjord Valnesfjord, Langesundet, Vikna, Vinjefjord, Batnesfjord, Buagrunn, Nærøysund, Finnøy, Siragrunnen, Lista nordvest, Risør ytre)
	* 
D'Aloia et al. (2014) Molecular Ecology 23:2902-2913: Excluded bc can't get lat/long coordinates for sampling sites
	* 
Damerau et al. (2012) Polar Biol. 35:1073-1086: Only pulling from South Orkney Islands, bc Antarctic Peninsula site spread out over too big a range; Cr15 (C. aceratus, C. rastrospinosus), Trne53 (C. aceratus, C. rastrospinosus, T. newnesi), Ca26 (C. aceratus, T. newnesi), Cr127 (C. gunnari, C. rastrospinosus), Trne20 (C. gunnari, C. rastrospinosus, G. gibberifrons, T. eulepidotus), Cr38 (C. rastrospinosus, G. gibberifrons), Cr236 (C. rastrospinosus), Cr259 (C. rastrospinosus, G. gibberifrons, T. eulepidotus), Trne35 (G. gibberifrons, T. eulepidotus), Trne37 (L. squamifrons) excluded bc out of HWE; Cr127 (G. gibberifrons), Trne55 (G. gibberifrons) excluded bc not amplified in S. Orkney Islands pop



04.02.2020

Continuing to pull data for msat papers

Notes:
	* 
Damerau et al. (2014) Molecular Ecology 23:284-299: Excluding South Orkney Island sites because previously reported in Damerau et al. (2012) Polar Biol. 35:1073-1086; C. aceratus: Cr127 (Bouvet Island), Trne53 (Elephant Island), Ca26 & Cr259 (South Georgia), C. gunnari: Cr38 (Bouvet Island, Elephant Island, South Georgia), Cr236 (Bouvet Island) Trne20 (Bouvet Island, Elephant Island, South Georgia), Trne53 (Boutvet Island, Elephant Island), Cr15 (South Georgia) excluded bc not in HWE
	* 
Damerau et al. (2014) Molecular Ecology 23:284-299: Trne37 (South Orkney Islands), Trne66 (South Georgia, Bouvet Island) excluded bc not in HWE
	* 

Dammannagoda et al. (2011) Canadian Journal of Fisheries and Aquatic Sciences 68:210-223: UTD328 (Negombo, Weligama, Tangalle, Kalmunei, Trincomalee, Laccadive, Maldives), UTD535 (Weligama), UTD523 (Weligama), UTD172 (Weligama, Tangalle, Kalmunei, Laccadive, Maldives), UTD73 (Tangalle, Kalmunei, Trincomalee), UTD523 (Laccadive, Maldives) excluded bc not in HWE
	* 
D'Anatro, Pereira & Lessa (2011) Environ. Biol. Fish 91:407-420: Soc201 (Laguna de Rocha), Soc204 (Laguna de Rocha, San Luis), Soc049 (Laguna de Rocha), Soc085 (La Paloma, San Luis), Soc243 (Montevideo), Soc049 (Montevideo), Soc252 (Laguna de Castillos, San Luis) excluded bc not in HWE
	* 
Davies et al. (2011) Mar. Biol. 158:2727-2740: Central Med only keeping 07 sample bc in same location and same sampling size (07 more recent); Tth4 (MED_05), Tth254 (MED_05), Tth17 (MED_07, CS2_05), Tth16-2 (CS2_05, CS1_06, BB1_06), Ttho6 (WI2_07) excluded bc not in HWE



04.11.2020

Continuing to pull data for msat papers

Notes:
	* 
Díaz-Ferguson et al. (2012) Ciencias Marinas 38:129-142: Sol GA12 (Lisbon, Rio San Pedro, Cadiz Bay), Sol MIJ (Punta Umbria), Smax-02 (Chipiona, Rio San Pedro), Sse 38 (Rio San Pedro) excluded bc not in HWE
	* 
DiBattista, Feldheim & Bowen (2011) Conservation Genet. Resour. 3:159-162: Ahy54 excluded bc not in HWE
	* 
DiBattista et al. (2012) Journal of Heredity 103:617-629: excluded bc het calculated across sites >3 deg apart and raw seqs not provided to calculate by hand
	* 
DiBattista et al. (2015) J. Biogeogr. 42:1919-1929: excluded bc het calculated across sites >3 deg apart and raw seqs not provided to calculate by hand
	* 
Dick, Shurin & Taylor (2014) Molecular Ecology 23:575-590: excluded bc het stats not provided (Supplemental File they are found in is corrupted)
	* 
Divya et al. (2019) Marine Biodiversity 49:381-393: Kolkata excluded bc het calculated across sites >3 deg apart; RCalD11 (Veraval, Cochin, Chennai), RCalE05 (Veraval, Cochin, Chennai), RCalBA04 (Veraval), RCalBA11 (Veraval, Cochin, Chennai), RCalH01 (Veraval), RCalH10 (Veraval, Cochin, Chennai), RCalBD08 (Cochin), RCalBF07 (Cochin), RCalD08 (Chennai), RCalBD10 (Chennai), RCalH01 (Chennai), RCalD07 excluded bc not in HWE
	* 
Dohna et al. (2015) Ecology and Evolution 5:1717-1733: Okinawa, Misool, Pisang, Papisol & Triton Bay excluded bc either no data or no individuals sampled



04.12.2020

Continuing to pull data for msat papers

Notes:
	* 
Domingues et al. (2019) Aquatic Conserv: Mar. Freshw. Ecosyst. 29:102-114: Excluding Gulf of Mexico, US Atlantic Coast & Venezuela bc no lat/long coordinates provided and too broad to get by place name; Northern Brazil (Pernambuco) excluded bc not considered in msat analyses; Cli-100 excluded from Southern Brazil (Santa Catarina) bc not in HWE
	* 
Drinan et al. (2016) Journal of Fish Biology 89:2571-2594: Only using dataset 1 bc other data set includes EST-linked markers; HER excluded bc long coordinates broader than 3 deg
	* 
Eble et al. (2011) Mar. Ecol. Prog. Ser. 428:245-258: Only using Chichi-jima, Saipan & Pohnpei sites as others have individuals previously reported in Eble, Toonen & Bowen (2009) Mar. Biol. 156:689-698
	* 
Eiríksson & Árnason (2013) Can. J. Fish. Aquat. Sci. 70:1151-1158: Excluded bc het stats calculated across >3 lat/long degrees
	* 
Fauvelot & Borsa (2011) Biological Journal of the Linnean Society 104:886-902: D61Sc (New Caledonia), E27Sc (Java Sea),  L42Sc (Bali), excluded bc not in HWE
	* 
Félix-Hackradt et al. (2013) Marine Environmental Research 92:23-34: M. barbatus: Mbar130 (Garrucha), M. surmuletus: Mbar11 (Adra, San Pedro, Santa Pola, Valencia), Mbar130 (Adra, Aguilas, Santa Pola, Javea, Valencia) excluded bc not in HWE



04.13.2020

Continuing to pull data for msat papers

Notes:
	* 
Fernandez-Silva et al. (2015) J. Biogeogr. 42:2402-2413: Mo'orea, Saipan, Mahe, Diego Garcia, Faafu Atoll, Muscat, Farasan Banks, Thuwal, Yanbu & Gulf of Aqaba/Eilat excluded bc no microsat stats provided (and no raw seqs to calculate by hand)
	* 
Franchini et al. (2012) ICES Journal of Marine Science 69:41-50: Lesina only using 2001 sample bc largest # individuals; SaGT01, SaGT26, SaGT31, SaGT32 (Gulf of Cadice, Lesina, Bacoli, Sabaudia, Tortoli, Santa Gilla, Cabras) excluded bc already reported in De Innocentiis et al. (2004) Fisheries Science 70:852-859; SaGT32 (Trapani), Pb-Ovi-D106 (Bisceglie), SaUD69 (Santa Gilla) excluded bc not in HWE
	* 
Frédérich, Liu & Dai (2012) Evol. Biol. 39:359-370: Pom25 (Nakanoshima Island), Pom2269 (Sesoko Island) excluded bc not in HWE
	* 
Gaither et al. (2011) PLoS ONE 6:e28913: Excluding Hawai'i, New Caledonia, Gascoyne bc >3 deg lat/long; excluding Necker bc sample size <4; excluding Raita, Salmon Banks bc can't get lat/long coordinates from name
	* 
Galindo, Loher & Hauser (2011) Mar. Biotechnol. 13:1027-1037: Using largest sample size from each location (only recording once per location -- whichever is biggest by date and by sex); Hhi 59 (Queen Charlotte's Island, Portlock Bank) excluded bc not in HWE
	* 
Gao et al. (2016) Biochemical Systematics and Ecology 68:170-177: SR7-7 & SR15-8 excluded from Yantai bc didn't amplifed well
	* 
Garcia et al. (2019) Molecular Biology Reports 46:5525-5530: DELIG-22, DELIG-23, DELIG-31, DELIG-32 (Iquique) excluded bc monomorphic; DELIG-6 (Puerto Montt), DELIG-12 (Puerto Montt), DELIG-15 (Puerto Montt), DELIG-25 (Puerto Montt), DELIG-18 (Iquique) excluded bc not in HWE
	* 
Gardner et al. (2017) Environ. Biol. Fish. 100:1411-1425: PauD118 (Denham Sound, Monkey Mia) excluded bc not in HWE
	* 
Gharrett, Riley & Spencer (2012) Transactions of the American Fisheries Society 141:370-382: Some sites (AL3 & AL4) >180, looks like should subtract excess over 180 from 180 and change from W to E based on same location coordinates on google maps
	* 
Ghasemi & Shadi (2018) Turk. J. Fish. Aquat. Sci. 18:983-990: S32 (Genaveh, Bushehr) excluded bc not in HWE
	* 
Glover et al. (2011) ICES Journal of Marine Science 68:2145-2151: Using het stats from dataset of 5 loci (ones not showing null alleles/in HWE); excluding GOT (Baltic Sea, Gotland) sample bc some individuals already reported in Limborg et al. (2009) Marine Ecology Progress Series 379:213-224
	* 
Gold et al. (2011) North American Journal of Fisheries Management 31:209-223: Lca20 (Florida Keys) excluded bc not in HWE 



04.14.2020

Continuing to pull data for msat papers

Notes:
	* 
Gonzalez, Aritaki & Taniguchi (2012) Fisheries Science 78:603-611: Excluded bc het stats presented in bar graph format only
	* 
Gonzalez et al. (2015) ICES Journal of Marine Science 72:635-641: Rockall & Vesteraalen using sampling time point with largest sample size; Faroe Islands: Bbrom2, MmolM12, MmolC1 excluded bc not in HWE



04.16.2020

Continuing to pull data for msat papers

Notes:
	* 
González-Wangüemert & Pérez-Ruzafa (2012) Marine Ecology 33:337-349: Excluding coastal lagoon pops bc not sure truly "marine"; DVUL4 (Banyuls, Murcia, Quarteira), DVUL61 (Banyuls, Murcia, Galicia, Quarteira), DVUL6 (Banyuls, Murcia, Galicia, Quarteira) excluded bc not in HWE (null alleles)
	* 
González-Wangüemert et al. (2012) Journal of Sea Research 68:41-48: Only including marine wild populations (not aquaculture or pop that is restocked from aquaculture farm); DVUL12 (Pachino, Marsala, Messina), DVUL6 (Pachino, Marsala), DVUL61 (Messina) excluded bc not in HWE; SAI19 (Marsala) excluded bc monomorphic
	* 
Green et al. (2019) Marine and Freshwater Research 70:1007-1019: Only keeping Australia & Seychelles pop bc PNG calculated across >3 lat/long range; ALS9 (Seychelles), ALS11 (Seychelles, East Australia), ALLS14 (Seychelles), ALS51(Seychelles, East Australia) excluded bc monomorphic; East Australia: ALS7 & ALS42 excluded bc not in HWE
	* 
Guinand et al. (2013) Journal of Sea Research 80:35-49: Excluded bc He not reported (only Ho) and raw seqs not provided to calculate on own
	* 
Guinand et al. (2016) Mar. Ecol. Prog. Ser. 558:115-127: Excluded bc He not reported and raw seqs not provided to calculate on own
	* 
Gwilliam et al. (2018) Journal of Fish Biology 92:1163-1176: Dvu-I4 (Port Elizabeth, Tsitsikamma, Flamingo), Dvu-I33 (Port Elizabeth, Tsitsikamma), Ome-I58 (Port Elizabeth), Dsa-MS34 (Tsitsikamma, Flamingo), Dvu-I84 (Flamingo) excluded bc not in HWE
	* 
Hargrove et al. (2018) Genetica 146:393-402: Excluded bc He calculated across >3 lat/long range
	* 
Harrison et al. (2014) Conservation Genet. Resour. 6:803-806: ONLY ONE POP PAPER; Lca027, Lca582; Lca107; Lca411; Lca492 excluded bc not in HWE
	* 
Healey et al. (2018) ICES Journal of Marice Science 75:1465-1472: Kenya, Tanzania & South Africa excluded bc not in HWE; Seychelles excluded bc He calculated across >3 lat/long range



04.17.2020

Continuing to pull data for msat papers

Notes:
	* 
Andreotti et al. (2016) Mar. Ecol. Prog. Ser. 552:241-253: Excluded bc He stats not provided and raw seqs not given to calculate by hand
	* 
Haffrey et al. (2012) Aquat. Living Resour. 25:173-183: Excluding aquaculture sites; modified some lat/long coordinates based on site names bc ones provided in text (mainly long) didn't seem right



Pulling papers (second 100) from msat paper pass list and saving to computer (saving under Marine Genetic Diversity Papers/msat_Paper)

04.18.2020

Continuing to pull data for msat papers

Notes:
	* 
Helyar et al. (2011) Conservation Genet. Resour. 3:173-176: Cc-02 & Cc-34 excluded from both MAR & Azores bc not in HWE
	* 
Henriques et al. (2014) PLoS ONE 9:e87907: Geelb5 (Port Alfred), Geelb16 (Arniston), Geelb29 (Port Alfred), Geelb32 (Lucira) excluded bc not in HWE



04.20.2020

Continuing to pull data for msat papers

Notes:
	* 
Henriques et al. (2016) Molecular Ecology 25:5843-5861: Excluded bc He calculated >3 degree lat/long range
	* 
Henriques et al. (2017) Fisheries Research 187:86-95: Reporting sample with largest sample size/most recently collected per location of those that are in HE; excluded Child's Bank & Eastern Agulhas bank bc not in HWE
	* 
Henriques et al. (2018) African Journal of Marine Science 40:13-24: Flamingo River excluded bc can't get lat/long coordinates



04.21.2020

Continuing to pull data for msat papers

Notes:
	* 
Herrera et al. (2016) Biol. Lett. 12:20160309: Excluded bc He not provided and don't have raw sequences to calculate by hand
	* 
Hess et al. (2014) Can. J. Fish. Aquat. Sci. 71:81-92: Excluded bc He calculated by clade not by sampling site
	* 
Horne et al. (2013) Journal of Heredity 104:532-546: Only including sample with largest sample size for sites sampled at more than 1 time point
	* 
Hull et al. (2019) Hydrobiologia 838:121-137: Excluding Adriatic Sea, Angola, South Africa bc He calculated >3 deg lat/long range
	* 
Huyghe & Kochzius (2018) PLoS ONE 13:e0193825: Excluded bc He not provided and don't have raw sequences to calculate by hand
	* 
Jackson et al. (2014) PLoS ONE 9:e97508; Excluded A108 from all populations bc some were out of HWE (and don't know which); A10 (Turneffe Atoll, Caye Glory, Bajo de Sico), A117 (Grand Cayman), A6 (Dog Rocks, Lee Stocking), A3 (Dog Rocks, Bajo de Sico), A113 (St. Thomas) excluded bc out of HWE (think)



04.22.2020

Continuing to pull data for msat papers

Notes:
	* 
Jakobsdóttir et al. (2011) Evolutionary Applications 4:562-573: Excluded bc het stats not provided for neutral microsats and raw seqs not included to calc by hand
	* 
Jansson et al. (2017) ICES Journal of Marine Science 74:2135-2147: A Coruña, Mulroy Bay & Varberg excluded bc not in HWE (het deficit)
	* 
Jue, Coleman & Koenig (2014) Marine Biology 161:1905-1918: Excluded bc het stats not provided and raw seqs not given to calc by hand
	* 
Jue et al. (2015) PLoS ONE 10:e0120676: excluded bc can't get lat/long coordinates (location name not specific enough)



05.11.2020

Continuing to pull data for msat papers

Notes:
	* 
Karl, Castro & Garla (2012) Mar. Biol. 159:489-498: Brazilian Island lat/long coordinates averaged from two sites that were pooled together; Gc-07 & Gc-10 (Natal) excluded bc not in HWE
	* 
Kempter et al. (2017) Food Chemistry 221:944-949: Fenoarivo excluded bc sample size <4; Phnom Pehn excluded bc sourced from market and don't know where on coast was caught
	* 
Kennington et al. (2017) Fisheries Research 193:242-249: Reporting samples with largest sample size or most recent time period; excluding Arabis Seamount & Rotuma Shoals & Tonga bc >3 lat/long coords apart
	* 
Kijewska, Więcaszek & Kijewski (2011) Acta Ichthyologica et Piscatoria 41:307-314: Puck Bay: all loci except for Tch11 excluded bc not in HWE, Pomeranian Bay, Øresund, Isle of Bear excluded bc all loci not in HWE, Gulf of Gdańsk: all loci except for Gmo8 excluded bc not in HWE, Flamborough Head: Gmo35, Tch11, Tch12 excluded bc not in HWE
	* 
Klein et al. (2016) PLoS ONE 11:e0165881: Reporting samples with largest sample size or most recent time period; ARR_A & ARR_R treated as same sampling location (keeping ARR_A_12 over ARR_R_12 despite same sample size as all other samples were adults, not recruits); Lp13 (São Martinho do Porto, Peniche, Lisbon, Marine Park of Arrábida, Almograve, Barranco), Lp14 (Peniche, Marine Park of Arrábida, Barranco), Lp24 (São Martinho do Porto, Lisbon, Marine Park of Arrábida), Lp9 (Lisbon), Lp20 (Marine Park of Arrábida), Lp21 (Sines, Almograve) excluded bc not in HWE
	* 
Knutsen et al. (2011) Molecular Ecology 20:768-783: Excluded bc no het stats and raw data not provided to calculate by hand
	* 
Knutsen et al. (2012) Marine Ecology Progress Series 460:233-246: Excluded bc no het stats and raw data not provided to calculate by hand
	* 
Kongseng, Phoonsawat & Swatdipong (2020) Fisheries Research 221:105372: Excluded bc all pops not in HWE



05.12.2020

Continuing to pull data for msat papers

Notes:
	* 
Kousteni et al. (2015) Heredity 114:333-343: Scan02 (Ionian Sea), Scan03 (Ionian Sea, Cyclades Island) excluded bc not in HWE



05.14.2020

Continuing to pull data for msat papers

Notes:
	* 
Kousteni et al. (2016) Medit. Mar. Sci. 17:371-383: DFH429 (North Aegean Sea), Saca6396 (Cyclades Islands) excluded bc not in HWE
	* 
Lane, Symonds & Ritchie (2016) Fisheries Research 174:19-29: Otago all loci except for GH0OIK and Hokitika all loci excluded bc not in HWE
	* 
Le Port et al. (2017) Proc. R. Soc. B 284:20171300: For CROP MPA, only looking at adults; non-MPA and MPA basically overlap -- should treat as two or one site?; excluded Pma28-28, Pma2, CM1742, Pma4-32, CM1143 from both sites bc big diff between He & Ho (Fis > 0.100) -- don't know if out of HWE but likely something weird here (null alleles) as both out in both sites
	* 
Li, Zhong & Wang (2013) Zoological Science 30:83-91: Excluded bc all sites out of HWE in both species (null alleles)
	* 
Li et al. (2013) Biochemical Systematics and Ecology 50:331-338: LYC0016 & LYC0018 excluded in all pops bc out of HWE (null alleles)
	* 
Li et al. (2019) Aquat. Living Resour. 32:8: par20 (Sonmiani, Ormara) & par12 (Sonmiani, Ormara, Beihai) excluded bc monomorphic
	* 
Limborg et al. (2012) Heredity 109:96-107: Only keeping sites reported exclusively in this study and not those reported in Limborg et al. 92009) Mar. Ecol. Prog. Ser. 379:213-224;  excluding the two Black Sea sites (west and east) bc not truly marine; Spsp133 (Eastern Skagerrak, English Channel, Bay of Biscay), Spsp275 (Gulf of Lion) excluded bc not in HWE



05.15.2020

Continuing to pull data for msat papers

Notes:
	* 
Liu et al. (2012) Mar. Ecol. Prog. Ser. 458:155-167: Excluded bc msat stats grouped across regions >3 degrees lat/long and raw data not provided to calculate by hand



05.16.2020

Continuing to pull data for msat papers

Notes:
	* 
Liu et al. (2016) PLoS ONE 11:e0154020: Lpol04 (Dongying), Lpol06 (SYA), Lpol17 (SYB), Lpol05 (SYC), Lpol09 (SYC), Lpol16 (Yangtze River Estuary, Xiapu) excluded bc not in HWE



05.17.2020

Continuing to pull data for msat papers

Notes:
	* 
Liu et al. (2017) Fisheries Research 194:50-54: Excluded bc species/sites already recorded in Liu et al. (2016) PLoS ONE 11:e0154020
	* 
Loher et al. (2016) Journal of Sea Research 107:90-99: Excluded bc het not provided and raw sequences not given to calculate by hand
	* 
Longmore et al. (2014) Can. J. Fish. Aquat. Sci. 71:1-12: Excluded bc het not provided and raw sequences not given to calculate by hand
	* 
Lotterhos & Markel (2012) Molecular Ecology 21:5009-5026: Treating all sampling locations as one site bc het stats provided that way and <3 deg lat/long; Spi10 excluded bc not in HWE
	* 
Lotterhos , Dick & Haggarty (2013) Evolutionary Applications 7:238-259: Excluded Barkley Sound site bc previously recorded for same species/location/loci in Lotterhos & Markel (2012) Molecular Ecology 21:5009-5026
	* 
Luo et al. (2015) Fisheries Research 172:318-324: Excluded bc het not provided and raw sequences not given to calculate by hand
	* 
Ma et al. (2018) BMC Evolutionary Biology 18:190: E. polyphekadion: French Polynesia, New Caledonia, Palau, Philippines, Maldives excluded bc >3 degrees lat/long & EP25 (Fiji), EP35 (Fiji), EP34 (Fiji), EP33 (Pohnpei) excluded bc not in HWE & EP11 (Fiji), EP28 (Scott Reef) excluded bc monomorphic; P. areolatus: Solomon Islands, Palau, East Indonesia, Maldives excluded bc >3 degrees lat/long & Pm3 (Cocos (Keeling) Islands), Pb111 (Thuwal) excluded bc not in HWE; P. lepoardus New Caledonia, Palau, Palawan excluded bc >3 degrees lat/long & Pma106 (Fiji, Townsville, Swains Reef, Cebu), Pm3 (Fiji, Taiwan, Scott Reef), Pleo005 (Taiwan), Pma109 (Hick's Reef, Lizard Island, Townsville, Swains Reef), Ple002 (Swains Reef), Pma112 (Hick's Reef, Lizard Island, Townsville, Swains Reef, Taiwan, Scott Reef, Abrolhos Islands) excluded bc not in HWE



05.18.2020

Continuing to pull data for msat papers

Notes:
	* 
Machado-Schiaffino, Juanes, Garcia-Vazquez (2011) Biological Conservation 144:330-338: If temporal replicates, keeping sample with largest sample size; BGM1 excluded bc most of loci are out of HWE
	* 
Madduppa, Timm & Kochzius (2013) PL0S ONE 9:e90648: Averaged lat/long coordinates for each site (<3 deg lat/long); A. ocellaris: AC1578 (Samalona) excluded bc not in HWE; A. perideraion: CF42 & AC1578 excluded bc not in HWE
	* 
Madsen et al. (2016) Polar Biol 39:969-980: Changing Shelf break north (Shn) coordinates from 76 98' N to 77 bc google maps not finding 98'?; tch-5 (Shelf west), gmo-c71 (Shelf west), gmo-c291 (Dove Bugt), gmo-08 (Shelf break middle) excluded bc not in HWE; gmo-c291 (Tyrolerjorden, Billefjord) excluded bc monomorphic; Dove Bugt: tch-5 doesn't have N, so using 42 bc that is what most other loci have (42 or 43)



05.19.2020

Continuing to pull data for msat papers

Notes:
	* 
Maduna et al. (2016) Mar. Ecol. Prog. Ser. 544:183-196: Angola lat/long coords averaged between two sites provided in text; Mh25 (Angola, Langebaan Lagoon MPA, Robben Island, False Bay, Kalk Bay, Struis Bay, Jeffreys Bay, Durban), McaB39 (Angola, Langebaan Lagoon MPA, Robben Island, False Bay, Kalk Bay, Jeffreys Bay, Durban), McaB37 (Robben Island), Mh2 (False Bay, Durban), Mca25 (Struis Bay, Jeffreys Bay), McaB5 (Struis Bay, Durban), McaB6 (Struis Bay, Durban), McaB22 (Struis Bay, Jeffreys Bay), Mh1 (Durban) excluded bc not in HWE; McaB27 (Angola, Langebaan Lagoon MPA, Struis Bay, Durban), McaB37 (Angola), Mh1 (Langebaan Lagoon MPA, Robben Island) excluded bc monomorphic
	* 
Maduna et al. (2016) Ecology and Evolution 7:1462-1486: Mustelus mustelus excluded bc previously reported in Maduna et al. (2016) Mar. Ecol. Prog. Ser. 544:183-196 -- diff markers but same sites; SEAO & SWIO lat/long coords averaged when necessary; G. galeus SWIO excluded bc >3 lat/long range; M. palumbes: Mmu8 (Mossel Bay), Mmu11 (Mossel Bay), Mmu13 (Mossel Bay), Mmu6 (Mossel Bay), G. galeus: Mmu11 (SEAO), T. megalopterus Mmu3 (Port Elizabeth), Mmu11 (SWIO), Mmu5 (SEAO) excluded bc not in HWE
	* 
Magallón-Gayón, Diaz-Jaimes & Uribe-Alcocer (2016) PeerJ 4:e2583: Temporal samples averaged for msat analyses at each site; Sbr18 (Santa Clara) excluded bc not in HWE; Ta149 (Santos) excluded bc monomorphic




05.20.2020

Continuing to pull data for msat papers

Notes:
	* 
Mamoozadeh et al. (2018) ICES Journal of Marine Science 75:892-902: Western Central Atlantic, Caribbean Sea, Morocco, Gulf of Mexico excluded bc can't get precise lat/long coordinates
	* 
Matić-Skoko et al. (2018) Scientific Reports 8:676: M. barbatus: Mbar3, Mbar130, Mbar101, Mbar63 excluded from all pops bc out of HWE, South Adriatic, Levantine Sea (Isreal & Cyprus), Balearic Sea pops excluded bc out of HWE; M. surmuletus: Mbar130, Mbar133 excluded from all pops bc out of HWE, North Adriatic (Croatia), Levantine Sea (Cyprus), Balearic Sea, Portugal pops excluded bc out of HWE
	* 
McCusker & Bentzen (2011) Mar. Biol. 158:1869-1878: Excluded bc can't get exact lat/long coordinates (>3 deg)
	* 
Miralles, Juanes & Garcia-Vazquez (2014) Transactions of the American Fisheries Society 143:1308-1315: Florida excluded bc can't get exact lat/long coordinates (>3 deg)
	* 
Montanari et al. (2011) Ecology and Evolution 2:310-329: Excluded Christmas Island hybrid sample; excluded Marshall Islands sample (C. lunulatus) bc can't get exact lat/long coords (>3 deg); C. lunulatus: Lun07 (Christmas Island), C. trifasciatus: Lun05 (Christmas Island), Lun10 (Christmas Island, Cocos (Keeling) Islands, Zanzibar), Lun29 (Christmas Island, Cocos (Keeling) Islands), Lun36 (Cocos (Keeling) Islands) excluded bc not in HWE
	* 
Montes et al. (2012) Mar. Ecol. Prog. Ser. 471:183-191: Ttho-1 (Adriatic Sea), Ttho-4 (Adriatic Sea, Balearic Sea), Ttho-6 (Adriatic Sea, Balearic Sea, South Atlantic, Indian Ocean, North Pacific), tth62 (Adriatic Sea, Tyrrhenian Sea, Balearic Sea, Celtic Sea, South Atlantic, Indian Ocean, North Pacific, South East Pacific, South West Pacific), tth1-31 (Tyrrhenian Sea, Balearic Sea, Celtic Sea), tth178 (Tyrrhenian Sea, Balearic Sea, Celtic Sea, North Pacific), tth14 (Balearic Sea), Tth-5 (Balearic Sea, Celtic Sea, South Atlantic, Indian Ocean, North Pacific, South East Pacific, South West Pacific), tth226 (South Atlantic), tth4 (South Atlantic, South West Pacific), Ttho-7 (Indian Ocean, South West Pacific) excluded bc not in HWE
	* 
Moore & Chaplin (2013) Environ. Biol. Fish 96:1087-1099: Excluded bc not actually sure these are microsatellite loci
	* 
Muths & Bourjea (2011) Conservation Genet. Resour. 3:629-631: EPI-04 (Mohéli), EPI-53 (Mohéli, Moroni), EPI-16 (Moroni) excluded bc not in HWE



05.21.2020

Continuing to pull data for msat papers

Notes:
	* 
Moore & Chaplin (2013) Environ. Biol. Fish 96:1087-1099: Could calculate by hand with allele frequencies provided in supp BUT wouldn't know which were out of HWE bc don't have Ho to compare with, so sticking with mean/average by site for now
	* 
Muths et al. (2011) Mar. Ecol. Prog. Ser. 443:167-180: Excluded bc don't provide het and no raw sequences to calculate by hand
	* 
Muths , Tessier & Bourjea (2015) Marine Ecology 36:447-461: Excluded bc all locations out of HWE
	* 
Muths et al. (2015) Journal of Sea Research 95:70-74: Méjean, Jarre, 3PP excluded bc not in HWE; Saména excluded bc <4 individuals; Veyron excluded bc can't get lat/long coords
	* 
Mzingirwa et al. (2019) Molecular Biology Reports 46:5079-5088: Shimone & Mohéli excluded bc all loci out of HWE; Pf1.6B3 (Mahé, Dar es Salaam), Pfi1.7E (Mahé, Dar es Salaam, Toliara, Durban), Pfi2.2E (Mahé, Dar es Salaam, Toliara), Pfi4A (Mahé, Toliara), Pfi1.3A (Mahé, Dar es Salaam, Toliara, Petite Riveire), Pfi1.5C (Mahé, Dar es Salaam, Toliara, Durban), Pfi1.6B2 (Dar es Salaam, Toliara, Petite Reveire, Durban), Pfi2.8E (Dar es Salaam), Pfi2.8A (Toliara), Pfi2.9C (Toliara, Petite Riveire) excluded bc out of HWE
	* 
Nakajima et al. (2014) Can. J. Fish. Aquat. Sci. 71:301-314: Hatchery samples and temporal replicates collected after hatchery release excluded; Sni21 (Ise Bay), Sni29 (Shibushi Bay), Sni13 (Kasasa), Sni26 (Goto, Wakasa Bay) excluded bc not in HWE
	* 
Nance et al. (2011) PLoS ONE 6:e21459: Manta excluded bc don't have precise lat/long coordinates (>3 deg); SLE053 (Mazatlan, Santa Catalina), SLE071 (Santa Catalina), SLE086 (Santa Catalina), Cli12 (Gulf of Panama) excluded bc not in HWE
	* 
Newby, Darden & Shedlock (2014) Copeia 3:503-512: Excluded bc het calculated across all sampling locations (>3 deg) and don't have raw data to calculate by hand
	* 
O'Donnell et al. (2017) Coral Reefs 36:223-232: Excluded D. trimaculatus sites bc some loci are out of HWE but don't know which loci they are
	* 
O'Donnell, Reichert & Darden (2019) North American Journal of Fisheries Management 39:725-737: Carolinas location excluded bc can't get precise lat/long coords (samples span >3 deg lat/long)



05.21.2020

Continuing to pull data for msat papers

Notes:
	* 
O'Leary et al. (2013 PLoS ONE 8:e66126: Excluded bc all sites out of HWE and per/locus he (w/in sites) not provided
	* 
O'Leary et al. (2016) Bull. Mar. Sci. 92:33-50: Excluded bc het stats not provided and raw data not given to calculate by hand
	* 
Otwoma et al. (2018) Hydrobiologia 806:237-250: Microsats developed in a hybrid (part A. leucosternon & part A. nigricans), so recorded as cross species marker; Ahy203, Ahy54, Ahy182, Ahy75 excluded from all pops bc out of HWE; Kisite Mpunguti, Tanga excluded bc out of HWE
	* 
Otwoma & Reuter (2019) Marine Environmental Research 151:104760: A. leucosternon samples excluded bc previously reported in Otwoma et al. (2018) Hydrobiologia 806:237-250



05.22.2020

Continuing to pull data for msat papers

Notes:
	* 
Ouazzani et al. (2017) Marine Biology Research 13:342-350: Excluded all sites except AN25 & AS38 bc out of HWE
	* 
Palof, Heifetz & Gharrett (2011) Mar. Biol. 158:779-792: Lat/long coords are an average across sites combined for a given sample; He provided is corrected (Nei)
	* 
Pampoulie et al. (2011) Transactions of the American Fisheries Society 140:307-319: Excluded bc can't get exact lat/long coords for sample sites
	* 
Pampoulie et al. (2012) ICES Journal of Marine Science 69:508-515: A2 & A3 excluded bc already included in McCusker & Bentzen (2010) Molecular Ecology 19:4228-4241; A1 excluded bc can't get exact lat/long coords
	* 
Pampulie et al. (2014) ICES Journal of Marine Science 71:2390-2397: Only sites 5-13 (Iceland sites except Southwest & Southeast) kept, others excluded because can't get exact lat/long coords
	* 
Pampoulie et al. (2015) Marine Ecology: Progress Series 522:219-230: Cha1059 (NASH), Cha17 (NSSH12) excluded bc not in HWE



05.25.2020

Continuing to pull data for msat papers

Notes:
	* 
Papetti et al. (2012) J. Biogeogr. 39:1361-1372: Kept 2002 sample for Elephant Island only bc larger sample size; South Shetlands excluded bc lat/long coords >3 deg; Cr15 (Elephant Island), Cr127 (Elephant Island, Joinville Island), Cr171 (Elephant Island, Joinville Island, South Orkneys), Cr236 (Elephant Island) excluded bc not in HWE
	* 
Papetti et al. (2013) ICES Journal of Marine Science 70:387-398: Ss142 (Northern-western, Central-western) excluded bc not in HWE
	* 
Phinchongsakuldit et al. (2013) Aquacult. Int. 21:197-217: A11 (Phuket) excluded bc not in HWE
	* 
Piñeros et al. (2015) Evol. Biol. 42:235-249: Absa17 (Lobos Island, Xcalak), 2al2 (Lobos Island)), Absa18 (Tuxpan), Absa30 (Antón Lizardo) excluded bc not in HWE
	* 
Piñeros & Gutiérrez-Rodríguez (2017) Coral Reefs 36:877-890: Excluded Tuxpan Reef System & Veracruz Reef System bc previously reported in Piñeros et al. (2015) Evol. Biol. 42:235-249; Absa30 (Mesoamerican Reef System, Porto Bello-San Blas), Absa1 (Porto Bello-San Blas), Absa17 (Porto Bello-San Blas, Santa Marta), 2al13 (Porto Bello-San Blas) excluded bc not in HWE; 2al10, 2al13, 2al2 excluded from Fernando de Noronha bc amplified in <4 individuals
	* 
Pirog et al. (2019) Ecology and Evolution 9:12980-13000: Seychelles, Mozambique, South Africa, Madagascar, New Caledonia excluded bc can't get accurate lat/long coords; Florida excluded bc not in HWE
	* 
Pita et al. (2016) Fisheries Research 179:291-301: Some samples from same stock very close together (less than half a degree apart), so treating as same location and only keeping one with largest sample size: SS00, SS01, SS02 -> keeping SS00 & SS08, SS10 -> keeping SS08 & NS01, NS02 -> keeping NS02 & NS03, NS05 -> keeping NS05 (more recent) & NS09, NS10 -> keeping NS09; SS00, SS04, SS05 excluded bc all loci out of HWE; Mmer-hk3b (SS03, NS00), Mmer-hk9b (SS03, SS07, SS08, NS00, NS08, NS09), Mmer-hk29 (SS03, SS07, SS08, NS02, NS05, NS08, NS09), Mmer-hk34b (SS03, SS07, NS00, NS02, NS05, NS08), Mmer-hk20b (SS08, NS00, NS05, NS08)
	* 
Pita et al. (2017) Fisheries Research 191:108-119: Excluded bc same species/stock previously reported in Pita et al. (2016) Fisheries Research 179:291-301
	* 
Poortvliet et al. (2013) 3:4558-4571: Monterey Bay, Bahia Asuncion excluded bc sample size <4; A4 (Caleta Chanaral) excluded bc monomorphic; A7 (Caleta Chanarel), C7 (Caleta Chanarel) excluded bc not in HWE
	* 
Porrini et al. (2015) Neotropical Ichthyology 13:431-438: Excluded bc no het stats and allele freqs/raw data not provided in reliable enough format to calculate by hand

		* 
In a figure, but not sure precise enough to be accurate



05.26.2020

Continuing to pull data for msat papers
	* 
Before downloading last set of papers (gave Marial final 50 to pull from), went back and checked to make sure downloaded everything in this set

		* 
Seemed to miss a few, so downloaded those and adding data now



Notes:
	* 
García-De León et al. (2018) PLoS ONE 13:e0194646: Excluded Puget Sound & Northern Gulf of California bc previously reported in Iwamoto et al. (2015) ICES J. Mar. Sci. 72:2720-2731; other sites also collected from an earlier study but that study didn't report on microsatellites so kept; Southern of Baja California site excluded bc can't get latitude coordinates
	* 
Knutsen et al. (2013) PLoS ONE 8:e67492: Ardtoe, Strangeford Lough, Galicia, Lisbon, Algarve excluded bc out of HWE
	* 
O'Bryhim et al. (2015) Conservation Genetics Resources 7:463-465: Excluded bc can't get lat/long coordinates
	* 
Nanninga et al. (2014) Molecular Ecology 23:591-602: Excluded bc no het stats and raw data not provided to pull from



Pulling papers (last 112) from msat paper pass list and saving to computer (saving under Marine Genetic Diversity Papers/msat_Papers)
	* 
Last 50 already pulled and saved for Marial to go through, so pulling middle 62 papers and putting in msat_Papers folder



Continuing to pull data for msat papers from last downloaded set

Notes:
	* 
Portnoy et al. (2013) Molecular Ecology 22:301-313: Excluded bc can't get lat/long coords
	* 
Portnoy et al. (2016) ICES Journal of Marine Science 73:2322-2332: Pulled lat/long coords from paper figure & google maps; excluded Texas bc >3 lat/long span
	* 
Priest, Halford & McIlwain (2012) Ecology & Evolution 2:3195-3213: Excluded separate Guam sites (scored at additional microsat) bc pretty much covered in previous Guam sites; Sfus-98 (Dumaguete), Sfus-8 (Babeldaob, Wreck, Cocos, Governors, Ipan, Pago, Taguisson, Coral Ocean, Laulau, Wing Beach, Peniya, Xavier, Sokeh, Nan Modal, Airport) excluded bc not in HWE; Sfus-9 (Cocos, Governors, Pago, Coral Ocean, Laulau), Sfus-56 (Peniya, Xavier, Sokeh, Nan Modal, Airport) excluded bc monomorphic
	* 
Puebla, Bermingham & McMillan (2012) Molecular Ecology 21:5675-5688: Excluded bc don't know if heterozygosity is He or Ho, and raw data not provided to calculate by hand (or is it?)



05.27.2020

Continuing to pull data for msat papers

Notes:
	* 
Pujolar et al. (2013) Diversity Distrib. 19:1531-1342: For temporal replicates, kept sample with largest sample size
	* 
Purcell & Edmands (2011) Can. J. Fish. Aquat. Sci. 68:1861-1875: Excluded bc can't get exact lat/long coordinates
	* 
Pusack et al. (2014) Molecular Ecology 23:3396-2408: For temporal replicates, kept sample with largest sample size; All sites previously recorded in 2010 in Christie et al. (2010) Molecular Ecology 19: 1042-1057 but doesn't look like included in dataset? so keeping for now
	* 
Qin, Shi & Sun (2013) Genet. Mol. Res. 12:5833-5841: Excluded bc no het stats by pop, and raw data not provided to calculate by hand
	* 
Qin et al. (2014) Genet. Mol. Res. 13:10600-10606: Excluded bc no het stats by pop, and raw data not provided to calculate by hand
	* 
Qiu & Miyamoto (2011) Copeia 2:264-269: T. orientalis: Tth 185 & T. albacares: Tth 114, Ttho-1, Ttho-7 excluded bc not in HWE
	* 
Quintela et al. (2016) Integrative Zoology 11:162-172: Excluded bc species divided & het calculated by color morph -- don't know if these truly represent two different species/populations, and raw data not provided to calculate all together
	* 
Rabone et al. (2015) Mar. Biol. 162:1611-1624: Assume FLAP loci developed in F. lapillum and GCAP loci developed in F. capito; GCAP7 (F. lapilum), GCAP11 (F. lapilum, F. capito) excluded from all sites bc out of HWE



05.28.2020

Continuing to pull data for msat papers

Notes:
	* 
Radhakrishnan et al. (2018) Aquat. Living Resour. 31:22: J10Sc (Veraval), Sa2657 (Veraval), Sa2770 (Veraval, Cochin, Chennai), SCA8 (Veraval, Mangalore, Cochin, Chennai), SCA30 (Veraval), SCA37 (Veraval, Mangalore, Cochin, Chennai), F6Sc (Mangalore), J43Sc (Mangalore), L42Sc (Chennai), SM3 (Chennai)
	* 
Ramírez-Amaro et al. (2018) Mar. Biol. 165:3: Excluded bc can't get accurate lat/long coordinates and some trawls seem to span >3 deg lat/long range
	* 
Ravago-Gotanco et al. (2018) PLoS ONE 13:e0193220: Sfus21 (Hong Kong, Morong), Sfus56 (Hong Kong, San Fernando, Masinloc, Morong), Sfus76 (Hong Kong, San Fernando, Masinloc, Morong), Sfus95 (Hong Kong), Sfus98 (Hong Kong, Patnanungan), Sfus97 (Currimao, San Fernando, Masinloc, Morong, Patnanungan), Sfus113 (Currimao, San Fernando, Bolinao, Morong), Sfus22 (Masinloc), Sfus167 (Masinloc, Morong), Sfus6 (Morong), Sfus9 (Morong) excluded bc out of HWE; Sfus56 & Sfus76 ALWAYS the same -- linked? something weird here?
	* 
Reid et al. (2016) Molecular Phylogenetics and Evolution 103:98-103: Excluded bc het stats not provided by location and can't pull lat/long coordinates
	* 
Reid et al. (2016) Heredity 117:307-315: Portugal, Senegal, Angola excluded bc can't get precise lat/long coordinates; Elf46 (Berg River), Elf37 (Berg River, False Bay, East London, Port Edward, St. Lucia), Elf19 (Berg River), Elf44 (Port Edward) excluded bc not in HWE



05.29.2020

Continuing to pull data for msat papers

Notes:
	* 
Reis-Santos et al. (2018) Scientific Reports 8:10343: Nplaf_27 (Baltic Swedish coast, Baltic Polish coast, Galician shelf), Nplaf_28 (Bay of Biscay, Galician shelf), Nplaf_35 (Galician shelf), Nplaf_14 (Galician shelf), Nplaf_30 (Galician shelf) excluded bc not in HWE; Nplaf_30 (Baltic Swedish coast) excluded bc monomorphic; Portuguese coast excluded bc He not calculated for that site and raw data not provided to calculate by hand
	* 
Ren et al. (2015) Biochemical Systematics and Ecology 61:303-311: Only keeping Sendai Bay small, bc don't know if there is true diff between Sendai Bay large and small, and small appears closer to the actual Sendai Bay than large; Ape302 (Rebun Island, Ishikari Bay, Qingdao), Ape341 (Rebun Island, Ise Bay, Sendai bay small, Hyogo), Ape308 (Qingdao, Cape Soya, Hachinohe) excluded bc not in HWE



05.30.2020

Continuing to pull data for msat papers

Notes:
	* 
Ribout et al. (2018) Biological Journal of the Linnean Society 125:142-151: Ignored geographical coordinates in Table 2 -- don't know what they represent, instead pulling from google maps & paper figure; GYMC24 (Pop1, Pop2, Pop4), GYMC30 (Pop1, Pop3), GYMC33 (Pop1, Pop2, Pop3, Pop4), GYMC39 (Pop1, Pop4), GYMC48 (Pop1, Pop2), GYMC47 (Pop1, Pop2, Pop3, Pop4), GYMC32 (Pop1, Pop2, Pop3, Pop4), GYMC43 (Pop3, Pop4), GYMC19 (Pop4) excluded bc out of HWE, GYMC02 (Pop1, Pop3) excluded bc monomorphic
	* 
Ritchie et al. (2013) Deep-Sea Research II 98:431-437: Pops 1, 6 & 9 excluded bc out of HWE; assumed presented SD not SE as by locus is SD



06.01.2020

Continuing to pull data for msat papers

Notes:
	* 
Rose, Nelson & Mello (2011) Can. J. Fish. Aquat. Sci. 68:152-169: Excluded bc no het stats and raw data not provided to calculate by hand
	* 
Rossi et al. (2019) Hydrobiologia 834:103-117: Excluded bc can't tell if sites out of HWE or not, and raw data not provided to test by hand
	* 
Roy, Hurlburt & Ruzzante (2012) Can. J. Fish. Aquat. Sci. 69:415-419: Excluded bc het stats calculated by STRUCTURAMA cluster, and can't recover lat/long coordinates reliably enough
	* 
Ruggeri et al. (2012) Can. J. Fish. Aquat. Sci. 69: 1698-1709: Keeping temporal replicate with largest sample size (for Chiogga, even though 1978 had larger sample size by 5, kept second largest (2009) instead bc far more recent and comparable to Vieste)
	* 
Ruggeri et al. (2013) Scientia Marina 77:565-574: Chioggia & Vieste excluded bc previously analyzed for most of the same loci in Ruggeri et al. (2012) Can. J. Fish. Aquat. Sci. 69:1698-1709; SARBA07 (San Benedetto del Tronto), SpIII93 (Trani, Northern Catania Gulf), SAR1.5 (Catania), SAR1.12 (Northern Catania Gulf), Sp10 (Northern Catania Gulf) excluded bc not in HWE
	* 
Ruggeri et al. (2016) Marine Ecology 37:518-531: Am-AGAT1, Am-AGAT36, Am-AC34, Am-AAT3, Am-AC1 excluded from all sites bc putatively under selection; Am-AGAT12 (Pula), Am-AC3 (Ancona, Palma de Mallorca), Am-AC16 (Ortona), Am-AGAT12 (Livorno, Palma de Mallorca), Am-AC8 (Palma de Mallorca) excluded bc not in HWE (others also not in HWE, but previously excluded bc under selection)
	* 
Ruggeri et al. (2016) PLoS ONE 11:e0153061: Enja-148 excluded from all pops, EJ27.1 (Montenegro_A, Boka Kotorska, Sperlonga), Enja83 (Montenegro_A), Ee2-407 (Northern Adriatic Sea, Bari_B, Castellammare del Golfo), Ee2-507 (Northern Adriatic Sea), Ee2-508 (Northern Adriatic Sea, Bari_A, Boka Kotorska, Ancona, Pescara_B), Ee10 (Bari_B, Rijeka, Castellammare del Golfo), EJ35 (Bari_B), EJ41.1 (Jablanac) excluded bc out of HWE
	* 
Ruggeri et al. (2016) PLoS ONE 11:e0151507: Kept contemporary (most recent) samples for each sites bc, even though other temporal replicates have slightly larger sample sizes, are a few decades old; Ee2-508m (Vieste) excluded bc out of HWE



06.02.2020

Continuing to pull data for msat papers

Notes:
	* 
Saenz-Agudelo et al. (2012) Molecular Ecology 21:4695-4705: Sites and species previously analyzed in Saenz-Agudelo et al. (2011) Proc. R. Soc. B. 278:2954-2961, but since original paper doesn't report He or raw data, including this paper instead; Kept temporal replicate with largest sample size
	* 
Sahyoun et al. (2016) PLoS ONE 11:e0167441: Included adult samples only; Bld15 (Bari, Punta Penna, Casalabate), Dv11 (Bari, Monopoli, Hotel La Darsena, Torre Guaceto MPA, Casalabate, San Andrea), Om27 (Bari, Hotel La Darsena, San Andrea), Om38 (Bari, Torre Guaceto MPA, Punta Penna, Casalabate, San Andrea), Dv6 (Bari, Monopoli), Dv61 (Bari, Monopoli, Punta Penna, Casalabate, San Andrea), Dv4 (Bari, Hotel La Darsena, Punta Penna, Casalabate), Dv33 (Torre Guaceto MPA, Punta Penna, Casalabate, San Andrea) excluded bc out of HWE
	* 
Sato et al. (2014) Conservation Genet. Resour. 6:685-688: AfAp-02 (A. frenatus) excluded bc out of HWE
	* 
Sato et al. (2017) Ecology and Evolution 7:7859-7871: AfAp-03 (A. frenatus), AfAp-05 (A. perideraion), D103 (A. perideraion) excluded bc out of HWE
	* 
Sato et al. (2018) Journal of Sea Research 142:79-90: Using only three sampled locations for testing microsat markers developed in paper as all other pops are in Tokyo Bay (1 of 3 sites used for testing) and ~on top of each other; Pyo41 (Yamaguchi), Pyo34 (Kagawa), Pyo45 (Kagawa), Pyo37 (Tokyo Bay) excluded bc out of HWE



06.03.2020

Continuing to pull data for msat papers

Notes:
	* 
Schunter et al. (2011) Journal of Experimental Marine Biology and Ecology 401:126-133: D076 (Menorca), SC06 (Menorca, Senegal), RHCA7 (Senegal) excluded bc out of HWE
	* 
Schunter et al. (2011) Molecular Ecology 20:5167-5181: Sc05 (Greece, Otranto, Sardinia, Cap de Creus, Blanes, Menorca, Mallorca, Cabo de Palos, Cabo de Gata, Herradura), Sc08 (Otranto, Cap de Creus), Sc06 (Blanes), Sc07 (Menorca), Sc14 (Herradura) excluded bc out of HWE
	* 
Sebastian et al. (2017) Conserv. Genet. 18:951-964: Sar1-D06 (B) (Mangalore, Calicut), Sar 9 (Calicut), Sar B-A08 (Calicut) excluded bc out of HWE
	* 
Šegvić-Bubić et al. (2011) Aquaculture 318:309-315: Only keeping wild populations; excluded CBW1 bc looks like contaminated with hatchery escapes from local fish farm
	* 
Šegvić-Bubić et al. (2016) Fisheries Research 179:271-279: S. rivoliana: Azores excluded bc can't get accurate lat/long coordinates
	* 
Sekino et al. (2011) Conserv. Genet. 12:139-159: Excluding sites previously analyzed in Ortega-Villaizán Romo et al. (2006) Fish. Sci. 72:556-567; Excluding sites that may have been influenced by released hatchery fish
	* 
Semenova et al. (2015) Polar Biol. 38:951-965: Excluded transcriptome-derived microsats bc (similar to EST-linked ones) may be under selection and in current paper, were significantly lower in het than genome-derived ones; Cha1059 (Chupa Inlet, Kara River) excluded bc not in HWE; Cpa113 (Volonga River, Chesha Bay, Indiga Bay), Cha1020 (Chesha bay) excluded bc not amplified in those pops
	* 
Semenova et al. (2018) Genetika 54:349-360: Using same set of loci as in Semenova et al. (2015) Polar Biol. 38:139-159 (as well as Cpa7 & Cpa 27) for aforementioned reasons; If temporal replicates (same coordinates) then kept either largest sample or most recent collection; Cha1059 (Taui Bay (Yanskii), BER13_2) excluded bc out of HWE; Cpa7 (Central Sea of Okhotsk, BER14_1, BER14_2) excluded bc not amplified in those pops
	* 
Seyoum et al. (2017) Bull. Mar. Sci. 93:691-713: Excluded bc no het stats and raw data not provided to calculate by hand
	* 
Sherman et al. (2017) Front. Mar. Sci. 4:393: Excluded Unknown site bc don't know where samples originated from; Bimini excluded bc only <4 individuals sampled; Est340 (Abaco, Eleuthera, Exuma), Est33a (Abaco, Andros, Eleuthera, Exuma, Great Inagua), Est338 (Abaco, Exuma, Great Inagua), EACD08 (Abaco, Andros, Eleuthera, Exuma, Hail Mary, Great Inagua), Est420 (Abaco, Eleuthera, Hail Mary), Est92 (Abaco), Est360 (Abaco, Andros, Eleuthera, Exuma, Great Inagua), EACD02 (Abaco, Andros, Exuma, Hail Mary), Est262 (Abaco, Andros, Eleuthera, Long Island, Great Inagua), Est290 (Abaco, Exuma), Est376 (Abaco, Exuma, Hail Mary), Est267 (Abaco, Eleuthera, Exuma), Est265 (Abaco, Eleuthera, Exuma, Hail Mary), Est416 (Eleuthera, Great Inagua) excluded bc not in HWE
	* 
Siegle et al. (2013) PLoS ONE 8:e71083: Southeast Alaska, Brooks Bay, Oregon, Washington excluded bc can't get accurate lat/long coordinates; Bowie Seamount, Cape St. James, Triangle excluded bc out of HWE



06.04.2020

Finished pulling data for msat papers (Marial pulling from last 50 papers)

Notes:
	* 
Silva, Horne & Castilho (2014) J. Biogeogr. 41:1171-1182: Included, but didn't test for HWE at pop level, so unsure if in HWE
	* 
Silva et al. (2017) Scientific Reports 7:2983: Excluded E. encrasicolus pops and USA pop bc previously reported in Silva, Horne & Castilho (2014) J. Biogeogr. 41:1171-1182; also as with previous Silva paper, didn't test for HWE at pop level, so including for now but unsure if truly in HWE
	* 
Smirnova, Orlova & Orlov (2019) Journal of Ichthyology 59:555-565: Excluded bc He presented as average only (no raw data), and some loci were EST-derived
	* 
Song et al. (2016) Int. J. Mol. Sci. 17:467: Gma107 (The Yellow Sea, Eastern Hokkaido), Gma108 (The Okhotsk Sea), Gma102 (The Sea of Japan), Gma104 (The Sea of Japan) excluded bc out of HWE
	* 
Sorenson, McDowell & Graves (2011) Conservation Genet. Resour. 3:721-723: Excluded bc can't get lat/long coordinates for sites
	* 
Sorenson et al. (2013) Conservation Genet. Resour. 5:293-297: Excluded bc can't get lat/long coordinates for sites
	* 
Spies (2012) Transactions of the American Fisheries Society 141:1557-1573: If temporal replicates, kept sample with largest sample size; Gma107 (South Atka Island) excluded bc out of HWE
	* 
Steffani et al. (2015) Heredity 115:527-537: Canaries Islands, Mauritania excluded bc can't get accurate lat/long coordinates; Portugal Central, Cadiz excluded bc not in HWE
	* 
Steinberg et al. (2016) Coral Reefs 35:959-970: Included, but didn't test for HWE at pop level, so unsure if in HWE
	* 
Sun, Ge & Cheng (2019) Brazilian Journal of Oceanography 67:e19251: YC79 (Shidao, Dongtou, Haikou, Beibuwan), YC90 (Shidao, Lianyungang, Lvsi, Zhoushan, Xiapu, Haikou, Beibuwan), YC275 (Shidao, Lianyungang, Xiapu, Haikou, Beibuwan), YC339 (Shidao, Lianyungang, Lvsi, Zhoushan, Dongtou, Haikou, Beibuwan), YC353-2 (Shidao, Lvsi, Zhoushan, Dongtou), YC459 (Shidao, Lianyungang, Lvsi, Zhoushan, Dongtou, Xiapu, Haikou, Beibuwan), YC687 (Shidao, Lianyungang, Lvsi, Zhoushan, Dongtou, Xiapu, Haikou, Beibuwan), YC705 (Shidao, Lianyungang, Lvsi, Xiapu, Haikou), YC731 (Shidao, Lianyungang, Lvsi, Zhoushan, Dongtou, Xiapu, Haikou, Beibuwan), YC742 (Shidao, Lianyungang, Lvsi, Xiapu, Haikou, Beibuwan), YC792 (Shidao, Lianyungang, Lvsi, Zhoushan, Xiapu), YC754 (Lianyungang, Lvsi, Zhoushan, Xiapu, Haikou), YC764 (Lianyungang, Lvsi, Zhoushan, Dongtou, Xiapu, Beibuwan), YC353-1 (Dongtou), YC140 (Xiapu, Haikou, Beibuwan) excluded bc out of HWE
	* 
Taillebois et al. (2016) PeerJ 4:e2418: Excluded bc samples pooled across >3 lat/long range to calculate het stats, and raw data not provided to calculate by hand
	* 
Taillebois et al. (2017) 10:978-993: Prd044 (Roebuck Bay), Prd023 (Roebuck Bay, Wadeye, Offshore Darwin), Prd036 (Roebuck Bay), Prd018 (Camden Sound, Maningrida), Prd012 (Wadeye, Maningrida), Prd020 (Peron Islands, Offshore Darwin, Arafura Sea), Prd024 (Peron Islands), Prd042 (Offshore Darwin) excluded bc out of HWE
	* 
Tanner et al. (2014) Estuarine, Coastal and Shelf Science 142:68-75: Mmer-hk29b (Celtic Sea, Armorican Shelf, Galician Shelf, Portugal, Gulf of Cádiz, Balearic Islands, Sardinia), Mmer-hk9b (Galician Shelf, Portugal, Gulf of Cádiz, Balearic Islands, Sardinia), Mmer-hk20b (Gulf of Cádiz) excluded bc out of HWE



Now need to go back and check that not double-recording ones from before, merge files, get proper species names, etc.

06.05.2020

Going back to include any studies with only 1 sample site for msat
	* 
If adding paper, editing msat_screening_notes_2011-2020 as needed for continuity (32 papers downloaded)



Notes:
	* 
An et al. (2011) Genes & Genomics 33:605-611: KTm252 & KTm279 excluded bc not in HWE
	* 
An et al. (2012) Animal Cells and Systems 16:50-56: KEm134, KEm184, & KEm283 excluded bc not in HWE; KEm118, KEm154, KEm219 excluded bc monomorphic
	* 
Bai et al. (2011) Genetics and Molecular Research 10:2065-2068: Excluded bc can't get accurate lat/long coordinates
	* 
Chen & Cheng (2013) Biochemical Systematics and Ecology 47:111-115: P. crocea excluded bc can't get accurate lat/long coordinates; P. polyactis: XHY6, XHY7, XHY18, XHY24, XHY27, XHY29 & XHY33 excluded bc out of HWE
	* 
Deng et al. (2015) Genet. Mol. Res. 14:758-762: Smd1-112 & Smd2-80 excluded bc out of HWE
	* 
Domínguez-Contreras et al. (2018) Mar. Biodiv. 48:2207-2210: KX879732, KX879737, KX879740, KX879742, KX879743, KX879744 & KX879745 excluded bc out of HWE
	* 
Ferrada-Fuentes et al. (2014) Conservation Genet. Resour. 6:833-836: Spfu4, Spfu5, Spfu12, Spfu13, Spfu14, Spfu24, Spfu27, Spfu28, Spfu31, Spfu32 & Spfu39 excluded bc out of HWE
	* 
Henriques, McKeown & Shaw (2012) Conservation Genet. Resour. 4:85-87: Geelb28 & Geelb31 excluded bc out of HWE
	* 
Hu et al. (2013) Genet. Mol. Res. 12:2633-2636: B22, A63, A66 & A119 excluded bc out of HWE
	* 
Kang et al. (2013) Genet. Mol. Res. 12:5485-5493: Eb02-nfrdi, Eb32-nfrdi, Eb42-nfrdi, Eb50-nfrdi & Eb63-nfrdi excluded bc out of HWE; Eb11-nfrdi, Eb40-nfrdi, Eb46-nfrdi, Eb47-nfrdi & Eb50-nfrdi excluded bc out of HWE
	* 
Kitanishi et al. (2014) Conservation Genet. Resour. 6:951-953: Pyo14 (Yamaguchi), Pyo15 (Yamaguchi, Tokyo Bay) & Pyo21 (Yamaguchi) excluded bc out of HWE



06.07.2020

Continuing to go back to include any studies with only 1 sample site for msat

Notes:
	* 
Li et al. Genet. Mol. Res. 12:2751-2754: Two Sf1-27-3 loci listed, so calling second one Sf1-27-3_2 in spreadsheet; Sf1-37-3 & Sf1-47 excluded bc out of HWE
	* 
Li et al. (2014) Conservation Genet. Resour. 6:53-55: Sm3-63 excluded bc out of HWE
	* 
Liu, Gao & Liu (2014) Conservation Genet. Resour. 6:397-399: Lpol07 & Lpol09 excluded bc out of HWE
	* 
Liu et al. (2015) Conservation Genet. Resour. 7:487-488: Excluded bc can't get exact lat/long coordinates
	* 
Lü et al. (2013) Biochemical Systematics and Ecology 51:314-319: 26 FR, 79 FR, 90 FR, B95 FR, B96 FR & 97 FR excluded bc out of HWE
	* 
Ma, Ma & Ma (2011) Genet. Mol. Res. 10:14455-1460: Excluded bc can't get exact lat/long coordinates
	* 
Ma et al. (2011) Fish. Sci. 77:707-711: Excluded bc can't get exact lat/long coordinates
	* 
McWilliam, Minchinton & Ayre (2015) Conservation Genet. Resour. 7:539-541: Tc05 excluded bc not in HWE
	* 
Oury et al. (2019) Molecular Biology Reports 46:1357-1362: E. spinax: Et04 & Et05 excluded bc out of HWE
	* 
Paz-García et al. (2017) Mol. Biol. Rep. 44:251-256: Lupe03, Lupe38 & Lupe53 excluded bc out of HWE



06.08.2020

Finished going back to include any studies with only 1 sample site for msat

Notes:
	* 
Piñeros, Gutiérrez-Rodríguez & Lance (2015) Conservation Genet. Resour. 7:103-105: Absa3, Absa9, Absa12, Absa20, Absa21, Absa22, Absa26, Absa33, Absa42, Absa45 & Absa48 excluded bc out of HWE
	* 
Priest et al. (2015) Mar. Biodiv. 45:861-866: Excluded bc can't get accurate lat/long coordinates
	* 
Yang et al. (2011) Genet. Mol. Res. 10:4006-4011: An13 & An 29 excluded bc out of HWE
	* 
Yin et al. (2012) Russian Journal of Genetics 48:1264-1266: Seb1076, Seb1387, Seb1578, SebJ836, Seb2397 & Seb3856 excluded bc out of HWE



Going back to check Malin's spreadsheets and make sure that not double-recording studies

Working from msat.csv --> this is Malin's compiled csv from all the individual Fish-lat-msat ones plus from his overfished paper
	* 
output from assemble_data_msat.R script

		* 
Work off this script to merge my datasets into new updated one --> just going through original assemble script and editing to merge mine with old assembled one

			* 
Calling new script assemble_data_msat_2.R 



06.09.2020

Continuing to clean up and assemble msat datasets
	* 
When checking lat/lon against SRDBmatch lat/lon, found a few errors --> corrected in msat_assembled_2.R script:

		* 
Shubina et al. 2004 Env Biol Fish: One of the Russian sites (Olyuter) had - long when should be + (in Eastern hemisphere, according to google maps) (169.2293)
		* 
Schmidt 2005 dissertation: MAG96 (Irminger Sea), one of the loci had wrong coordinates SEB46: 60.44, -28.2
	* 
When checking CrossSpp, found some missing data --> corrected in msat:

		* 
Appleyard, Williams & Ward. 2004. CCAMLR Science 11:21-32: To2 & To5 found to be same-species (patagonian toothfish) dinucleotide markers CrossSpp 0, Repeat 2
		* 
Diaz-Jaimes & Uribe-Alcocer 2006 Fish Sci: T-35, assume is cross-species bc all others are (had to changed those) (ALL CrossSpp = 1)

