# Notes on msat and mtDNA papers entered
From Diana Li, Kieryn Graham, Ash Bhattacharya, Sarah Picon, Stan Piotrowski, Chloe Lewis, and Michael Weiss

1/31/2012
Diana Li

	Fishery lat msats 000; All References 3

	Canino et al 2005 Mol Ecol Notes
		First paper; starting to get the hang of it; straightforward primer note

	Carmen & Ablan 2006
		Confused about "country" and "sites" columns since the msats data table is only reported wrt individual msats across all locations
		Emailed Malin; since the locations are vague/not within 2º of each other, not using paper

2/1/2012
Diana Li

	Adams et al 2006
		Had to go online (http://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2006.02859.x/suppinfo) to find msat data in supplementary Table 1; for "repeat" supplementary data 2 seems to indicate that they only used tri repeats - since they used msats from Adams et al 2005, I looked there and confirmed that these are all tri; the data presented is kind of tricky to understand... might want to go over with Malin later to make sure

2/2/2012
Diana Li

	Almuly et al 2005 Fisheries Science
		This doesn't seem wild; family structure seems known (mentions "full-sib"); partially domesticated stoke from Atlit fish farm; decided to not use paper

	An et al 2005 Mol Ecol Notes
		"Screened in 30 captive samples" - does not seem wild; decided to not use paper; plus location was very vague

	An et al 2006 Mol Ecol Notes
		same problem as An et al 2005; decided not to use

	Reid et al 2005 J Fish Bio
		samples used were "wild fish retained as broodstock for aquaculture;" does not seem wild; decided to not use

	Antoro et al 2006 Marine Biotech *
		looks like I'll need to enter allele frequencies - there are no He data per ind msat per location; they do have data with He and Hese per population (I guess averaged across all 4 msat loci); emailed Malin
		trouble with Table 1: seems that their coordinates are Long/Lat, not Lat/Long as the column header indicates
		The msats used are from two other papers (Nugroho et al 1998; Taniguchi & Nugroho 2000 in japanese) - I can't seem to view/find the second, but the title of Nugroho et al implies that the repeats are all 2; I don't know what the repeats from the msats in T & N 2000 are; Nugroho et al states that the msat was orginally developed in a different Epinephelus species - still not quite sure about the repeat, but I think their data state it's 2

2/3/2012
Diana Li

	Babbucci et al 2006 Mol Ecol Notes
		Primer note; easy to understand

	Bekkevold et al 2005 Evolution *
		Took a while to find the supplementary materials online with the He data; uses msats from McPherson et al 2001b (Clupea harengus) and Olsen et al. 2002 (Clupea pallasi); for some reason, my off-campus connection is not working anymore so I need to still get the Olsen et al paper to fill in the repeat data; lots of data to enter but straightforward;

2/5/2012
Diana Li

	Bekkevold et al 2005 (cont'd)
		Back on campus; retrieved Olsen et al paper; re-enter He data for non-HWE loci since it could be a statistical mistake and authors chose to keep these loci

	Berrebi et al 2006 Mol Ecol Notes
		3 Gobi species; 2 are marine, 1 is amphidromous; use the 2 marine species regardless of marine or non-marine msats
		the data are very spotty (lack of amplification in certain loci for certain species) and a lot of the loci show significant departure from Fis = 0; 
		exclude loci with Fis significantly different from zero; include loci that were amplified in some species but not for others

	Blanco et al 2006 Mol Ecol Notes
		2 species - black and white anglers; based on black angler msats; 2 loci deviate significantly from HWE (not used); 1 only amplified in black angler 
		straightforward primer note; location - along Cantabrian coast from Cantabrian coasts from Santander, Cantabria to La Coruña, Galicia (those two places have latitudes within .5 degrees of each other as does the coastline between them so I just used the latitude of Santander, Cantabria)

2/6/2012
Diana Li

	Buonaccorsi et al. 2005 Conservation Genetics
		use msat markers previously used in a different species of rockfish (copper) from Buonaccorsi et al. 2002, but I can't seem to find info on repeats in the 2002 paper; the 2002 paper gives no indication of supplementary material or specific indication of the repeats in the msats they used; currently not using Buonaccorsi et al 2005 unless I find the repeat data
		already entered by Malin

	Carlsson et al. 2006 ICES J Marine Science
		unclear where they gathered data in years 1999 & 2002; the map shows a range from 62º-57º latitude and 30º-13º longitude; currently not using paper

	Carreras-Carbonell et al 2006 Mol Ecol Notes
		These authors published multiple papers in Mol Ecol Notes in 2006, but the one that is on the "entered papers" spreadsheet is different from this one
		trying to find supplementary material... this is proving to be quite tricky; man, I just can't find the supplementary materials online - the url isn't working and it brings me to the wiley webpage with the article but no indication of supplementary materials; 

	Carreras-Carbonell et al 2006 Mol Ecol Notes 
		primer note; Malin already entered this one

	Casas et al. 2005 Mol Ecol Notes
		did not use; derived from ESTs

	Castilho & Ciftci 2005 J Fish Bio
		did not provide individual msat data or allele frequencies per site, only an average from each site without standard error; can use - just leave Hese blank
		cited msats from Castilho & McAndrew 1998b, Ciftci et al. 2002; Garcia de Leon et al 1997

	Castillo et al. 2005 ICES J Marine Science
		already entered by Malin

	Chen et al 2005 Mol Ecol Notes
		EST based msats; did not use

	Chevolot et al. 2006 J Sea Research
		report that 2 locations showed significant departure from HWE but they don't say that the whole loci are not in HWE... emailed Malin
		based on msats from Chevolot et al 2005 Mol Ecol Notes; so far, I entered all of them, but we'll see based on what Malin says - authors don't exclude loci so it's safe to include them, but if the locus is non HWE in every (or almost every) location sampled, then check back with Malin to make sure

2/7/2012
Diana Li

	Chevolot et al. 2006 Mol Ecol
		similar to other Chevolot paper from JSR - uses same msats on same species in different locations
		one msat shows as monomorphic in 2 locations but is polymorphic in all the rest. the authors don't mention anything about it though; keeping the locus, but omitting it for that location entry
		Three locations were from fish markets, so the authors did not know the long and lat of the original catch site; omitting these entries since we don't know where, exactly, the fish market samples came from
		Some locations are significantly not in HWE, but the authors did not exclude those loci in their paper, so I've kept the data

	Curley & Gillings 2006 Mol Ecol Notes
		does not specify location - only: "four coastal reefs in New South Wales, Australia." The span of coast is too big to stay within 2º; not using paper

	D'Amato 2006 Marine Biotech
		already entered by Malin

	Daly-Engel et al. 2006 Env Bio of Fishes
		There seem to be no data on expected and observed heterozygosity; the authors aimed to find out the number of sires for a litter; not using paper

	Diaz-Jaimes & Uribe-Alcocer 2006 Fisheries Sci
		Already entered by Malin

	Dudgeon et al. 2006 Mol Ecol Notes
		Samples from captives in aquaria; not using paper

	Farias et al. 2006 Mol Ecol Notes
		Already entered by Malin

	Feng et al. 2005 Mol Ecol Notes
		Location vague - "South China Sea near Indonesia;" not using paper

2/8/2012
Diana Li

	Franch et al. 2006 Genetics
		Deals with linkage map; decided to not use

	Galarza et al. 2006 Mol Ecol Notes
		looks good; authors note that 2 loci are not in HWE due to Fis values significantly different from zero, so excluding loci to be safe (based on decision made for Berrebi et al 2006)

	Galetti et al 2006 Genetica
		Seems like a review that doesn't have any actual msat or He data; it does, however, talk about papers that did do msat work with He data; I think this qualifies as a meta-analysis?

	Gessner et al. 2006 J Appl Ichth
		Seems also like a review that doesn't have any actual msat or He data; it does talk about a paper that did do msat work and report He data; may come back to later


	Gharrett et al 2006 Transactions of American Fisheries Society
		Does not have He data; only has significance of test of homogeneity between allel freqs reported in their paper against previous papers; no indications of supplementary data; also, it's a study to determine siblings, so not using this paper

	Gilbert-Horvath et al. 2006 Mol Ecol
		already entered by Malin

	Gomez-Uchida & Banks 2005 CJFAS
		already entered by Malin

	Gomez-Uchida & Banks 2006 J of Heredity
		uses same sample as the G-U &  2005 CJFAS paper, only this one reports He by year of birth, not location or msat; not using paper b/c relevant data referenced is already entered

	Graves & McDowell 2006 Bull Mar Sci
		uses cross species msats from Buonaccorsi & Graves 2001; not sure whether to use this paper... does provide data based on msat per location, but only with all the sampling years averaged - the data with the He per year per msat per location is "not shown;" unfortunately, it does not include any Hese value; there is one location of on msat entry (averaged across all sampling years) that is not in HWE but the authors decide to keep it anyway for all subsequent tests; also, the msat data per location is grouped in broad locations - 3 of the 4 groupings are all of the same place over many years, but one has the Dominican Republic and Cumaná, Venezuela - are we allowed to omit this location entry and keep the others? yes
		only used Cape May, NJ data and Santos, Brazil data since the others were too vague with respect to location

2/13/2012
Diana Li

Had to clean up data spreadsheet, applying new rules about the Collection Year - pick with highest sample size; if a tie, pick most recent

	Guo et al 2005 Mol Ecol Notes
		all but 1 locus deviated significantly from HWE without the Bonferroni correction - omitting these loci

	Jørgenson et al 2005 ICES J Mar Sci
		already entered by Malin

	Hauser et al. 2006 Intl Pacific Halibut Commission
		authors say that tests for genetic differentiation between sample locations were similar regardless of whether the 3 non-HWE loci were included - do not explicitly say that they will not consider those 3 loci for analysis, so I'm keeping them
		uses cross species primers from McGowan & Reith 1999, Coughlan et al. 2000 - even after looking up McGowan & Reith 1999, I still can't find the repeat info for locus HhiA44 - used genbank: tandem repeat = di
		From Coughlan et al 2000 - repeat motif for Hhi52 is (CTGTAACATACAACAA)3 (CTGTAACATACAA)2 is a minisatellite - do not use locus http://onlinelibrary.wiley.com/doi/10.1046/j.1365-294x.2000.00915-3.x/full

2/14/2012
Diana Li

	Jakobsdóttir et al 2006 Mol Ecol Notes
		Locations "Iceland" and "Baltic Sea" along with a sample of 1957 archived otolith samples from "Iceland" - locations too vague; did not use paper

	Jean et al 2006 Mol Ecol Notes
		The authors don't state in their text that loci are out of HWE but the table indicates that all but one locus significantly deviate from HWE via p-values (the authors do not list Fis values)
		currently leaving them....out to be safe?

	Jones & Barber 2005 Mol Ecol Notes
		Reports data from juveniles and larvae - asked Malin if we use juvenile/larval data; also confused about "cohort"
		From Malin: Like you thought, a cohort means that they're all of similar age. As long as the larvae and juveniles are from different locations, we can use both. Otherwise, use the juveniles since they're closer to adults.
		Only using juveniles since location is the same; leaving out two loci since authors state that they significantly deviate from HWE

	Jørgenson et al 2005 Mol Ecol Notes 
		Already entered by Malin

	Karlsson & Mork 2005 ICES J Mar Sci
		looked at 2 loci - one of which was significantly out of HWE and the other which was almost out of HWE (only keeping the latter); latter msat from 
		Brooker et al. 1994 Canadian J of Fisheries & Aqu Sci
		the He data is averaged over 15 years... with no Hese value

	Kokita et al. 2006 Mol Ecol Notes
		straightforward to understand 4 of 5 loci in HWE so entered those 4

2/16/2012
Diana Li

	LeClair et al. 2006 Trans of the Am Fish Soc
		provide only allele frequencies, but straightforward

2/17/2012
Diana Li

	Lemos et al 2006 Aquaculture R
		5 sites - includes 1 average between sites that are too far and 1 that is too vague; entering only 3 of the 5 sites
		They only have the average of all msats per location with no Hese values - apparently there is supplementary material but the site that is indicated does not contain this paper's supp materials....
		skip it (for now) - we may come back for data later if needed

	Li et al. 2006 Mol Ecol Notes
		already entered by Malin

	Lo et al. 2006 Mol Ecol Notes
		already entered by Malin

	Louie & Bardeleben 2006 Mol Ecol Notes
		Only report as one sample from "Pacific coast of North America" - too spread out for our criteria
		not using

	Mackiewicz et al. 2005 Mar Bio
		straightforward; authors say that they omitted 1 of the 4 loci, so I did too

	Miller-Sims et al. 2005 Mol Ecol Notes
		straightforward; good

	Miller et al 2005 CJFAS
		already entered by Malin

	Neilson et al 2005 CJFAS
		they do not seem to have He data or allele frequencies (no indication of supplementary materials or that data in the appendix); also locations vague
		not using paper

2/18/2012
Diana Li

	Ortega-Villaizán et al. 2006 Fisheries Sci
		straightforward; looked up Ortega- Villaizán et al. 2003 for the repeat info on the msats
		authors state that there is a small (slightly significant) deviation from HWE in some loci in some locations, but they still keep the loci and they state that there was no locus that deviated in all the locations

	Ovenden et al. 2006 Mol Ecol Notes
		locations too vague; not using paper

	Papetti et al. 2006 Mol Ecol Notes
		no location listed; not using paper

	Park et al. 2006 Mol Ecol Notes
		location too broad (coastal waters of Korea); not using paper

	Piñera et al. 2006 Mol Ecol Notes
		location too vague (Mediterranean coast of Spain); not using paper

	Portnoy et al. 2006 Mol Ecol Notes
		does not seem to list any location; not using paper

	Poulsen et al. 2006 Mol Ecol
		includes data from archived otoliths and a recent sampling for each of 2 locations; since locations are the same, i'm choosing the one with a larger sample size;
		He data in appendix; uses msats (cross sp and non-cross sp) from 4 papers - Brooker et al 1994; Miller et al 2000; O'Reilly et al 2000; Hutchinson et al 2001

2/19/2012
Diana Li

	Purcell et al 2006 Proc R Soc B
		info in supplementary material
		msats from Williams 2004a,b  Mol Ecol Notes 
		authors explicitly state that 1 locus from the 1st species and 3 loci from the 2nd species are out of HWE in a significant number of locations, so i've dropped these loci; there are other loci that deviate in a few locations, but i've kept them since the authors do not state this about these other loci

	Ramirez et al 2006 Mol Ecol Notes
		already entered by Malin

2/20/2012
Diana Li

	Renshaw et al 2006 Mol Ecol Notes
		straightforward primer note

	Rogers et al. 2006 Mar Bio
		1 locus was monomorphic in all but 1 population, so not using; 1 locus was out of HWE in 2 pops and the authors cite previous studies in which this locus was also out of HWE, so not including
		2 locations had various degrees that were > ±3º, so not using those 2 locations

	Ryan et al 2005 ICESJMS
		already entered by Malin

	Saillant et al 2006 ICESJMS
		already entered by Malin

	Saillant & Gold Fish Bull
		already entered by Malin

	Sanchez et al 2006 Mol Ecol Notes
		vague location (Japan); not using paper

	Seibert & Ruzzante
		already entered by Malin

2/21/2012
Diana Li

	Senger et al 2006 Genomics
		deals with making linkage maps; has no He or allele freq data; not using paper

	Small et al 2005 TAFS
		already entered by Malin

	Smith-Keune et al 2006 Mol Ecol Notes
		location is in too broad of a lat/long range (center of Great Barrier Reef); not using paper

	Spies et al 2005 Mol Ecol Notes
		straightforward

2/22/2012
Diana Li

	Thompson et al 2006 Mol Ecol Notes
		locations: Moorea, Raiatea, Huahine, Bora Bora, and Rangiroa... all averaged into "French Polynesia" - degrees lat/long are greater than 3º apart between these 5 locations; not using paper

	Van Houdt et al. 2006 Mol Ecol Notes
		straightforward

	van Herwerden et al 2006 Fish Res
		already entered by Malin

	Wesmajervi et al 2006 Aquaculture REs
		location too vague - "coastline of Norway;" not using paper

	Wilson 2006a Mol Ecol
		not sure if we keep all loci... "global probability tests failed to reject null hypothesis of HWE" - we do keep these loci
		cites msats in Wilson 2006b Mol Ecol

	Wilson 2006b Mol Ecol
		lumps all 4 locations along Pacific Coast of US together - no Hese values either; not using unless allelic richness means something?? - it does not matter to us

	Xia et al. 2006 Mol Ecol Notes
		"Southeastern China" too broad for location
		not using paper

	Yang et al 2006 Mol Ecol Notes
		location too vague (South China sea near indonesia); not using paper

	Yokoyama et al 2006 Mol Ecol Notes
		already entered by Malin

	Yoshida et al 2005 Mol Ecol Notes
		already entered by Malin

	Zhang et al 2006 Mol Ecol Notes
		already entered by Malin

	Zhu et al 2005 Mol Ecol Notes
		already entered by Malin

	Nielsen et al 2006 Mol Ecol
		no He data and no specific allele frequency data; not using paper

	Jue 2006 J Fish Bio
		already entered by Malin

2/28/2012
Diana Li

	Hardie et al 2006 CJFAS
		averaged He without Hese - plus, with cross species msat among non-cross species ones; 
		for averaged He: leave MarkerName blank; average cross-species (if there is a mix between cross and non-cross species); average the repeat #
		we can include it even if there is no Hese

3/1/2012
Diana Li

	entered Castilho & Ciftci

	ALL REFS 1

	Ahonen et al 2009 Mol Ecol
		locations too vague - no coords; did not use

	An et al 2010 Anim Cell Syst
		already entered by Malin

	André et al. 2011 Heredity
		supplementary material online!
		Cpa112 is an "outlier" so i'll not be using it
		two years of samples taken for each location; chose the ones with highest sample size

	Arnott et al. 2010 Mar Ecol Prog Series
		does not talk about heterozygosity; did not use

	Bai et al. 2011 Gen and Mol Research
		location too vague "northern coastal waters of Yellow Sea"
		not use

3/2/2012
Diana Li

	Beldade et al. 2009 Cons Gen Resources
		straightforward

3/3/2012
Diana Li

	Boissin et al. 2011 Bio J of the Linnean Society
		confusing "resampled" avg He values (with Hese) as opposed to "non-resampled" average He values without Hese - ask malin?
		nevermind, can use allele frequencies from appendix!
		confused about which loci are out of HWE - I can't seem to find an explicit statement saying if and what loci are; their Fis values are confusing in Table 4
		use only the last 2 loci since the first two are non HWE - use allele frequencies

	Borrell et al 2011 J Fish Bio
		says that samples were taken from "marine farm" - this does not sound wild; not using paper

3/4/2012
Diana Li

	Bradbury et al 2009 J Fish Bio
		authors basically include all loci, since the removal of certain ones that seem suspicious have no significant impact on further analyses
		authors study 3 species; there are samples of each species taken from each of 3 places, but the He values are only considered per species, not location; however the locations are within 2 degrees of each other, so I'm going to use the per-species data
		of the 3 species, 1 is anadromous - don't use!
		...oops. forgot to check if the data was already entered... but it was. I guess I'll keep it for error checking?

	Bradman et al. 2010 Cons Gen Resources
		of the two locations, only 1 was specific enough - using Fremantle data only
		the authors say that "5 of the 22 loci failed to meet HWE" but they don't say which five... I can't figure out which five they are...
		Malin says drop SwoA8, SwoB6, SwoB108, and SwoB112 from Coral sea, and drop SwoA115, SwoB6, SwoB9, and SwoC4 from Fremantle.

	Breton et al 2010 Cons Gen Resources
		Locations too vague; not using paper

3/5/2012
Diana Li

	Broderick et al 2011 J Fish Bio
		can't find supplementary data - otherwise, the only data listed lumps all the locations into 1 entry per msat, but the locations are too far for our use. 
		leave for now until supplementary data is found

	Buonaccorsi et al 2011 Mol Ecol
		this is very confusing
		come back to later
		turns out that the first part is what we want while the second part of their Table 2 is under selection so we don't use that; authors state that after major corrections that none of the "neutral" loci are out of HWE
	
3/6/2012
Diana Li

	Burford et al. 2011 Marine Bio
		deals with hybrids; not use

	Buston et al. 2009 Mol Ecol
		straightforward; sites msats from Fauvelot et al. 2009 Mol Ecol Resources; supplementary material for He online

	Byrne & Avise 2009 Marine Biology
		straightforward

	Canales-Aguirre et al. 2010 Cons Gen
		Already entered by Malin

	Canales-Aguirre et al. 2010 Fisheries Res
		locations specifically listed, but range too broad; not use

	Canino et al. 2010 Marine & Coastal Fisheries: Dynamics, Management, and Ecosystem Science
		3 microsatellites were said to have heterozygote deficiency so they were removed for subsequent analysis; i did not enter those loci; of the loci entered 1 is from a "personal communication" so I can't quite find the repeat info - other papers that use this microsat also say it's a personal communication, but it's clearly a cross-species for this study

3/8/2012
Diana Li

	Canino et al. 2010 Mol Ecol
		authors state that 2 loci, 1 each in a different location deviate from hwe after multiple tests, but USE ALL OF THEM
		typically we don't remove just 1 locus from a certain location - if we remove it, we remove the locus for ALL sites
		msats from Canino et al 2005, O'Reilly et al 2000, and Hutchinson et al 2001 (latter two from Poulsen et al 2006)

	Mtg notes 3/8/2012

	if two loci are significantly linked (with significant linkage disequilibrium - ask about it!)
	"haplotypes" are used more often for mitochondrial DNA
	at one site is fine for out of HWE

3/11/2012
Diana Li

	Still working on Boissin et al. 
		Ran into a bit of trouble trying to get latitude coordinates for the locations
		unsure about lagoons and estuaries - checked: lagoons and estuaries ok

3/13/2012
Diana Li

	Revisiting Buonaccorsi et al. 2011
		A bit of discrepancy of exact location - inds from either San Miguel Island or from other places in southern California bight - which is still within 3º range, so i'm using coordinates for San Miguel Island
		Msat repeat data from Berntson et al 2007

	Cárdenas et al. 2009 Fisheries Research
		already entered by Malin

	Carlsson et al. 2011 Deep-Sea Research Pt I: Oceanographic Research Papers
		some msats out of HWE but authors state that the data overall are not affected by these deviations, so I'm keeping all loci
		wait actually, there are "n/a" He values for a locus - turns out it's monomorphic - ask malin! - malin says to keep locus and enter 0 for the N/A He values
		msat repeat data from http://tomato.biol.trinity.edu/
	
	Mtg notes on 3/15/2012

	If info not present, leave blank!

3/15/2012
Diana Li

	Boissin et al. 2011 Bio J of the Linnean Society
		done!
		found the lat degrees for the locations with Malin's help
		msats cited in Pampoulie et al. 2004

3/16/2012
Diana Li

	Carson et al. 2009 North Am J of Fisheries Mgmt
		authors state that 3 of the 16 loci deviated from HWE in two sites or more (of the four sites) - a few deviated in every site, so I'm not including these 3 loci either
		I can't access the article with the msat repeat data - it seems like princeton is not subscribed to it? Saillant et al. 2004 http://goms.disl.org/tocpages/june2004vol22no1.htm
	
	Ceballos et al. 2011 Cons Gen Resources
		straightforward	

3/17/2012
Diana Li

	Cha et al. 2009 Cons Gen Resources
		already entered by Malin

	Chabot & Nigenda 2011 Cons Gen Resources
		sample locations averaged (which is not within our constraints.... by quite a lot)
		not using paper

	Chang et al. 2009 Genes & Genomics
		already entered by Malin

	Chapman et al. 2009 Mol Ecol
		does not report He values in study

	Chen et al. 2009 Cons Gen
		fat greenling
		of 10 loci - 4 out of HWE and 3 have null alleles (1 of the 3 is the same as 1 of the 4) - only entering the remaining 4 loci

	Chen et al. 2009 Cons Gen
		yellow croaker
		already entered by Malin

	Chen et al. 2009 Cons Gen
		black sea bass
		does not sound wild: "selectively collected from Huanghai Aquaculture Ltd., Haiyang, China" 
		not using paper

	Christie et al. 2010 Mol Ecol
		only have average He per location
		have data for adults and larval recruits, but since they're of the same locations, i'm just going to enter the adult data
		adult data has 2 locations each with 2 of the 7 msats out of hwe, but in no other locations are there non-hwe values
		there are 2 locations with small linkage diseq, but not after correction - so i'm going to keep all locations

3/18/2012
Diana Li

	Coscia & Mariani 2011 Bio J of Linnean Society
		averaged He without Hese; no loci out of HWE; msats from Chistiakov et al 2004, Ciftci et al 2002, de Leon et al 1995, Tsigenopoulos et al 2003 - all di

	Cuveliers et al. 2011 Mol Ecol
		uses archived otolith samples; reports otolith samples with regard to sampling year groups - of the 5 sampling groups, 1 is not within 3º so i'm not including that data; also have data from 3 contemporary locations; including those
		authors exclude Sos(AC)30, 40, 45

3/20/2012
Diana Li

	Finished up Cuveliers et al. 2011
		msats from Iyengar et al 2000, Garoia et al 2006, Porta & Alvarez 2004, Funes et al 2004

	D'Anatro et al. 2011 Env Bio of Fishes
		already entered by Malin

	Daly-Engel et al. 2010 Marine Ecol Prog Series
		He sampling is > ± 3º
		not using

	Danancher & Garcia-Vazquez 2009 Mar Bio
		already entered by Malin

3/22/2012
Diana Li

	DiBattista & Feldheim 2011 Cons Gen Res
		tested 2 species - for one species, cor16 out of HWE (dropped); for other species cor14 & cor21 out of HWE (dropped)
		repeat for cor16 = "complex"?? - looked up in genbank - seems like repeat of 4
	
	DiBattista et al. 2011 Cons Gen Res
		msats from a hybrid
		not using

	Ding et al. 2009 Mol Ecol Res
		authors state that 4 loci deviate significantly from HWE - excluding those

3/25/2012
Diana Li

	Dong et al. 2011 Genetics and Mol Research
		location too vague - "along the coast of the East China Sea" - not using

	Earl et al. 2010 Cons Gen
		data separated according to management units - check if the regions that they encompass are within 3º
		of six locations, I can't find the area for one of them - temporarily leaving it out - come back to this
		took average of boundary coordinates - they were within 3º

3/26/2012
Diana Li

	continue Earl et al. 2010
		authors state that any departure from HWE or LD after corrections is presumed minimal; including all loci
		samples collected sometime between 1990 and 2001...? average the years

	Eble et al. 2011 Marine Ecol Progress Series
		only have He by location (averaged msat)
		includes larval and adult, but I don't see the data separating them
		says that there is significant LD in one of the sample locations, but they say it's more likely to be due to sampling error instead of actual LD so i'm keeping all the data
		Standard deviation given - I entered that in the Hese column

	Ferrara et al. 2010 Collect. Vol. Sci. Pap. ICCAT 
		uses ESTs; not using paper	

3/27/2012
Diana Li

	Fitzpatrick et al. 2011 Mol Ecol
		authors state which loci are neutral - i'll be using only those
		supplementary material has He data - only lists data for sample sizes > 5, so excludes 3 locations
		need to find coords of locations to see if they're within 3º
		np in He slot in their data means no polymorphism - so I put 0 for our He column
		samples collected between 1997 & 2008? - average the years
	

3/28/2012
Diana Li

	Therkildsen et al. 2010 CJFAS
		have data from archived otoliths and a recent catch - otolith and recent location is basically the same (include for now?)
		data for p value of deviation from HWE all seem to be insignificant, so including all loci

3/29/2012
Diana Li

	Gaggiotti et al. 2009 Evolution
		does not give data on He (even in supplementary material) - only Fst
		not using

4/1/2012
Diana Li

	Gaither et al. 2010 Cons Gen Res
		2 locations: Hawaii and New Caledonia - Hawaii is not specific enough; NC falls within 3º
		authors name a locus from each location that is out of HWE, so excluding the one from NC; also, one msat failed to amplify for NC, so not including that
	
	García et al. 2010 Cons Gen Res
		potentially too vague - Upper Gulf of California, Mexico (the whole gulf extends more than 3º, but the "upper" part is about 3º, but they're not clear where, exactly, they are considering "upper")
		not using for now

	Gardner et al. 2011 Cons Gen Res
		17 of 22 locations are in HWE; 2 of 22 msats are linked; data arranged weirdly... but useable; 2 monomorphic; the first 12 were tested in 4 different locations, but these locations were within 3º of each other in the same archipelago so only average He is listed but still per individual msat... there is no error, but a the range of the values is given (not that useful)

	Garvin et al. 2011 Trans of the Am Fisheries Soc
		there seems to be no He data and no suppl material; not using paper

	Glover et al. 2011 ICES J Marine Sci
		authors name 3 loci to have contributed to most of the HWE deviations; they still include 2 sets of data - avg He with all 8 locai and avg He with on 5 loci (the non-HWE 3 removed), so i'll use the 5-averaged set

	Gold et al. 2010 Fish Res
		already entered by Malin

	Gold et al. 2011 N Am J of Fisheries Mgmt
		fish houses??

4/2/2012
Diana Li

	Golestani et al. 2010 Zoology in the Middle East
		apparently I only have the first page of this... find the rest

	Gong et al. 2009 Cons Gen
		straightforward

	Gold et al. 2011 N Am J of Fisheries Mgmt (again)
		Malin says to use the paper - they appear wild
		only Florida Keys samples had HWE and LD trouble, but since these loci are otherwise fine in other locations, we should keep them? double check this - because just one from one location, keep it

4/4/2012
Diana Li

	González-Wangüemert et al. 2010 J Exp Marine Bio and Eco
		already entered by Malin

	González-Wangüemert et al. 2011 Marine Bio Res
		the only msat-related data is taken from González-Wangüemert et al. 2010 which was already entered by Malin anway; not using

	Gordeeva et al. 2010 Hydrobiologia
		the 2 species in question are anadromous; not using paper

	Griffiths et al. 2010 Proc R. Soc. B.
		He only according to "clades" - North and South, but the clades encompass a 10º span; not using paper

	Gu et al. 2009 Cons Gen
		hatchery samples; not use

4/5/2012
Diana Li

	Gubili et al. 2009 Mar Bio
		they use 7 msats - 5 from Pardini et al 2000; 1 from Schrey & Heist 2002; and 1 that they call "a new species specific primer" which they developed but don't give much info about (has no accession number either)
		done but without that new locus - INCLUDE NEW LOCUS (it's non cssp; leave rpt blank)

4/6/2012
Diana Li

	Guinand et al. 2011 Mar Env Res
		nursery samples; not using

	Galarza et al 2009 CJFAS
		1 msat deviated from HWE in every location, so i left that out; the authors also mention that they excluded 2 locations in further analyses b/c the samples sizes were smaller than the rest of the sample sizes (15 instead of 50ish) but I kept these since they still fit our criteria

4/7/2012
Diana Li

	Finished up Galarza et al. 2009

	Helyar et al. 2011 Cons Gen Res
		2 msats are non HWE in both pops - exclude
		2 pairs in LD but not in both pops - ask? - Malin says to leave them in

	Helyar et al. 2010 Cons Gen Res
		authors say that 1 pair of msats showed significant LD - not clear if for both pops, but will exclude
		1 msat is non HWE in only 1 of 2 locations - ask? - Malin says to leave it out

	Hogan et al. 2010 Marine Ecol Prog Series
		there is a massive supplementary data table speckled with loci that are out of HWE in various populations.... this will take some time to sort through
		there is also a table indicating # pops that are in LD (range from 0-8 of 58 pops) - not too sure what to do...? maybe include them all since it's not that much?
	GATA40 has 15 out of HWE aftcor
	AAT40 has 0 out of HWE after corrections! AAC44 has 20 out AAC33 has 11 out
	TG16 has 10 out of HWE aftcor
	GGA7 only has 2 out of HWE after corrections TG53 has 5 out TG13 has 6 out GT10 has 4 out
		including the ones that are less than 10 out for now, but will check with Malin
	>>DON'T USE THIS PAPER

4/8/2012
Diana Li

	Hone et al. 2010 Cons Gen Res
		5 out of HWE; of those 5, 3 may be Linked - excluding these 6

4/9/2012
Diana Li

	Hu et al. 2011 Cons Gen Res
		anadromous species; not using paper

	Jakobsdóttir et al. 2011 Evolutionary Appl
		they test 6 neutral msats and 1 under selection, but they only provide an average He which includes all 7, which we don't want since that 1 is under selection; plus, they don't have it divided by location, only by sample year or cohort
		not using

4/10/2012
Diana Li

	Järvi -Laturi et al. 2011 J of Zoology
		does not talk about He data; not using

	Johnson et al. 2009 Copeia
		they do not separate He data by location - locations are greater than ±3º from each other; not use

	Jones et al. 2010 Coral Reefs
		they don't supply a table of He data nor does there seem to be any supplemental data; they only have a graph, which is not very helpful...
		not using

	Karl et al. 2011 Cons Gen
		amphidromous; not using

	Karlsson et al. 2009 Mar Bio
		already entered by Malin

	Kasapidis et al. 2009 Mol Ecol Res
		locations too vague - North Atlantic & Western Mediterranean; not use

	Kim et al. 2010 Fisheries Sci
		already entered by Malin

	Knutsen et al. 2009 Cons Gen Res
		tested on 2 species from same location; for the original species, 3 loci out of HWE but they say that 2 are barely significant (these are the uncorrected values) while the third showed clear significant deficiency of heterozygotes - so I'll exclude that one

4/12/2012
Diana Li

	Knutsen et al. 2009 Mol Ecol
		does not talk about He; not using paper

	Kovach et al. 2010 Mol Ecol Prog Series
		only has He data averaged over 11 msats + 5 Single-nucleotide polymorphism markers; not using paper

	Landínez-garcía et al. 2009 Ciencias Marinas
		asked about artisanal fisheries - almost always wild; 

	Larmuseau et al. 2010 Heredity
		average He reported per location; only includes 8 of 9 msats due to avoid genotyping error

4/14/2012
Diana Li

	Revisted Landínez-garcía et al. 2009 Ciencias Marinas
		cannot find repeat data - can't find Bagley & Geller 1998; otherwise, fine.

	Larmuseau et al. 2009
		does not have He data; not using paper

	Larson et al. 2011 Cons Gen
		it is unclear from which samples the He data are from - or the sample size; or HWE departure significance; 
		looks like these used all the sharks? and leave out <.005 HWE ones

	Larsson et al. 2010 Heredity
		Ht = gene diversity = heterozygosity

4/15/2012
Diana Li

	Lawton et al. 2011 Mol Ecol
		pretty straightforward; no loci excluded; just locus Tri22 (GAGAGGA)8 repeat?? they say it's a msat and that it's "tandem"
		ASK - malin says to call it a di, since that's the dominant repeat within what they report as the repeat

	Lawton et al. 2010 Cons Gen Res
		not sure of non-HWE loci v. loci suggestive of null alleles/homozygote excess - malin says to exclude non-HWE but keep the null allele ones

	Le Boucher et al. 2011 Aquaculture Res
		uses broodstock for family analysis; not using paper

	Leray et al. 2010 Evolution
		some msats deviate from HWE in different pop groups, but not uniformly throughout groups - so i'll keep them

	Lin et al. 2011 Gen & Mol Res
		straightforward

4/16/2012
Diana Li

	Liu & Avise 2011 Marine Bio
		straightforward He data but the species lives in both salt and freshwater.... they live on the CA coast but also in the wetlands - wanders into freshwater, but doesn't live there

	Liu et al. 2011 Acta Oceanologica Sinica
		authors point out the loci that are out of HWE per population, but don't explicitly exclude them, so I'll keep them too; there are 5 sampling pops, one of which is a hatchery, so I'll leave that one out

	López et al. 2010 Cons Gen
		straightforward

	Lynch et al. 2010 Fish Bull
		already entered by Malin

	Ma et al. 2011 Gen & Mol Res
		state that 3 of 10 loci are out of HWE but could be due to scoring error - keep

	Ma et al. 2011 Fish Sci
		4 msats deviate from HWE but NOT due to scoring error, they say - exclude 

4/18/2012
Diana Li

Revisiting:

	Gubili et al.
		I thought I entered this, but apparently not? Well, I know what to enter now anyway, so I'm redoing it into "Lat msats 000"
		including Ccar6.27x; excluding Ccar6 b/c of out hwe; including others

	Hogan et al.
		made sure it wasn't entered

	Larson et al. 2011
		done

	Larsson et al. 2010
		done

	Lawton et al. 2011
		done

	Lawton et al. 2010
		done

4/21/2012
Diana Li

	Ma & Chen 2011 Cons Gen
		He was not calculated due to the limited number of individuals tested; not using paper

	Ma et al. 2011 Env Bio of Fishes
		does not give location; not using paper

4/24/2012
Diana Li

	Machado-Schiaffino et al. 2011 Bio Cons
		authors exclude certain loci, but their He data only reports the average of ALL of them - aksed Malin

	Maggio et al. 2009 ICES JMS
		some loci deviate from hwe while others don't for different sample sites
		COME BACK

	Marshall et al. 2011 CJFAS
		does not provide He data; not use

	Martinez & Aguilar 2011 Cons Gen Res
		uses EST datasets; not use paper

	Massault et al. 2010 Anim Gen
		uses farmed fish for samples; not use paper

	McCraney et al. 2010 Mol Ecol
		straightforward except one msat has a repeat of (ATCTCT)5.... - 6

4/25/2012
Diana Li

	McCusker & Bentzen 2010 Mol Ecol (1)
		straightforward

	McCukser & Bentzen 2010 Mol Ecol (2)
		meta-analysis

	McCusker & Bentzen 2011 Mar Bio
		only has He data averaged over all msats per location but authors end up excluding 7 of the 16 msats - don't use
	
4/29/2012
Diana Li

	Qiu & Miyamoto 2011 Copeia
		pretty straightforward; authors state which loci they are out of hwe after corrections, and i drop these
		they use allozymes instead of allele number and repeat to analyze data? is this okay? yes
		left out cspp data for Ttho-4

	Mobley et al. 2011 J Fish Bio
		some review; does not have He data; not use

	Mobley et al. 2010 J Biogeography
		straightforward; none excluded; missing 1 rpt value

5/2/2012
Diana Li

	Canales-Aguirre et al. 2010 Gayana
		no He data; not use

	Dammannagoda et al. 2011 CJFAS
		straightforward

5/8/2012
Diana Li

	Muths & Bourjea 2011 Cons Gen Res
		authors say they excluded 3 but didn't specify - so i assumed they meant the 3 loci that departed significantly from hwe in BOTH populations

	Na-Nakorn et al. 2010 Aquaculture
		captive stocks used as sample; not using paper

	Nance et al. 2011 PLoS ONE 
		assuming "fish camp" is wild; also excluding Manta sample since it is too vague to be ±3º
		the repeat info for the same-species loci (Nance et al 2009) is not listed.... and on genbank, it just says "tandem" - I will figure it out later - apparently this means 2


5/9/2012
Diana Li

	Nielsen et al. 2006 Mol Ecol
		not sure about hitch-hiking? the authors do state that one locus is studied as it may be under selection (Gno 132)
		heterozygosity is apparently ln(RH)?? dont use

	Nielsen et al. 2010 Cons Gen 
		weird average-y stuff. 
		include paper
	
5/10/2012
Diana Li

	Nordeide et al. 2011 Mar Ecol Prog Series
		meta-analysis

	Ovenden et al. 2011 Mar Biol
		this is confusing; COME BACK

5/13/2012
Diana Li

	Palof et al. 2011 Mar Biol
		He uses Nei's correction; otherwise, straightforward

	Pampoulie et al. 2011 Trans of the Am Fish Society
		they only have avg He per location with the loci and the suspected nonneutral marker separate; 
		they state that the nonneutral was indeed nonneutral but it did not deviate from HWE; however 3 samples did deviate and Gmo34 was also found to deviate from neutral expectations
		is neutrality the same as HWE??
		don't use this paper

	Pampoulie et al. 2009 ICES
		already entered by Malin

	Papetti et al. 2011 Cons Gen Res
		straightforward; excluded 3 loci that they said were not in HWE

	Parati et al. 2011 Cons Gen Res
		broodstock; not using paper

	Paterson et al. 2010 Cons Gen Res
		location too vague; not using paper

	Chopelet et al. 2009 Mol Ecol Res
		straightforward; excluded 1 pair LD and one of those was non HWE anyway

	Pini et al. 2011 Coral Reefs
		gene diversity; I can't find the sample sizes... COME BACK
		enter and leave it blank

5/14/2012
Diana Li

	Pinsky et al. 2010 Evolution
		keep all loci; need repeat data for the 2 from this paper

	Pita et al. 2010 Thalassas
		no He data; not use

	Pita et al. 2011 Continental Shelf Research
		excluded 3 that they state deviated significantly in every sample

	Plank et al. 2010 J Fish Bio
		need repeat info - genbank only says "tandem" - implies 2

	Portnoy et al. 2009 Cons Gen
		locations too far; only had averaged data per mast over all locations; not using paper

	Pumitinsee et al. 2009 Aquaculture Research
		already entered by Malin

	Purcell et al. 2009 Mol Ecol Res
		locations too vague; not using paper

5/15/2012
Diana Li

	Putte et al. 2009 Polar Biol
		authors state that 2 of the species tested were out of hwe on the whole, so i have excluded these two species

4/29 - 9.5h
5/13 - 12h
5/27 - 20.5h

5/19/2012
Diana Li

	Ravago-Gotanco et al. 2010 Cons Gen Res
		straightforward

	Ren et al. 2009 Cons Gen Res
		Yellow Sea too vague; not using paper

	Renshaw et al. 2009 N Am J of Aquaculture
		already entered by Malin

	Renshaw et al. 2010 Cons Gen 
		already entered by Malin

	Reza et al. 2011 Fisheries Sci
		two sampling locations (G and S) for species T.r. - TrG in hwe at 1 msat; TrS in hwe at a different msat; collected a different species from a different location, but NONE in hwe.... what to do? not include paper

	Riccioni et al. 2010 PNAS USA
		already entered by Malin

	Ring et al. 2009 Mol Ecol Res
		1 out of hwe; excluded that; otherwise, straightforward

	Roberts & Ayre 2010 Mar Ecol Prog Series
		This is confusing; COME BACK LATER

	Roy et al. 2010 Con Gen Res
		Q solution - don't worry; go ahead.

	Sabadin et al. 2010 Revista de Biologia Marina y Oceanografia
		does not report He data; not use paper

	Saenz-Agudelo et al. 2011 PRSB
		I can't seem to access the data supplement http://rspb.royalsocietypublishing.org/content/suppl/2011/02/07/rspb.2010.2780.DC1.html which may have the He data - the paper doesn't list any - email the peeps

	Saillant et al. 2010 ICES J Mar Sci
		authors state that they use all loci, so i'm not excluding anything either;

5/20/2012
Diana Li

	Finished up Saillant et al. 2010

	Sala-Bozano et al. 2009 Mol Ecol
		already entered by Malin

	Sala-Bozano et al. 2009 Cons Gen
		already entered by Malin

	Salas et al. 2010 Mar Biol
		authors exclude 3 loci from further analysis, so I excluded them too

	Sbrocco & Barber 2011 Cons Gen Res
		straightforward

	Schmidt et al. 2010 Endangered Sp Res
		does not give He data; not using paper

	Schneider et al. 2009 Cons Gen
		location vague; not using paper

5/22/2012
Diana Li

	Schunter et al. 2011 J of Exp Mar Biol & Ecol
		hmm... try again later - use supp mat

5/23/2012
Diana Li

	Utter & Seeb 2010 Mol Ecol
		does not have He data; not using paper

	Sekino et al. 2011 Cons Gen
		it seems that they use all the loci even though some loci have a sample that deviates from hwe - i'll include all of them?
		also, they compare with a previous study (Ortega-Villaizan et al 2006) and there is mention of captive fish? but it is unclear whether these samples have captured fish? perhaps i'll leave this one for later - it seems okay though
		enter only the approved locations but all loci okay

	Sellas et al. 2011 Cons Gen Res
		straightforward

	Sha et al. 2009 Cons Gen
		authors state that 1 locus out of hwe, so I excluded it; otherwise straightforward

	Shubina et al. 2009 Mol Biol
		already entered by Malin

	Shulzitski et al. 2009 Fish Bull
		already entered by Malin

	Smith & Campana 2010 CJFAS
		only has allele freq averaged per location and they assume HWE but don't actually say if it is? - look at original papers instead

	Sorenson et al. 2011 Cons Gen Res
		straightforward

	Stroganov et al. 2010 Moscow U Biol Sci Mull
		they don't seem to provide the full He data for all the loci - just the max and min; not using paper

	Stroganov et al. 2009 Russian J of Mar Biol
		it's a bit unclear, but it looks like they include the allele frequencies per location of only 1 locus but the averaged He per location as well? also, nothing is out of HWE - I will just include the averaged He

	Sun et al. 2011 Gen & Mol Res
		amphidromous; not using paper

	Hess et al. 2011 CJFAS
		not including collections that have a sample size of 1; report avg He per location; none out of HWE

	Teske et al. 2010 Mar Biol
		only report avg He, but 1 locus is out of hwe in 2 locations & LD in 3 total (2 pairs) - keep all

	Teske et al. 2009 Mol Ecol Notes
		straightforward; excluded 3 due to non-HWE and LD

	Tripp-Valdez et al. 2010 Fish Res
		there are some loci that are out of hwe in certain samples, but they say that they did not find a clear tendency of non-hwe in specific loci or location, so there's a chance that those deviations are due to stochastic events in allelic frequency distributions - does this mean that they are NOT due to selection? should i keep them all? yes

	Tzeng et al. 2009 ICES J Mar Sci
		already entered by Malin

5/24/2012
Diana Li

	Umino et al. 2009 Fish Sci
		authors state that all loci in HWE

	Underwood 2009 Cons Gen Res
		straightforward; 3 of 10 loci excluded

	Underwood 2010 Cons Gen Res
		straightforward; 1 of 10 loci excluded

	Cushman et al. 2009 CJFAS
		already entered by Malin

	Veríssimo et al. 2010 Mol Ecol
		LD in 1 pair of loci in one pop, but no global LD shown, so including all loci - they only have avg He anyway; only 2 locations are within 3º, so using those

	Veríssimo et al. 2011 ICES J Mar Sci
		does not provide He by location; not using paper

	Veríssimo et al. 2011 J of Heredity
		has "gene diversity" which is heterozygosity

	Veríssimo et al. 2010 Cons Gen Res
		location not within 3º

	Villegas-Sánchez et al. 2010 Cora Reefs
		does not offer He data or allele freq data; not using paper

	Viñas et al. 2011 Reviews in Fish Biol and Fisheries
		locations grouped together, but are not within 3º; not using paper

5/25/2012
Diana Li

	Vliet et al. 2009 Cons Gen Res
		does not provide location for the samples..... not using paper

	Wallace & Tringali 2010 J Fish Bio
		does not have He per location; not using paper

	Wang et al. 2010 Gen and Mol Res
		excluded 3; assume that "fishing ground" is wild

	Was et al. 2009 Mar Biol
		already entered by Malin

	White et al. 2011 Heredity
		I'm not sure, but it seems like the avg He data in table 1 already excludes the 4 loci that the authors decide should be out due to nulls, so I'll just enter them
	Mtg

5/27/2012
Kieryn Graham

		Durand, JD; Collet, A; Chow, S; Guinand, B; Borsa, P	MARINE BIOLOGY	10.1007/s00227-005-1564-2	2005	
			Already entered

		Selkoe, KA; Vogel, A; Aftab, BT	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00859.x	2005	
			Already entered

		Mariani, S; Hutchinson, WF; Hatfield, EMC; Ruzzante, DE; Simmonds, EJ; Dahlgren, TG; Andre, C; Brigham, J; Torstensen, E; Carvalho, GR	MARINE ECOLOGY-PROGRESS SERIES	10.3354/meps303245	2005	
			Already entered

		Narum, Shawn R.; Buonaccorsi, Vincent P.; Kimbrell, Carol A.; Vetter, Russell D.	Copeia	10.1643/CG-02-061R2	2004	
			Already entered

		Pereyra, RT; Saillant, E; Pruett, CL; Rexroad, CE; Rocha-Olivares, A; Gold, AR	MOLECULAR ECOLOGY NOTES	10.1111/j.1474-8286.2004.00801.x	2004	
			Already entered

		Matala, AP; Gray, AK; Gharrett, AJ; Love, MS	NORTH AMERICAN JOURNAL OF FISHERIES MANAGEMENT	10.1577/M03-061.1	2004	
			Already entered

		De Innocentiis, S; Lesti, A; Livi, S; Rossi, AR; Crosetti, D; Sola, L	FISHERIES SCIENCE	10.1111/j.1444-2906.2004.00879.x	2004	
			Already entered

		Funes, V; Zuasti, E; Catanese, G; Infante, C; Manchado, M	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00690.x	2004	
			Already entered

		Buonaccorsi, VP; Westerman, M; Stannard, J; Kimbrell, C; Lynn, E; Vetter, RD	MARINE BIOLOGY	10.1007/s00227-004-1362-2	2004	
			Already entered

		Crivello, JF; Danila, DJ; Lorda, E; Keser, M; Roseman, EF	JOURNAL OF FISH BIOLOGY	10.1111/j.1095-8649.2004.00424.x	2004	
			Already entered

		O'Reilly, PT; Canino, MF; Bailey, KM; Bentzen, P	MOLECULAR ECOLOGY	10.1111/j.1365-294X.2004.02214.x	2004	
			Already entered

		Watts, PC; Nash, RDM; Kemp, SJ	JOURNAL OF SEA RESEARCH	10.1016/j.seares.2003.09.003	2004	
			Already entered

		Zatcoff, MS; Ball, AO; Sedberry, GR	MARINE BIOLOGY	10.1007/s00227-003-1236-z	2004	
			Already entered

		Matala, AP; Gray, AK; Heifetz, J; Gharrett, AJ	ENVIRONMENTAL BIOLOGY OF FISHES	10.1023/B:EBFI.0000022876.71349.be	2004	
			Already entered

		Clark, TB; Ma, L; Saillant, E; Gold, JR	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2004.00572.x	2004	
			Already entered

		Shubina, EA; Mel'nikova, MN; Glubokov, AI; Mednikov, BM	ENVIRONMENTAL BIOLOGY OF FISHES	10.1023/B:EBFI.0000022883.13864.ff	2004	
			Already entered

		Alarcon, JA; Magoulas, A; Georgakopoulos, T; Zouros, E; Alvarez, MC	AQUACULTURE	10.1016/S0044-8486(03)00434-4	2004	
			Already entered

		Rocha-Olivares, A; Leal-Navarro, RA; Kimbrell, C; Lynn, EA; Vetter, RD	SCIENTIA MARINA		2003	
			Already entered

		Ramsak, A; Garoia, F; Guarniero, I; Mannini, P; Tinti, F	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00509.x	2003	
			Already entered

		Romo, M. Ortega-Villaizan; Nakajima, M.; Taniguchi, N.	Molecular Ecology Notes		2003	
			Already entered

		McCartney, MA; Acevedo, J; Heredia, C; Rico, C; Quenoville, B; Bermingham, E; McMillan, WO	MOLECULAR ECOLOGY	10.1046/j.1365-294X.2003.01946.x	2003	
			Already entered

		Hutchinson, WF; van Oosterhout, C; Rogers, SI; Carvalho, GR	PROCEEDINGS OF THE ROYAL SOCIETY OF LONDON SERIES B-BIOLOGICAL SCIENCES	10.1098/rspb.2003.2493	2003	
			Already entered

		Jeong, DS; Umino, T; Kuroda, K; Hayashi, M; Nakagawa, H; Kang, JC; Morishima, K; Arai, K	FISHERIES SCIENCE	10.1046/j.1444-2906.2003.00705.x	2003	
			Already entered

		Gomez-Uchida, D; Hoffman, EA; Ardren, WR; Banks, MA	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00458.x	2003	
			Already entered

		Burridge, CP; Smolenski, AJ	NEW ZEALAND JOURNAL OF MARINE AND FRESHWATER RESEARCH		2003	
			Already entered

		Rhodes, KL; Lewis, RI; Chapman, RW; Sadovy, Y	MARINE BIOLOGY	10.1007/s00227-002-1002-7	2003	
			Already entered

		Bernal-Ramirez, JH; Adcock, GJ; Hauser, L; Carvalho, GR; Smith, PJ	MARINE BIOLOGY	10.1007/s00227-002-0972-9	2003	
			Already entered

		Ovenden, JR; Street, R	MARINE AND FRESHWATER RESEARCH	10.1071/MF02142	2003	
			Already entered

		Chapman, RW; Ball, AO; Mash, LR	MARINE BIOTECHNOLOGY	10.1007/s10126-002-0038-5	2002	
			Already entered

		Turner, TF; Wares, JP; Gold, JR	GENETICS		2002	
			Already entered

		Hauser, L; Adcock, GJ; Smith, PJ; Ramirez, JHB; Carvalho, GR	PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA	10.1073/pnas.172242899	2002	
			Already entered

		Bouza, C; Presa, P; Castro, J; Sanchez, L; Martinez, P	CANADIAN JOURNAL OF FISHERIES AND AQUATIC SCIENCES	10.1139/F02-114	2002	
			Already entered

		Yu, HT; Lee, YJ; Huang, SW; Chiu, TS	MARINE BIOTECHNOLOGY	10.1007/s10126-002-0035-8	2002	
			Already entered

		Kim, WJ; Kim, KK; Lee, JH; Park, DW; Park, JY; Lee, JY	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00524.x	2003	
			Already entered

		Garoia, F; Guarniero, I; Tinti, F	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00438.x	2003	
			Already entered

		Hoffman, EA; Arguello, JR; Kolm, N; Berglund, A; Jones, AG	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00691.x	2004	
			Entered. Excluded Pka13

		Aboim, MA; Rogers, AD; Menezes, GM; Maggioni, R; Pearson, CVM	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00337.x	2003	
			Excluded Hd008, Hd044, Hd095 because out of HWE. Called Hd063 as a 2-repeat (not 9, though ambiguous). (MLP 2012-06-13)

		King, JR; Withler, RE	MOLECULAR ECOLOGY	10.1111/j.1365-294X.2005.02438.x	2005	
			Could not find repeats; used eggs from clutches to reconstruct parental genotypes at 5 loci

		Williams, DA; Purcell, J; Hughes, CR; Cowen, RK	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00506.x	2003	
			Entered. Excluded loci out of HWE.

		Miller-Sims, V; Delaney, M; Atema, J; Kingsford, MJ; Gerlach, G	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2005.00950.x	2005	
			Entered

		Keeney, DB; Heupel, MR; Hueter, RE; Heist, EJ	MOLECULAR ECOLOGY	10.1111/j.1365-294X.2005.02549.x	2005	
			Entered

		Stockley, B; Menezes, G; Pinho, MR; Rogers, AD	MARINE BIOLOGY	10.1007/s00227-004-1479-3	2005	
			Entered

		Heist, EJ; Jenkot, JL; Keeney, DB; Lane, RL; Moyer, GR; Reading, BJ; Smith, NL	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00348.x	2003	
			Entered

		Garoia, F; Guarniero, I; Piccinetti, C; Tinti, F	MARINE BIOTECHNOLOGY	10.1007/s10126-004-3045-x	2004	
			Entered.

		Carreras-Carbonell, J; MacPherson, E; Pascual, M	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00688.x	2004	
			Entered.

		Olsen, JB; Merkouris, SE; Seeb, JE	FISHERY BULLETIN		2002	
			Entered.

		Miller-Sims, V; Atema, S; Kingsford, MJ; Gerlach, G	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00689.x	2004	
			Entered. 3 markers excluded because sig heterozygote deficit after correction for multiple comparisons, suggesting presence of null alleles

		Kumagai, K; Barinova, AA; Nakajima, M; Taniguchi, N	MARINE BIOTECHNOLOGY	10.1007/s10126-003-0006-8	2004	
			Entered. Chinese samples excluded because "cultured in Japan."

		Knutsen, H; Jorde, PE; Andre, C; Stenseth, NC	MOLECULAR ECOLOGY	10.1046/j.1365-294X.2003.01750.x	2003	
			Entered. Some loci at some sites out of HWE, but don't seem to find consistent differences. (MLP 2012-06-13)

		Saillant, Eric; Gold, John R.	Proceedings of the Gulf and Caribbean Fisheries Institute		2004	
			Entered. Included juveniles. Larger population is bycatch sample.

		Withler, RE; King, JR; Marliave, JB; Beaith, B; Li, SR; Supernaulta, KJ; Miller, KM	ENVIRONMENTAL BIOLOGY OF FISHES	10.1023/B:EBFI.0000022898.49007.bc	2004	
			Entered. Msat info was via pers. Comm. Hence repeats not known.

		Shaw, PW; Arkhipkin, AI; Al-Khairulla, H	MOLECULAR ECOLOGY	10.1111/j.1365-294X.2004.02327.x	2004	
			Entered. No loci remained out of HWE after correction.

		Carlsson, J; McDowell, JR; Diaz-Jaimes, P; Carlsson, JEL; Boles, SB; Gold, JR; Graves, JE	MOLECULAR ECOLOGY	10.1111/j.1365-294X.2004.02336.x	2004	
			Entered. Repeat numbers for some markers not found. One locus excluded because affected by null alleles.

		Knutsen, H; Andre, C; Jorde, PE; Skogen, MD; Thuroczy, E; Stenseth, NC	PROCEEDINGS OF THE ROYAL SOCIETY OF LONDON SERIES B-BIOLOGICAL SCIENCES	10.1098/rspb.2004.2721	2004	
			Excluded. He's not given.

		Hoarau, G; Piquet, AMT; van der Veer, HW; Rijnsdorp, AD; Stam, W; Olsen, JL	JOURNAL OF SEA RESEARCH	10.1016/j.seares.2003.12.002	2004	
			Excluded. He's not given.

		McPherson, AA; O'Rielly, PT; Taggart, CT	TRANSACTIONS OF THE AMERICAN FISHERIES SOCIETY	10.1577/02-106	2004	
			Excluded - Only Ho's seem to be provided.

		McCauley, L; Goecker, C; Parker, P; Rudolph, T; Goetz, F; Gerlach, G	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00724.x	2004	
			Entered. Excluded 4 loci out of HWE.

		Williams, DA; Purcell, J; Cowen, RK; Hughes, CR	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00713.x	2004	
			Entered. 5 loci excluded because out of HWE.

		Appleyard, SA; Williams, R; Ward, RD	CCAMLR SCIENCE		2004	
			Entered. Excluded De2 and De9

		Sekino, M; Hara, M; Taniguchi, N	AQUACULTURE	10.1016/S0044-8486(01)00885-7	2002	
			Excluded - "hatchery" strains

		Sotka, EE; Hempelmann, JA; Biermann, CH	MARINE BIOTECHNOLOGY	10.1007/s10126-004-0437-x	2005	
			Excluded - Allele frequencies/Hes not given

		Asahida, T; Gray, AK; Gharrett, AJ	ENVIRONMENTAL BIOLOGY OF FISHES	10.1023/B:EBFI.0000022884.30278.32	2004	
			Excluded - Allele frequencies/Hes not given

		Saillant, E; Mousseau, TA; Gold, JR	TRANSACTIONS OF THE AMERICAN FISHERIES SOCIETY	10.1577/T03-006	2003	
			Excluded - already entered

		Chapman, DD; Prodohl, PA; Gelsleichter, J; Manire, CA; Shivji, MS	MOLECULAR ECOLOGY	10.1111/j.1365-294X.2004.02178.x	2004	
			Excluded - averaged over sampling locations > 3 degrees

		Garcia Alcazar, A.; Chereguini, O.; Porta, J.M.; Porta, J.; Alvarez Herrero, M.C.; Abellan Martinez, E.	Boletin Instituto Espanol de Oceanografia		2005	
			Excluded - cannot access English version

		Bahri-Sfar, L; Lemaire, C; Chatain, B; Divanach, P; Ben Hassine, OK; Bonhomme, F	AQUATIC LIVING RESOURCES	10.1051/alr:2005007	2005	
			Excluded - French version only

		Nugrohoa, E; Taniguchi, N	FISHERIES SCIENCE	10.1111/j.1444-2906.2004.00851.x	2004	
			Excluded - hatchery offspring only

		Feldheim, KA; Gruber, SH; Ashley, MV	EVOLUTION	10.1554/04-023	2004	
			Excluded - He's and allele freqs not given

		Lemaire, C; Versini, JJ; Bonhomme, F	JOURNAL OF EVOLUTIONARY BIOLOGY	10.1111/j.1420-9101.2004.00828.x	2005	
			Excluded - mitochondrial DNA

		Jones, AG; Moore, GI; Kvarnemo, C; Walker, D; Avise, JC	PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA	10.1073/pnas.1131969100	2003	
			Excluded - no msats or allele freqs given

		Clemmesen, C; Buhler, V; Carvalho, G; Case, R; Evans, G; Hauser, L; Hutchinson, WF; Kjesbu, OS; Mempel, H; Moksness, E; Otteraa, H; Paulsen, H; Thorsen, A; Svaasand, T	JOURNAL OF FISH BIOLOGY	10.1046/j.0022-1112.2003.00060.x	2003	
			Excluded - no msats or allele freqs given

		Zhang, JB; Huang, LM; Huo, HQ	JOURNAL OF EXPERIMENTAL MARINE BIOLOGY AND ECOLOGY	10.1016/S0022-0981(03)00341-1	2004	
			Excluded - not msats; not wild

		Liu, YG; Chen, SL; Li, BF	AQUACULTURE	10.1016/j.aquaculture.2004.10.024	2005	
			Excluded - not wild populations

		Castro, J; Bouza, C; Sanchez, L; Cal, RM; Piferrer, F; Martinez, P	MARINE BIOTECHNOLOGY	10.1007/s10126-003-0004-x	2003	
			Excluded - not wild; gynogenetic

		McPherson, AA; Stephenson, RL; Taggart, CT	MARINE ECOLOGY-PROGRESS SERIES	10.3354/meps247303	2003	
			Excluded - only Ho's given

		Tonteri, A; Titov, S; Veselov, A; Zubchenko, A; Koskinen, MT; Lesbarreres, D; Kaluzhin, S; Bakhmet, I; Lumme, J; Primmer, CR	ANNALES ZOOLOGICI FENNICI		2005	
			Excluded - salmo salar 

		Quenouille, B; Bouchenak-Khelladi, Y; Hervet, C; Planes, S	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00646.x	2004	
			Excluded - sampling location not clear - lists "New Guinea" only

		Tsigenopoulos, CS; Hellemans, B; Chistiakov, DA; Libertini, A; Kotoulas, G; Volckaert, F	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00447.x	2003	
			Excluded - sampling location not given

		Ciftci, Y; Castilho, R; McAndrew, BJ	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2002.00327.x	2002	
			Excluded - sampling location not given

		Gharrett, AJ; Matala, AP; Peterson, EL; Gray, AK; Li, ZZ; Heifetz, J	TRANSACTIONS OF THE AMERICAN FISHERIES SOCIETY	10.1577/T04-055.1	2005	
			Excluded - sampling locations more than 3 degrees apart

		Narum, SR; Buonaccorsi, VP; Kimbrell, CA; Vetter, RD	COPEIA		2004	
			Excluded - sampling locations more than 3 degrees apart

		Miller, KM; Kaukinen, KH; Laberee, K; Supernault, KJ	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00792.x	2004	
			Excluded - sea urchin

		Chistiakov, DA; Hellemans, B; Tsigenopoulos, CS; Law, AS; Bartley, N; Bertotto, D; Libertini, A; Kotoulas, G; Haley, CS; Volckaert, FAM	ANIMAL GENETICS	10.1046/j.1365-2052.2003.01076.x	2004	
			Excluded - specific sampling location not given

		Reeb, CA; Arcangeli, L; Block, BA	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00385.x	2003	
			Excluded - specific sampling location not given

		O'Malley, KG; Abbey, CA; Ross, K; Gold, JR	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00379.x	2003	
			Excluded - specific sampling location not given

		Talley Farnham, Tiffany C.; Stequert, Bernard; Alvarado Bremer, Jaime R.	International Commission for the Conservation of Atlantic Tunas Collective Volume of Scientific Papers		2004	
			Excluded - specific sampling locations not given

		Aurelle, D; Guillemaud, T; Afonso, P; Morato, T; Wirtz, P; Santos, RS; Cancela, ML	COMPTES RENDUS BIOLOGIES	10.1016/j.crvi.2003.08.001	2003	
			Excluded - specific sampling locations not given

		Schrey, AW; Heist, EJ	CANADIAN JOURNAL OF FISHERIES AND AQUATIC SCIENCES	10.1139/F03-064	2003	
			Excluded - specific sampling locations not given

		Streelman, JT; Puchulutegui, C; Bass, AL; Thys, T; Dewar, H; Karl, SA	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00413.x	2003	
			Excluded - specific sampling locations not given

		Gallini, A; Zane, L; Bisol, PM	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2004.00817.x	2005	
			Excluded because it is estuarine (spends most of its lifecycle in the Venice Lagoon)

		Chevolot, M; Reusch, TBH; Boele-Bos, S; Stam, WT; Olsen, JL	MOLECULAR ECOLOGY NOTES	10.1111/j.1471-8286.2005.00951.x	2005	
			Excluded North Sea sample because >3 degrees

		Nielsen, EE; Nielsen, PH; Meldrup, D; Hansen, MM	MOLECULAR ECOLOGY	10.1046/j.1365-294X.2004.02097.x	2004	
			Entered. Excluded hybrid sampling locations.

		Pampoulie, C; Gysels, ES; Maes, GE; Hellemans, B; Leentjes, V; Jones, AG; Volckaert, FAM	HEREDITY	10.1038/sj.hdy.6800438	2004	
			Entered. Could not find some sampling locations.

		Castillo, AGF; Martinez, JL; Garcia-Vazquez, E	MARINE BIOTECHNOLOGY	10.1007/s10126-004-3027-z	2004	
			Excluded - cannot access full article

		Hoffman, EA; Kolm, N; Berglund, A; Arguello, JR; Jones, AG	MOLECULAR ECOLOGY	10.1111/j.1365-294X.2005.02538.x	2005	
			One locus (pka16 for population R) excluded because sig dep from HWE

		Cimmaruta, R; Scialanca, F; Luccioli, F; Nascetti, G	OCEANOLOGICA ACTA	10.1016/S0399-1784(02)01234-3	2003	
			Excluded because a brackish-water species (MLP 2012-06-13)

		Appleyard, SA; Ward, RD; Williams, R	ANTARCTIC SCIENCE	10.1017/S0954102002000238	2002	
			Excluded because same samples as Appleyard, Williams & Ward. 2004. CCAMLR Science 11, 21-32. (MLP 2012-06-13)

		Nielsen, EE; Hansen, MM; Ruzzante, DE; Meldrup, D; Gronkjaer, P	MOLECULAR ECOLOGY	10.1046/j.1365-294X.2003.01819.x	2003	
			Entered. Excluded hybrid zones.

		Furukawa, S; Takeshima, H; Otaka, T; Mitsuboshi, T; Shirasu, K; Ikeda, D; Kaneko, G; Nishida, M; Watabe, S	FISHERIES SCIENCE	10.1111/j.1444-2906.2004.00849.x	2004	
			Excluded - cannot access full article.

		Saillant, E.; Cizdziel, K.; O'Malley, K. G.; Turner, T. F.; Pruett, C. L.; Gold, J. R.	Gulf of Mexico Science		2004	
			Excluded - cannot find article

		Lage, C; Kuhn, K; Kornfield, I	FISHERY BULLETIN		2004	
			Excluded - only Ho's given

		Watts, PC; Veltsos, P; Soffa, BJ; Gill, AB; Kemp, SJ	MOLECULAR ECOLOGY NOTES	10.1046/j.1471-8286.2003.00579.x	2004	
			Excluded - cannot access full article

		Sekino, Masashi	Bulletin of Fisheries Research Agency		2004	
			Excluded - cannot find article

		Heist, Edward J.	Biology of sharks and their relatives. [CRC Marine Biology Series.].		2004	
			Excluded - cannot find article

		van Herwerden, L; Benzie, J; Davies, C	JOURNAL OF FISH BIOLOGY	10.1046/j.1095-8649.2003.00075.x	2003	
			Some sampling locations excluded - could not find exact GPS coordinates

		Kotoulas, G.; Mejuto, J.; Tserpes, G.; Garcia-Cortes, B.; Peristeraki, P.; de la Serna, J.M.; Magoulas	International Commission for the Conservation of Atlantic Tunas Collective Volume of Scientific Papers		2003	
			Excluded - cannot find article

5/29/2012
Diana Li

	White et al. 2010 Mol Ecol
		they say that markers may not be neutral - LnRH values and Fisher's - data analyzed in 2 groups: Crup7 alone and all others without Crup7

	White et al. 2010 Cons Gen Res
		does not specify where in the Mid-Atlantic Ridge samples were from; not using paper

	White et al. 2010 Cons Gen Res	
		does not specify where in the Mid-Atlantic Ridge samples were from; not using paper

	White et al. 2009 Cons Gen
		does not specify where in the Mid-Atlantic Ridge samples were from; not using paper

	Pakaki et al. 2009 Mol Ecol Res
		already entered by Malin

	Xu et al. 2009 Cons Gen
		3 excluded; straightforward

	Xu et al. 2011 Gen & Mol Res
		locations too broad; not using paper

	Zarraonaindia et al. 2009 ICES J of Mar Sci
		already entered by Malin

	Zhang et al. 2010 Cons Gen Res
		already entered by Malin

	Zhao et al. 2011 Biochem Systematics and Ecol
		just check - we're allowed to use Nei's gene diversity, yes? that's just like regular gene diversity?
		does not talk about hwe... i'll enter them anyway
		WAIT NEVERMIND - the use AFLP data - we don't use this??

	Zhong et al. 2009 Cons Gen
		location too vague; not using paper

6/1/2012
Diana Li

	Machado-Schiaffino et al. 2011 Bio Cons
		authors exclude certain loci, but their He data only reports the average of ALL of them - aksed Malin

	Nielsen et al. 2010 Cons Gen 
		weird average-y stuff. 
		include paper

	Pini et al. 2011 Coral Reefs
		gene diversity; I can't find the sample sizes... 
		enter and leave it blank
		they report several avg He per site and also an averaged avg He per site. I'll just enter the averaged avg. 

	Roy et al. 2010 Con Gen Res
		Q solution - don't worry; go ahead.

	Saenz-Agudelo et al. 2011 PRSB
		I can't seem to access the data supplement http://rspb.royalsocietypublishing.org/content/suppl/2011/02/07/rspb.2010.2780.DC1.html which may have the He data - the paper doesn't list any - email the peeps
		email sent

	Sekino et al. 2011 Cons Gen
		it seems that they use all the loci even though some loci have a sample that deviates from hwe - i'll include all of them?
		also, they compare with a previous study (Ortega-Villaizan et al 2006) and there is mention of captive fish? but it is unclear whether these samples have captured fish? perhaps i'll leave this one for later - it seems okay though
		enter only the approved locations but all loci okay

6/4/2012
Diana Li

	Teske et al. 2010 Mar Biol
		only report avg He, but 1 locus is out of hwe in 2 locations & LD in 3 total (2 pairs) - keep all

	Tripp-Valdez et al. 2010 Fish Res
		there are some loci that are out of hwe in certain samples, but they say that they did not find a clear tendency of non-hwe in specific loci or location, so there's a chance that those deviations are due to stochastic events in allelic frequency distributions - does this mean that they are NOT due to selection? should i keep them all? yes

6/7/2012
Diana Li

	Roberts & Ayre 2010 Mar Ecol Prog Series
		excluded 2 locations that were out of hwe; the only reported average He per location anyway

	Schunter et al. 2011 J of Exp Mar Biol & Ecol
		Two locations had 2 loci that were non HWE, but authors do not exclude those two loci or locations, so included everything

	end spreadsheet 001

	Smith & Campana 2010 CJFAS
		only has allele freq averaged per location and they assume HWE but don't actually say if it is? - look at original papers instead - Campana et al, 1999; 2000

	Camapana et al. 1999 CJFAS
		does not report He data; not using paper

	Campana et al. 2000 Fish Res
		also does not actually test for He data; not using paper

	White et al. 2010 Mol Ecol
		they say that markers may not be neutral - LnRH values and Fisher's - data analyzed in 2 groups: Crup7 alone and all others without Crup7 - we don't use ln(RH); not using paper

6/11/2012
Diana Li

	Maggio et al. 2009 ICES JMS
		some loci deviate from hwe while others don't for different sample sites
		COME BACK - of 84, 18 deviate; 1 locus has no deviations; some are scattered around; another locus has almost all deviate from hwe - Malin says enter it but NOTE that it is questionable - also uses data from Garioa et al. 2004 which wasn't already entered so i kept it in the Maggio entry

	Zhao et al. 2011 Biochem Systematics and Ecol
		just check - we're allowed to use Nei's gene diversity, yes? that's just like regular gene diversity?
		does not talk about hwe... i'll enter them anyway
		WAIT NEVERMIND - the use AFLP data - we don't use this

6/28/2013
Kieryn Graham

	McDowell et al 2007	
		Entered; sequences not entered

	Knudsen et al 2007	
		Not entered - mtDNA data not there

	Robertson et al. 2007	
		Not entered - mtDNA data not there

	Anderson 2007	
		Not entered - diversity averaged across wide area

	Borsa et al 2007	
		Not entered - mtDNA data not there

	Miller & Crib 2007	
		Not entered - mtDNA data not there

	Miyazaki et al 2007	
		Not entered - not marine species - freshwater

7/5/2013
Kieryn Graham

	Apostolidis et al. 2007	
		Not entered - mtDNA data not there

	Ball et al. 2007	
		Entered

	Bay, Crozier, & Caley 2006	
		Not entered - relevant mtDNA data not there

	Bentacur R Acero 2006	
		Not entered - no mtDNA data

	Bianco, Zupo, & Ketmaier 2006	
		Not entered - no mtDNA data

	Bowen et. Al. 2006	
		Entered

	Bowen, Muss, Rocha & Grant 2006	
		Not entered - averaged over large area

	Bremer et al. 2005	
		Not entered - relevant mtDNA data not there

7/8/2013
Kieryn Graham

	Casthilo et al. 2007	
		Not entere - only one site
	Chevolot et al. 2007	
		Entered - took greatest sample size where 2 samples from one location; averaged across locations for latitude where multiple latitudes (<3) but only one set of mtDNA data
	Chiang et al. 2006	
		Not entered - averaged over >3 degrees latitude
	Correia et al. 2006 	
		Entered - omitted river sample and sample sizes <4; Haplotype data not yet entered
	Craig et al. 2007	
		Entered

7/9/2013
Kieryn Graham

	de Souza et al. 2006	
		Entered
	Duncan et al. 2006	
		Entered
	Francisco Vieira Almada 2006	
		Entered
	Froukh Koschzius 2007	
		Entered

7/10/2013
Kieryn Graham

	Garcia-rodriguez Perez-Enriquez 2006	
		Not entered - lobster

	Gold Saillant 2007	
		Not entered - doesn't give mtDNA data

	Grant Bowen 2006	
		Not entered because a different spps for each sample site

	Grant Spies Canino 2006	
		Not entered but Metastudy

	Graves McDowell 2006	
		Not entered - doesn't give mtDNA data

	Gray et al. 2006	
		Not entered - doesn't give mtDNA data

	Haney Silliman Rand 2007	
		Entered - left out grouped samples (>3 degrees)

	Hayden Lipinski & Matthee 2007	
		Not entered - no relevant data

	Hempelmann et al. 2004	
		Not entered - relevant mtDNA data not there

	Herwerden et al. 2006	
		Not entered - no mtDNA data

	Hoelzel et al. 2006	
		Not entered - averaged over >3 degrees latitude

	Hoolihan Anand Herwherden 2006	
		Entered

	Hsiu, Chen Shao 2007	
		Not entered - doesn't give mtDNA data

	Itoi et al. 2007	
		Not entered - doesn't give mtDNA data

	Keeney & Heist 2006	
		Skipped for now - could calculate He and Pi from Table 2

	Kim Park Kim 2006	
		Entered

	Kuhn Gaffney 2006	
		Not entered - no useful data reported

	Leary et al. 2006	
		Not entered - no useful data reported

7/12/2013
Kieryn Graham

	Lessios Robertson 2006	
		Not entered - mtDNA data not there

	Liu et al. 2006	
		Entered

	Liu et al. 2007	
		Skipped for now - could calculate He and Pi from Table 2

	Mabuchi Okuda Nishida 2006	
		Not entered - mtDNA data not there

	Markos Rogers-Bennet Dennis 2007	
		Entered - pi given only, not He

	Ortega Villaizan Romo et al. 2006	
		Entered

	Paine Mcdowell Graves 2007	
		Not entered - mtDNA data not there

	Quattro et al. 2006	
		Skipped for now - could calculate He and Pi from Tables 1 and 3

	Read Bellwood van Herwerdern 2006	
		Not entered - mtDNA data not there

	Salles et al. 2006	
		Not entered - mtDNA data not there

	Santini Polacco 2006	
		Not entered - sample sizes <4

	Santos et al. 2006	
		Entered

	Shaoxiong et al. 2006	
		Not entered - doesn't give mtDNA data

	Smith McPhee Roberts 2006	
		Not entered - mtDNA data not there

	Steffani et al. 2006	
		Not entered - mtDNA data not there

	Steffani Knutsen 2007	
		Not entered - mtDNA data not there

	Taylor Hellberg 2006	
		Not entered - mtDNA data not provided

	Teske et al. 2007	
		Not entered - doesn't give mtDNA data

	Thacker Hardman 2005	
		Not entered - doesn't give mtDNA data

	Ursvik et al. 2007	
		Not entered - relevant mtDNA data not there

	van Herwerden Doherty 2006	
		Not entered- sampled around "hybrid zones"

	Vigliola et al. 2007	
		Not entered - no mtDNA data

	von der Heyden Lipinski Matthee 2007	
		Entered

	Wexler et al. 2007	
		Not entered - no relevant data

	Wilson 2006	
		Entered

	Yunguo et al. 2011	
		Not entered - msats only - no mtDNA

	Zane et al. 2006	
		Entered

	Zhu et al. 2008	
		Not entered - doesn't give mtDNA data

7/17/2013
Kieryn Graham

	Alvarado Bremer Baker Mejuto 1995	
		Excluded because just one sample site suitable 

7/25/2013
Kieryn Graham

	Arnason 2004	
		Not entered - no useful data reported

	Avise 1992	
		Not entered - but metaanalysis?

	Bargelloni et al. 1994	
		Not entered - no useful data reported

	Bentzen et al. 1998	
		Not entered - no relevent data

	Bernadi Goswami 1997	
		Not entered - sample sizes too small

	Billington Hebert 1991	
		Not entered - no useful data reported

	Bowen Grant 1997	
		Entered

	Bremer et al. 1998	
		Not entered - genetic diversity data not provided

	Bremer et al. 1999	
		Not entered - no useful data reported

	Bremer et al. 2005	
		Not entered - no useful data reported

	Bremer Mejuto Ely 1995	
		Not entered - sample range > 3 degrees

	Bremer Mejuto et al. 1999	
		Entered but He not there

	Brown Chapman 1991	
		Not entered because estuarine species

	Cao et al. 1998	
		Not entered - just one individual

	Carr et al. 1995	
		Entered

	Carr et al. 1999	
		Not entered - no useful data reported

	Cecconi Giorgi Mariottini 1995	
		Not entered - no useful data reported

	Cesaroni et al. 1997	
		Not entered - no useful data reported

	Crow et al. 1997	
		Excluded because looked at clutches

	Delarbre et al. 1998	
		Not entered - sequenced just one individual

	Elliot Smolenski Ward 1994	
		Not entered - relevant mtDNA data not reported

7/30/2013
Kieryn Graham

	Elliot et al. 1998	
		Excluded because not mtDNA sequence

	Gold & Richardson 1998	
		Not entered - looked at variation in restriction sites of mtDNA

	Gold & Richardson 1998 - Snappers and Groupers	
		Not entered - looked at variation in restriction sites of mtDNA

	Gold et al. 1993	
		Not entered - looked at variation in restriction sites of mtDNA

	Gold et al. 1994	
		Not entered - looked at variation in restriction sites of mtDNA

	Gold Kristmundsdottir Richardson 1997	
		Not entered - looked at variation in restriction sites of mtDNA

	Gold Richardson 1991	
		Not entered - looked at variation in restriction sites of mtDNA

	Gold Sun Richardson 1997	
		Not entered - looked at variation in restriction sites of mtDNA

	Gomes Dales Oxenford 1997	
		Not entered - doesn't report any useful information

	Graves McDowell 1994	
		Not entered - used RFLP analysis of mtDNA 

	Hare et al. 1994	
		Not entered - useful info not reported

	Hare et al. 1998	
		Not entered - doesn't report any useful information

	Jean et al. 1998	
		Entered

	Johns Avise 1998	
		Not entered - useful info not reported

	Jones Quattro 1999	
		Not entered - doesn't report any useful information

8/10/2013
Kieryn Graham

	Kai Nakayama Nekabo 2002	
		Not entered - one location only

	Lage et al. 2001	
		Not entered - msats only

	Lessios et al. 1995	
		Not entered - relevant mtDNA data not reported

	Mcmillan Palumbi 1995	
		Not entered - relevant mtDNA data not reported

	Milbury et al. 2004	
		Not entered - oysters

	Miya Nishida 1998	
		Not entered - doesn't report any useful information

	Pepin & Carr 1993	
		Skipped for now but can calculate pi and He from Table 4

	Richardson Gold 1997	
		Not entered - doesn't report any useful information

	Ritchie et al. 1996	
		Not entered - doesn't report any useful information

	Rocha-Olivares Rosenblatt Vetter 1999	
		Not entered - doesn't report any useful information

	Rocha-Olivares Vetter 1999	
		Entered

	Rosel Block 1996	
		Not entered - don't report He for each region sampled

	Saillant Bradfield Gold 2010	
		Not entered - msats only

	Saitoh 1998	
		Excluded because not mtDNA sequence

	Sarver Freshwater Walsh 1996	
		Not entered - phylogentic study - relevant mtDNA data not reported

	Scoles Collette Graves 1998	
		Entered - omitted Australian sample because only general (>3 degrees) location given

	Stepien 1999	
		Entered 

	Strecker et al. 1996	
		Excluded because only sampled 1 site

	Taylor & Bentzen 1993	
		Not entered - anadromous/lake resident

	Tinti et al. 1999	
		Not entered - doesn't report any useful information

	Vis et al. 1997	
		Entered

	Ward et al. 1998	
		Excluded because only looked at mtDNA restriction sites, not sequence

8/22/2013
Kieryn Graham

	Aboim et al. 2005
		Not entered - no info on sample size
		Malin: Agreed. No data on sample size.

	Alvarado Bremer et al. 2005
		Not entered - samples pooled over >3 degrees	

	Barber & Bellwood 2005
		Not entered - relevant mtDNA data not there	
	
	Cardenas et al. 2005
		Not entered - relevant mtDNA data not there	

	D'amato & Calvalho 2005
		Not entered - RFLP sequencing	
		Malin: The authors did RFLPs, not mtDNA sequencing, so we won't enter this one

	Durand et al. 2005
		Not entered - sample range too large and no pi	
		Malin: S2 has too wide of a longitude range. I can't find pi either.

	Francisco, Faria, Lengkeek et al. 2011
		Entered	
	
	Garber Tringali Franks 2005
		Entered	

	Gonzalez Beerli Zadoya 2008	
		Not entered - msat data only	

	Holcroft 2005
		Not entered - no relevant data	

	Hurler Patzner Sturmbauer 2005
		Not entered - mainly morphometrics, no relevant mtDNA data to enter	

	Kim et al. 2005
		Not entered - no relevant data	

	Kochzius Blohm 2005	
		Entered	

	Lemaire Versini Bonhomme 2005
		Entered but collection year not given and some sample sites didn't appear to be on the map	

	Lourie Green Vincent 2005
		Not entered because only 1 sample site per species	

	Nakadate et al. 2005
		Not entered because not mtDNA sequence	

	Pruett Saillant Gold 2005
		Not yet entered - use Table 1?	He and pi not reported, so we won't enter this paper for now.

	Rocha 2004
		Entered	

	Rocha et al. 2005
		Not entered - no relevant data	

	Schonhuth et al. 2005
		Not entered - no relevant data	

	Sotka et al. 2005
		Entered	Enter as 1 marker (both loci are on the mtDNA genome, so effectively one)

	Teske et al. 2005
		Entered	

	Tonteri et al. 2005
		Not entered - relevant mtDNA data not there	

	Valsecchi et al. 2005
		Not entered - relevant mtDNA data not there	

	Valsecchi Pasolini et al. 2005
		Not entered - no relevant data	

	Yu et al. 2005
		Entered
		Malin: Bottom-right of Table 1 has the sample sizes: Q1 is 15, Q2 is 16, and Q3 is 15. Hard to find!

9/11/2013
Kieryn Graham

	Akimoto et al. 2005
		Not entered - ppr in Chinese but doesn't appear to be any useful data anyway, at least in tables

	Bellwood et al. 2004	
		Not entered - no relevant data

	Bernadri & Vagelli 2004	
		Entered; biggest sample for each location entered; pi not given

	Betancur-R & Acero 2004	
		Not entered - no relevant mtDNA data

	Carlsson et al. 2004	
		Entered

	Chiou & Lee 2004	
		Not entered - relevant mtDNA data not there

	Crow et al. 2004	
		Not entered - no relevant data

	Gharrett et al. 2005	
		Not entered - doesn't contain any useful mtDNA data

	Hoolihan et al. 2004	
		Not entered - RFLP analysis of mtDNA

	Kai et al. 2004	
		Not entered - only sampled from one site

	Kimura et al. 2004	
		Not entered - only gives "mean of pairwise sequence differences of mtDNA"

	Near et al. 2004	
		Not entered - relevant mtDNA data not there

	Poulin et al. 2004	
		Not entered - relevant mtDNA data not there

	Quinta et al. 2004	
		Not entered - restriction analysis

	Sandoval-Castillo et al. 2004	
		Not entered - RFLP analysis of mtDNA

	Shaw et al. 2004	
		Not entered - RFLP analysis of mtDNA

	Shigenobu et al. 2005	
		Not entered - no relevant data

	Suzuki et al. 2004	
		Not entered - relevant mtDNA data not there

	Vinas et al. 2004 (Albacore)	
		Entered

	Vinas et al. 2004 (Bonitos)	
		Entered

	Yamanoue et al. 2004	
		Not entered - relevant mtDNA data not there

	Yanagimoto et al. 2004	
		Not entered - no relevant mtDNA data

9/12/2013
Kieryn Graham

	Banford et al. 2004	
		Not entered - doesn't contain any useful mtDNA data	

	Bay et al. 2004	
		Entered	

	Gold & Burridge 2004	
		Not entered - SSCP analysis	
		Malin: Correct, we don't enter this.

	Hoarau et al. 2004	
		Entered; dates not given	

	Knutsen et al. 2004	
		Not entered - no He values	
		Malin: No, we can't use p-values for our purposes.

	Lecomte et al. 2004	
		Entered	

	Lo Brutto et al. 2004	
		Not entered - RFLP analysis of mtDNA	

	Mabuchi et al. 2004 Apogon	
		Not entered - relevant mtDNA data not there	

	Mabuchi et al. 2004 Pseudolabrus	
		Not entered - doesn't contain any useful mtDNA data	

	Ovenden et al. 2004	
		Not entered - no He data for mtDNA sequence	
		Malin: Looks like the numbers in parentheses are what we need for sequenced mtDNA (Table 1). But I don't see any He data for the mtDNA sequence (Table 4 is for the RFLPs). 

	Rodriguez-Grana et al. 2004	
		Not entered - no relevant mtDNA data	

9/19/2013
Kieryn Graham

	Ahmed et al. 2004	
		Not entered - RFLP analysis of mtDNA	

	Alarcon et al. 2004	
		Not entered - allozyme and msat data only	

	Altman & Taylor 2003	
		Not entered - RFLP analysis of mtDNA	

	Appleyard et al. 2004	
		Not entered - RFLP analysis of mtDNA	

	Aurelle et al. 2003	
		Not entered - looked at msats only	

	Ball et al. 2003	
		Not entered - RFLP analysis of mtDNA	

	Bargelloni et al. 2003	
		Not entered - SSCP analysis	
		Malin: We can't use SCCP

	Bernal-Ramirez et al. 2003	
		Not entered - SSCP analysis	
		Malin: We can't use SCCP

	Bernardi et al. 2003	
		Not entered - doesn't contain any useful mtDNA data	

	Borsa 2003	
		Not entered - SSCP analysis	
		Malin: We can't use SCCP

	Brykov et al. 2004	
		Not entered - RFLP analysis of mtDNA	

	Carlin et al. 2003	
		Entered	

	Casey et al. 2004	
		Not entered - no relevant mtDNA data	

	Cope 2004	
		Entered	

	Cordes & Graves 2003	
		Not entered - RFLP analysis of mtDNA	

	Fauvelot et al. 2003	
		Not entered - one sample site only per species	

	Garber et al. 2004	
		Entered	

	Garcia-Machado et al. 2004	
		Entered	
		Malin: It's ok to enter even if it only has pi

	Hassan et al. 2003	
		Not entered - no relevant mtDNA data	

	Innoue et al. 2003	
		Not entered - relevant mtDNA data not there	

	Jerome et al. 2003	
		Not entered - doesn't contain any useful mtDNA data	

	Kai et al. 2003	
		Not entered - no relevant mtDNA data	

	Karaiskou et al. 2004	
		Not entered - RFLP analysis of mtDNA	

	Keeney et al. 2003	
		Entered	
		Malin: Looks like 1066-1067. So let's say 1066.

	Kochzius et al. 2003	
		Not entered - no relevant mtDNA data	

	Kontula & Vainola 2003	
		Not entered - relevant mtDNA data not there	

	Luzier & Wilson 2004	
		Entered	

2/23/2013
Kieryn Graham

	Balanov et al. 2004	
		Not entered - paper not in English	

	Mabuchi et al. 2003	
		Not entered - averaged over >3 degrees	

	Manchado et al. 2004	
		Not entered - no relevant mtDNA data	

	McCartney et al. 2003	
		Not entered - mtDNA data for one sample location only	

	Moller & Gravlund 2003	
		Not entered - no relevant mtDNA data	

	Near et al. 2003	
		Not entered - no relevant mtDNA data	

	Ovenden et al. 2003	
		Not entered - only give results for RFLP analysis of mtDNA	

	Ramon et al. 2003	
		Not entered - don't provide relevant mtDNA data	

	Rivera et al. 2004	
		Entered	
		Malin: Actually, pi is given as # nucleotides across the whole sequence (it's confusing!). So, you need to divide by 398 (the number of basepairs). E.g., 6.084/398 for Hawaii.

	Schrey & Heist 2003	
		Not entered - msat data, no new mtDNA data	

	Shedlock et al. 2004	
		Not entered - phylogenetic study, no useful mtDNA data reported	

	Sigurgislason & Arnason 2003	
		Not entered - relevant mtDNA data not there	

	Smith & Paulin 2003	
		Not entered -  insufficient details about sampling sites	

	Smith et al. 2003	
		Not entered - relevant mtDNA data not there	

	Tigano et al. 2004	
		Not entered - no relevant data	

	Vidal & Melendez 2003	
		Not entered - no relevant mtDNA data	

	Withler et al. 2004	
		Not entered - mainly msat data, no relevent mtDNA data reported	

	Yanagimoto 2003	
		Not entered - RFLP analysis of mtDNA	

10/12/2013
Kieryn Graham

	Akimoto et al. 2002	
		Not entered - doesn't contain any useful mtDNA data	

	Allegrucci et al. 1999	
		Not entered - doesn't contain any useful mtDNA data	

	Apostolidis et al. 2001	
		Not entered - doesn't contain any useful mtDNA data	

	Appleyard et al. 2002	
		Not entered - doesn't contain any useful mtDNA data	

	Arnason et al. 2000	
		Entered	
		Malin: Correct, because pi already converted for 100bp section

	Arnaud et al. 1999	
		Not entered - restriction analysis	

	Ball et al. 2000	
		Not entered - RFLP analysis of mtDNA	

	Bernardi & Crane 1999	
		Not entered - no relevant mtDNA data	

	Bernardi 2000	
		Entered - Pi not given; collection dates not given	

	Bernardi et al. 2002	
		Not entered - mtDNA data not split up by location, just per species	

	Borsa 2002	
		Not entered - looked at allozyme frequency data	

	Bostrom et al. 2002	
		Not entered - RFLP analysis of mtDNA	

	Bowen et al. 2001	
		Entered - collection dates not given	

10/15/2013
Kieryn Graham

	Brykov & Podlesnykh 2001	
		Not entered - RFLP analysis of mtDNA

	Bucciarelli et al. 2002	
		Not entered - no relevant data

	Buonaccorsi et al. 2001	
		Not entered - RFLP analysis of mtDNA

	Burridge & Smolenski 2003	
		Not entered - msats

	Burridge & White 2000	
		Not entered - phylogenetic study, no useful mtDNA data reported

10/26/2013
Kieryn Graham

	Burridge 1999	
		Not entered - doesn't contain any useful mtDNA data	

	Burridge 2000	
		Not entered - don't provide relevant mtDNA data	

	Chenoweth et al. 2002	
		Not entered - RFLP analysis of mtDNA	

	Chow & Takeyama 2000	
		Not entered - restriction analysis	

	Chow et al. 2000	
		Not entered - restriction analysis	

	Craig et al. 2001	
		Not entered - no relevant mtDNA data	

	Dawson et al. 2002	
		Not entered - mtDNA data not given by latitude, only by species	

	Derome et al. 2002	
		Not entered - doesn't contain any useful mtDNA data	

	Donaldson & Wilson 1999	
		Not entered - no relevant data	

	Ely et al. 2002	
		Not entered - sample locations unclear	
		Malin: Agreed, we can't tell where the samples were taken

	Garber et al. 2001	
		Not entered - no useful mtDNA data provided	

	Gardner & Ward 2002	
		Not entered - allozyme analysis and analysis of polymorphisms in length of restriction fragments from mtDNA	

	Gharrett et al. 2001	
		Not entered - restriction analysis	

	Gold et al. 2001	
		Not entered - used restriction analysis and don't provide details of sampling sites	

	Grant & Leslie 2001	
		Not entered - no relevant data	
		Malin: Not a meta-analysis, but I couldn't find any relevant data

	Graves & McDowell 2003	
		Not entered - no useful mtDNA data provided	

	Guarniero et al. 2002	
		Entered - excluded river samples	
		Malin: We can exclude the river samples.

	Hanel & Sturmbauer 2000	
		Not entered - no useful mtDNA data provided	

	Hanel et al. 2002	
		Not entered - no useful mtDNA data provided	

	Hauser et al. 2001	
		Not entered - RFLP analysis of mtDNA	

	Hickerson & Ross 2001	
		Entered	
		Malin: Actually, the only relevant data I could find were the DIAGONALS of Table 4 (these values are pi, but need to be divided by 378)

	Higuchi & Kato et al. 2002	
		Entered	

	Hoarau & Borsa 2000	
		Not entered - He and Pi not reported for sequenced mtDNA	
		Malin: They also sequenced 273bp, but I couldn't find He or pi reported

	Ishikawa et al. 2000	
		Not entered - no relevant data	

	Ishikawa et al. 2001	
		Not entered - no relevant data	

	Jean et al. 1999	
		Not entered - relevant mtDNA data not there	

	Kai et al. 2002	
		Not entered - do not give data by sample site	

	Karaiskou et al. 2003	
		Not entered - do not give data by sample site	

	Kojima et al. 2001	
		Not entered - do not give data by distinct sample sites	

	Kotoulas et al. 2003	
		Not entered - doesn't contain any useful mtDNA data	

	Lee & Tsai 1999	
		Not entered - doesn't contain any useful mtDNA data	

	Lundy et al. 2000	
		Not entered - relevant mtDNA data not there	

	Mamuris et al. 1999	
		Not entered - RFLP analysis of mtDNA	

	McCafferty et al. 2000	
		Not entered - doesn't contain any useful mtDNA data	

10/27/2013
Kieryn Graham

	Nesbo et al. 2009	
		Entered	

	Nugroho et al. 2001	
		Not entered - restriction analysis	

	Olsen et al. 2002	
		Not entered - restriction analysis	

	Papasotiropoulos et al. 2002	
		Not entered - restriction analysis	

	Perrin & Borsa 2001	
		Not entered - SSCP analysis	
		Malin: Ok

	Quinteiro et al. 2000	
		Not entered - restriction analysis	

	Reeb et al. 2000	
		Not entered - no useful mtDNA data provided	

	Riginos & Nachman 2001	
		Entered	
		Malin: Yes, pi is on the diagonal of Table 2. It needs to be divided by 408.

	Riginos & Victor 2001	
		Not entered - no relevant data	

	Rocha et al. 2002	
		Entered	

	Shaw et al. 1999	
		Not entered - mainly msat analysis; mtDNA RFLP analysis	

	Smith et al. 2002	
		Not entered - restriction analysis	

	Stepien et al. 2000	
		Entered	
		Malin: 443bp (I looked on Genbank for the accession numbers on page 1704)

	Stepien et al. 2001	
		Entered	

	Streelman et al. 2002	
		Not entered - doesn't contain any useful mtDNA data	

	Tabata & Taniguchi 2000	
		Not entered - restriction analysis	

	Takeyama et al. 2001	
		Not entered - RFLP analysis of mtDNA	

	Terry et al. 2000	
		Entered - Pi not given; collection dates not given	

	Tinti & Piccinetti 2000	
		Not entered - doesn't contain any useful mtDNA data	

	Tinti et al. 2000	
		Not entered - doesn't contain any useful mtDNA data	

	Tringali et al. 1999	
		Not entered - restriction analysis?	

	Turner et al. 1999	
		Not entered - restriction analysis	

	Mizuno et al. 2000	
		Not entered - restriction analysis	

2/22/2015
Stan Piotrowski

	Durand et al. 2005 (2/22/15): Nuclear and mitochondrial DNA markers indicate unidirectional gene flow of Indo-Pacific to Atlantic bigeye tuna (Thunnus obesus) populations, and their admixture off southern Africa 
	● “Protocols and data for the mtDNA ATCO segments were from Chow et a. (2000)”...RFLP analyses were used, not sequencing 
	● No data entered for ATCO locus

	Evans et al. 2010 (2/22/15): Strong genetic but not spatial subdivision of two reef fish species targeted by fishers on the Great Barrier Reef 
	● Marker name = “hyper-variable control region I (HVR-I)” of mitochondrial DNA 
		○ hypervariable control region is a location within the D-loop of the mtDNA in which base pairs of nucleotides have substitutions
		○ DNA sequence of the control region is termed hypervariable because it accumulates point mutations (single base mutations) approximately 10 time te rate of nuclear DNA
	● Haplotype diversity reported; no standard deviation reported 
	● Nucleotide diversity and standard deviation reported 
	● Variation sites = allele numbers? 

	Eytan & Hellberg 2010 (2/22/15): Nuclear and mitochondrial sequence data reveal and conceal different demographic histories and population genetic processes in Caribbean reef fishes 
	● Species tested: 
		a. Acanthemblemaria aspera (roughhead blenny): lives in shelters in live and standing dead coral off the reef surface (i.e. high-profile habitat) 
		b. Acanthemblemaria spinosa (spinyhead blenny): found at the base of coral skeletons and in coral rubble (i.e. low-profile habitat) 
	● Come back to this one 


	Farnsworth et al. 2010 (2/22/15): Genetic structure across the GBR: evidence from short-lived gobies 
	● Examined the extent to which barriers to gene flow conuptribute to cross-shelf patterns by examining mtDNA of gobies (genus Eviota; perfect candidates because of their short life spans and inferred high rate of mutation, as well as small size and sedentary lifestyle as adults which limits dispersal capabilities)
		○ E. albolineata (spotted fringefin goby): found exclusively on mid- and outer-shelf reefs 
			■ 392 bp of COI gene amplified 
			■ Two mid-shelf areas = North Direction Island, Lizard Island (n=39); h=0.95, pi=1.05 +/- 0.59 
			■ Two outer-shelf areas = Hicks Reef, Day Reef (n=41); h= 0.92, pi=0.9 +/-0.52
		○ E. queenslandica (Queensland dwarfgoby): found exclusively on inner- and mid-shelf reefs 
			■ 408 bp of COI gene amplified 
			■ Two inner shelf-reefs = Martin Reef, Linnet Reef 
			■ Two mid-shelf reefs = North Direction Island, Lizard Island 
	● Samples sequenced by Macrogen Inc. (Korea) 

	Fauvelot & Borsa 2011: Patterns of genetic isolation in a widely distributed pelagic fish, the narrow-barred Spanish mackerel (Scomberomorus commerson)
	● Impetus for study was from reports of strong population structure for narrow-barred Spanish mackerel (usually pelagic fishes exhibit little geographic differentiation) 
		● Spp. = Scomberomorus commerson 
		● Common name = narrow-barred Spanish mackerel 
		● Country= 
			○ Indonesia (several different provinces, N=4)
				■ Java Sea 
				■ Bali
				■ West Papua 
			○ New Caledonia (special collectivity of France; all samples obtained from local fisherman)
				■ Belep Islands (north-western lagoon of New Caledonia)
				■ Nepoui (north-western lagoon of New Caledonia)
				■ Noumea (south-western lagoon)
				■ Canala (eastern coast) 
		● Collection years from 2007-2009, average to input in data sheet-- 2008
		● Marker name-- “highly variable 5’ end of the mitochondrial control region (380-383 bp)” 
		● Number of individuals sampled-
			○ Indonesia
				■ Java Sea (n=4)
				■ Bali (n=6) 
				■ West Papua (n=10) 
			○ New Caledonia (n=194) 
		● Length of mtDNA loci, in base pairs 
			○ Control region = 380-383
			○ cytochrome b = 281 
		● Note, the authors also amplified 8 microsatellite loci developed from samples taken from the Persian Gulf and Oman Sea in the individuals from Bali, Java, West Papua, and in a subsample of the randomly selected individuals from New Caledonia 
		● He = haplotype diversity (i.e. expected heterozygosity), pi = nucleotide diversity 
			○ Java Sea/Bali-- He = 1.000, pi = 0.052 
			○ West Papua-- He = 0.929, pi = 0.023 
	Are these values for the eight microsatellite loci or for the mtDNA control regions?  Very vague in the article-- need to clarify 

	Gaither et al. 2010: Genetic sequences of introducing allopatric lineages of Bluestriped Snapper (Lutjanus kasmira) to Hawaii 
	● used data from introductions in Hawaiian Islands and from source populations in Marquesas and Society islands.. can I not use the source populations since the locations are very vague? 
		● how about with the Hawaiian Islands?  Check to see if degrees lat. and lon. are okay 
		● use lat and lon from google maps to where the arrows are in the map figure 
	● No collection years listed, left blank 

	Friess & Sebderry 2011: Genetic evidence for a single stock of the deep-sea teleost Beryx decadactylus in the North Atlantic Ocean as inferred from mtDNA control region analysis 
	● decimal degrees not listed, estimated using map on the figure over using minute lat and lon 
	●average year listed (between 2003 and 2007 specimens were collected) 
	● bp of control region sequenced = 403

	Gaither et al. 2010: Genetic evaluation of marine biogeographical barriers: perspectives from two widespread Indo-Pacific snappers (Lutjanus kasmira and Lutjanus fulvus)
	● if multiple loci are reported, disregard all the nuclear DNA markers? only concerned with the mtDNA markers? 
	● No data were entered for either Lizard Island nor Okinawa, as there were no molecular diversity values indicated even though fish were sampled from these sites 
	● Question: how to find allele frequencies and sequences? 
	● No dates reported for collection 

	Fruciano et al. 2011: Atlantic-Mediterranean and within-Mediterranean molecular variation in Coris julis 
	● sequences deposited in GenBank aren’t listed in the paper
	● no haplotype diversity or pi values reported, didn’t input any data for this paper 

2/24/2015
Ash Bhattacharjee

	Ahonen et al, 2009 10:15pm
	Asked how to format data sheet
	- Source is written slightly differently
	- "A few of the samples came from individuals housed in aquaria known to be of eastern Australian 
	provenance." > this disqualifies the Eastern australian data as these are captive animals?
	- SITE: No specific site given, only names of oceans. What to enter?
	- "We applied two different genetic markers, mitochondrial DNA and microsatellites, to measure 
	the genetic diversity and differentiation between regions and to determine whether losses of 
	genetic variation are evident from any recent population bottlenecks." >does this mean more than
	one number of markers?
	- Number of alleles given under microsatellite data (or allelic richness): do I use this to compute
	allele frequency? 
	- Having a hard time finding allele sequence for Carcharias taurus in gen bank, when I put in 
	"Ahonen Carcharias taurus" 8 results only include microsatellite sequence. Otherwise if put in
	"control region Carcharias taurus" a different set of sequence hits show up. Which do I use?
	-- End 11:30 pm --

3/1/2015
Ash Bhattacharjee

	-- 1:00pm
	Correia et al, 2012
	- Spp: Listed as Conger conger on pub but Fishbase calls it Conger wilsoni (many different spp across
	geographic ranges, which to use?)
	- The congers used in this study were collected from the local artisanal fisheries mostly captured 
	by hook and pots in shallow coastal waters of the NE Atlantic and western Mediterranean between 
	December 2005 and September 2008. 
	*Unable to use this paper, as they used captive populations

	Anderson et al, 2012
	- "The nucleotide diversity (π) over all samples was 0.012 (SD = 0.0003)". The data on diversity 
	averaged across >3° latitude and longitude 
	*Unable to use this paper

	Andre et al, 2011
	- Only Fst values are given for each sample, no He values. Can Fst values be used in place?
	- No nucleotide diversity data available!
	*Did not enter data yet, waiting to see what to do for the diversity values (Fst values were also 
	available for Anderson et al, 2012)

	Antoni et al, 2011
	- "We surveyed genetic variation at a 617-base-pair fragment of the ND4 subunit of NADH dehydrogenase
	encoded by mitochondrial DNA in five localities along the northern Gulf of Mexico and the western 
	Atlantic Ocean." This means that the marker name is ND4? (that's how I entered it).
	- "Sample sizes ranged between 27 and 32 gray triggerfish per locality". Specific sample sizes/site
	not given. So far I have averaged them at 29 (couldn't put 29.5), thought it would be safe to round 
	down.
	- Lat & Long coordinates not given, looked them up for the overall region given (texas, louisiana...)
	The coordinates don't exactly match up with the rough map they provide in Figure 1, but I can't
	guess coordinates from the vague map... should I?
	- Collection year: 2008 & 2009, how do I average that? Input 2009 as of now, should I change that?
	- Allele frequency and/or numbers not given, haplotype distributions are given... is there some
	way I can use this to calculate allele frequency?
	- Allele sequence: Genbank Accession number given, alongside haplotype. Inputted each sequence from 
	1-20 (but it went up to 38).. I copy pasted all of them, but they look the same. Are they?
	** copy pasting in FASTA sequences took quite a while..

	Apostolidis et al, 2009
	- Uses mtDNA RFLPs, so unable to input data from this paper

	Balanov & Kukhlevskii, 2011
	- This study looks at morphological differences between the specimens.. I cannot find any
	diversity data anywhere in the paper or the references (pertaining to this particular spp).

	Banford, 2010
	- A morphological analysis of inshore halfbeak, molecular analysis only provides genetic distance.
	No diversity results provided in the paper (he or pi)

	Bariche & Bernardi, 2009
	- "Samples were collected for this study by spear, or at fish markets": are fish at fish markets 
	considered wild? They could be farmed, or wild caught... source is unclear *I've input the data 
	anyway, but let me know if I should remove this study since it is unclearwhether they are all
	wild populations.
	- Says 2 mtDNA regions were studied: Control region and ATPase 6&8, but diversity data only provided
	for ATPase. What is S7 in the table?
	- Some of the sources of their data are external such as GenBank or James Cook University (I've only
	included those from this study that were actually sampled).
	- From this only two sample sites: Mediterranean ocean and Indian ocean have diversity data available
	- Had to look up lat & long, coordinates not provided
	- Collection year left blank, not provided
	- "All sequences were deposited in GenBank (Accession Numbers: GQ396165–GQ396220)". Not sure how to 
	match the GenBank names to the alleles actually sampled in this paper that I am inputting

	Bay & Caley, 2011
	- Is ΦST (ts-tv) the haplotype diversity?
	- The diversity data for this paper is not given numerically, (not that I can tell), but graphed. 
	How do I obtain this data? I cannot find an appendix either, whichI thought may have had the info.
	*Unable to input (he/pi not given only pairwise ΦST), as I don't know what to put in for diversity 
	data. Should I estimate the he and pi values based on the graphs provided (Figure 4)?
	* It also seems like the data has been averaged over regions >3° latitude and longitude (seen in 
	table 1)

	Ben-Tzvi et al, 2012
	- Diversity data is not provided in paper nor in the supplemental pdf. 
	* Unable to use for data entry

	Bernal & Rocha, 2011
	- Previous sequences used from Genbank as well as another study (Atlantic samples). I did not include
	these in my data entry. This paper (2nd author) has sample sequences from Sao Tome.
	- No diversity data available *could not use this paper
	Following Rocha papers back I found another paper by Rocha (author that sampled the sites)
	http://www.luizrocha.com/academic/Publications_files/Rocha_etal_2002.pdf
	>But this seems like a different project 

	Betancur et al, 2011
	- Averages over 3 different sample sites (>3° latitude and longitude) included in results. I have
	omitted this from my data entry and only input data for invididual sample sites which were also 
	provided.
	- Further info was given for native vs invasive species (comparison) within the North Carolina 
	specimens, not sure how to/if I should input this data. I have not put it in as yet.
	- Detailed diversity data for Bahamas, NC, Western indonesia & phillipines (including standard error)
	found in other papers by collaborators *included as footnote in the results figure
	- Collection dates not available for NC, Bermuda, Indonesia & phillipines

	-- end 5:20 pm--

3/7/2015
Ash Bhattacharjee

	-- 3:00 pm
	Biscoito et al, 2011
	- Data on diversity averaged across a wide area (also vaguely identified)
	- Some specimens (individuals from Sesimbra & Azores) taken from commercial fisheries
	> unable to use for data entry

	Bloom & Lovejoy, 2012
	This is a phylogenetic study to determine relationships within Engraulini
	- Diversity data not given and I can't find another paper or appendix that directs me to a paper where
	I can find this information
	- Are these freshwater or marine species? Throughout they are referred to as freshwater, that migrate
	between freshwater and marine habitats... *Fishbase says they are marine √
	> unable to use for data entry

	Blower et al, 2012
	- Specimens acquired from incedental captures, tagged sharks for research purposes and some 
	commercial and recreational fisheries by-catch (is this considered wild-caught or captive?)
	I've included it assuming it is wildcaught
	- Is the sample site: South Australia too vague?
	- Collection dates: 1989-2004 (21 years!): averaged to 1997 (break tie, more recent year)
	- Phylogenetic analysis revealed that 2 eastern Australian haplotypes (Haplotypes 13 & 14; Table 4,
	were substantially more similar to western Indian Ocean (WIO) white shark haplotypes". There are 
	two tables provided with diversity indices, one excluding the Western Indian ocean-like haplotypes
	and one including it. I used the data from the one excluding it just to be safe (might be a different
	species), is this right? 
	I can change the data entry to include it if it does not matter.
	- One site is "unknown region" so I did not include this in my data entry (too vague)
	- There is no diversity data for samples from Tasmania, Western Australia & the unknown region (?)
	- Number of haplotypes given, not number of alleles or allele frequency. I encountered this with a 
	previous paper, which I noted (Antoni et al, 2011). Is there a way to get allele number/freq
	from this? As I see it, haplotypes are a group of alleles or SNPs put together so they aren't exactly
	interchangeable.
	- For allele sequences, GenBank Accession numbers provided (AY026196−AY026224) but not specified
	or matched up with specific alleles, not sure how to find this?

	Borrell et al, 2012
	- Allele numbers given only using microsatellite loci (I didn't input this because its not mtDNA)
	- I'm finding that most papers do not give allele number/frequencies for the mtDNA data or the 
	allele sequences (only found one paper so far with sequences which I got through genbank)... 
	is this normal?

	-- Stop 5:10 PM--

3/7/2015
Ash Bhattacharjee

	-- 7:00 PM
	Broderick et al, 2011
	- Name of species is Broad-barred king mackerel on fishbase, but named grey mackerel on the paper.
	I used the name from fishbase, assuming this is the accepted name and grey mackerel may be a name
	used in specific regions of the world. Is the fishbase one right to use?
	- Sampled from commercial catches across N. Australia. Is this considered captive? It's technically
	caught from the wild, and then sampled... so not sure. If so, then I will delete this data entry. 
	- Names of sample sites for this difficult to enter, several sites within one territory or coast.
	The collection locations were well mapped on Figure 1, so I used that to pin point the coordinates
	per sample site (P01-12). I input the coordinates for each sample collection, alongside the
	coast or territory the paper designates the site at (which was input for site). All of them are from
	Australia, so country was relatively simple.
	- No collection date given.
	- Once again, no allele frequency or sequence data available for mtDNA (numbers given for microsat-
	ellite loci as with other papers).

	Campo & Garcia-Vazquez, 2010
	- Paper also includes: "On the other hand, we also analyzed 94 individuals from
	different North-Atlantic areas of its congeneric species L. whiffiagonis in order to compare genetic
	variation patterns within this genus." The data for L. whiffiagonis does not have coordinates or a
	location, it simply states samples were taken at different ICES divisions (not specified). So I
	did not enter any data for these samples, as the sample site is too vague.
	- Allele number and sequences not available in paper.
	- Accession numbers for control region haplotype sequences are Accession numbers FJ590665–FJ590722.
	Not sure how to match these up with the sampled alleles. 

	Canino et al, 2010a
	- Table 1 gives a really good overview of the important data (sample coordinates, sample 
	sites, number of samples, collection date). 
	- I used the most recent collection date for Puget Sound (two collection years, they sampled twice
	in one site and as per data entry instructions I used the more recent sample since sample size
	was same).
	- Table 3 gives me expected number of mtDNA haplotypes as well as observed number of mtDNA haplotypes.
	If I am using this information for the number of alleles to calculate allele frequency, do I use
	expected or observed? Left blank for now, but I will fill it in once that is cleared.
	- I've been inputting 1 for number of markers so far in most papers, since it doesn't say otherwise. 
	However while reading through, they say they used two markers microsatellite markers and mtDNA markers,
	this would be true for most of the other papers as well. So does that mean nummarkers=2? If so,
	I will go back and change the previous ones. 
	- "A total of 135 substitutions (121 transitions and 14 transversions) at 134 polymorphic sites 
	defined 170 haplotypes (126 singletons) in 346 composite ND2⁄ cyt bfrom 20 to 28. 
	Haplotype diversity (h) ranged from 0.923 to 0.984 (mean h = 0.958) but did not show any
	geographical trends (Table 3)". Since the data in Table 3 is referenced here, I believe that the 
	marker name would be both ND2 and cytb, using the composite ND2/cytb. Is this right? 
	- NADH subunit 2 (ND2) is 1014 bp and cytb is 495 bp, so I assumed the composite bp length would be 
	the sum of the two = 1509

	Canino et al, 2010b
	- This paper only provides diversity data for the microsatellite marker (expected heterozygosity) but
	I could not find diversity indices for the mtDNA. I couldn't find an appendix.
	- "Mitochondrial DNA analysis.—Nucleotide diversity (Hp) and haplotype diversity (h) were estimated 
	using DnaSP version 4.50.3 (Rozas et al. 2003)". The abstract provides an estimate of nucleotide 
	diversity which I assume is an average over the total sample site. The source that is cited is 
	regarding the methodology, not any extra mtDNA info that wasn't included.
	- Given the sample sites range from 41-51 lat and 140-178 long, this would be >3 degrees and 
	I think that would mean I cannot use this data.

	-- Stop: 9:15 PM--

3/13/2015
Ash Bhattacharjee

	-- 3:00 PM
	Cannas et al, 2010
	- Only quantitative morphological differences and genetic distance data provided in this paper. 
	I cannot find the diversity data (nucleotide diversity or expected heterozygosity) to input.
	- Did not use this paper

	Cardenas et al, 2009
	- All data available and entered except allele sequences or accession numbers to find it.

	Carson et al, 2011
	- Used Figure 1 (map) to estimate lat/long coordinates of each sample site
	- Averaged collection date (2007-2009=2008)
	- Allele sequence available as accession numbers, so I went into the Genbank and obtained the FASTA
	format of the sequences and entered that into the data sheet.

	--Stop 4:00 PM--

Stan Piotrowski

	Gaither et al. 2011: High Connectivity in the Deepwater Snapper Pristipomoides filamentosus (Lutjanidae) across the Indo-Pacific with Isolation of the Hawaiian Archipelago
		-- online access article, just used the page numbers listed on the pdf on google drive 
		As of March 13, 1:00am… finished data entry for Hawaii, do Indo-Pacific later 

3/15/2015
Ash Bhattacharjee

	-- 9:00 PM
	Changing/Fixing previous data entries based on feedback

	Ahonen et al: Based on notes, only mtDNA has been entered (data from Table 2). There are definitely 
	samples from more than 2 sites, however there is no way to tell if each location can be specified
	within 3 degrees as the coordinates are not given. My data entry for lat long is based on the map
	provided in Fig. 1 and google maps. Is it okay that this paper was used?
	*Will enter sequence if you decide the sites are specific enough. 

	Correia et al: Paper was previously disqualified since I thought they were captive populations.
	I used the species name Conger conger, as per Fishbase. This paper has now been included into the 
	data entry. 
	- Understanding the marker name was difficult, because the paper said that the mitochondrial control 
	regions did not amplify as expected and "New modified primers based on L15774 (Kocher et al. 1989) 
	(CCTGAAGTAGGAACCAGATGTCAGTA) and H16498 (Kocher et al. 1989) (ACATGAATTGGAGGAATACCCGTTG) were used
	to amplify a fragment of c. 700 base pairs (bp)." A quick search of L15774 and H16498 lead me to 
	believe that these are the cytb and control regions (so I input cytb/control region. Is this 
	correct?
	- I found allele sequences on Genbank with the same author, so I used this for the data entry.
	*Correction - based on feedback I did not input FASTA sequence since I have diversity data (he & pi)
	! saved time.

	Andre et al: 
	- Would not let me input negative sign for longitude which is -0.## 
	*This might be a problem for other papers if there is a 0 degree longitude or latitude that is either
	in the Southern or Western hemisphere. 
	While reading through the paper, I found the Appendix 1a by accessing it through the website given.
	However the materials & methods section states "MtDNA variation was examined by analysis
	of restriction fragment length polymorphism in the mitochondrial NADH dehydrogenase ND3/4 subunit."
	So even though He values are available for the mtDNA in the appendix, I did not use this paper as it
	used RFLPs.

	Antoni et al: 
	- Changed the collection dates to 2008.5 and used the given map (Fig 1.) to find lat & long.
	- Referred to Table 1 which provides allele number but based on email instructions I didn't 
	input allele number because He and pi were already provided - saving time. 

	Bariche & Bernardi 2009:
	- Only report avg diversity across wide areas (>3 deg). Only Mediterranean is valid leaving one site.
	** I removed the data from this paper from my excel sheet. 

	Bay & Caley, 2011: 
	- I'm not familiar with how to use Data Thief, to pull the data off Figure 4. I downloaded the app
	from the site, but I'm not sure how to go about it. If you have some time to explain it, I could
	learn how to do that and input the data for this paper if it is available. Lat & long is available
	in 2.1.

	Betancur et al, 2011:
	Study uses data from invaded range, as per instructions I've deleted the data from the excel 
	sheet.

	Castillo-Olguín et al, 2012
	- Averaged collection date 2001-2003= 2002
	- Sample sizes were unclear, I used the data on Table 2 for sample sizes
	- Table 1 included diversity data (He & pi)

	** Went back and changed all the source data entries to match format.

	-- Stop 12:14 AM--

3/19/2015
Ash Bhattacharjee

	-- 4:00 PM
	Cawthorn et al, 2011
	- There is no diversity data provided for this paper *including the supporting material available
	online
	- Samples sizes for many species were less than 4 (many different species were sampled)
	- Did not input data for this paper

	Chen et al, 2010
	- Samples were taken in Eastern Taiwan twice, so I input the most recent sampling as per instructions
	- The locations/ coordinates of sampling are somewhat unclear and not labeled distinctly in the paper.
	I used figure 1 as a way to estimate the sampling sites.

	Cheng et al, 2011
	- No collection date was given, so I left that blank in my data entry

	Coscia et al, 2012
	- This paper does not provide coordinates for their sampling locations (not even in a figure for 
	estimation). I used google map to find coordinates, but its rather vague since the sample sites are
	countries: France, Ireland etc. There is no way to be sure the diversity data isn't averaged over
	> 3 degrees lat/long. Should I keep this data entry?

	-- Stop 5:00 PM --

3/24/2015
Ash Bhattacharjee

	-- 9:15 PM
	Coulson et al, 2011
	- This paper is different from previous ones I have read because they have looked at multiple 
	Skate species (some have been derived from other papers).
	- Certain species had altered diversity data after misassigned samples were corrected (I input the 
	corrected data)
	- I did not input data for species in which sample size was less than 4 
	- The round ray has an h=0 and it says n/a after that. I input 0 for h, but not sure why this was.
	- Also the aleutian skate & roughtail skate.
	- The paper does not provide sampling locations, so it is possible that the diversity data has been 
	averaged over more than 3 deg lat/long, but I'm not sure. There is a tree base ID file in the data
	accessibility section of the paper however I cannot open this for some reason, so not sure if any
	info is there. No other info was available in the other supporting material files that I found online.
	- I also went to the other two papers from which some of the species were compiled, but those papers 
	didn't have any other info on sampling location & collection date. So far these boxes are left blank.
	- This paper took a really long time because there were multiple species and lots of backtracking of 
	different papers and sources involved.
	*I have included this paper in the data entry as the diversity indices were available. Should I keep
	it, given the missing info?

	-- Stop 10:23 PM --

3/26/2015
Ash Bhattacharjee

	-- 12:30 PM
	Craig et al. 2011
	- All diversity information was available to input
	- The sampling locations/ coordinates were given on a map but some points were so close, it was hard 
	eyeball an exact coordinate. 
	- Collection date was not given.
	-- Stop 1:00 PM --

Stan Piotrowski

	Gaither et al. 2011: High Connectivity in the Deepwater Snapper Pristipomoides filamentosus (Lutjanidae) across the Indo-Pacific with Isolation of the Hawaiian Archipelago
	- March 26… working on Indo-Pacific 
	-- nothing out of the ordinary, just a lot of looking at Google Maps to find the lat and lon for all locations 

	Gaither et al. 2011: Phylogeography of the reef fish Cephalopholis argus (Epinephelidae) indicates Pleistocene isolation across the indo-pacific barrier with contemporary overlap in the coral triangle 
		-- open access article, used page numbers from the pdf, just as the previous Gaither et al. 2011 article in PloS ONE 
		-- sample locations listed as “Moorea (MR)” , do I list the letters in parentheses as well? 
		-- other than that, no oddballs in entering data

	Gaither et al. 2012: Coming out of the starting blocks: extended lag time rearranges genetic diversity in introduced marine fishes of Hawai’i
		Genetic diversity of three species introduced to the Hawaiian islands- 
		- Lutjanus fulvus
		- Cephalopholis argus 
		- Lutjanus kasmira (careful entering data for these, as the authors used previous data already entered in the sheet) 
		For L. fulvus… 
		-- didn’t enter data from source locations, as they were identical to those in Gaither et al. 2011 (already entered in the sheet) 
		-- I entered data for the Hawaiian islands, not already entered in previous entries 

		For C. argus…
		-- source population data already entered from previous paper 
		-- data from Hawaii not entered previously-- enter that for this paper 

		For L. kasmira… 
		-- no data entered from source populations, as they were already entered for Gaither et al. 2011 (society islands, etc.)
		-- entered data from the subset of of the L. kasmira specimens already entered for the Hawaiian islands, but it was a different locus (control region vs. cyt b in current paper)
		-- 2 markers listed, but only two sampling locations for cyt b 
			-- data from other marker (control region) already entered from previous paper

	Garcia et al. 2011: Population structure of Atlantic swordfish… 
		-- no data entered for this paper, authors used RFLP analysis for mtDNA loci 

	Garcia-Rodriguez et al. 2011: population structure of Pacific sardine 
		-- Authors used three different sampling zones based off of limits of SST
		1) CEN-- sardines from Ensenada w/ cold water
		2) TBM-- sardines from Bahia Magdalena w/ temperate water
		3) WBM-- sardines from Bahia Magdalena w/ warm water 
		-- averaged latitudes, all within 3° 
		-- Multiple markers reported from various references were used in genetic diversity… vague about which ones are used in Table 4, but it looks like it was the 500 bp segment of the control region.

Sarah Picon

	Bagley and Geller 1998. Characterization of microsatellite loci in the vermilion snapper Rhomboplites aurorubens (Percoidei: Lutjanidae)
	33.546707, -79.043071
		how many numbers do I round up to for lat and long?
		rounded to 4: 33.5467, -79.0431
		Hese → only if it is averaged, right? In this paper it is not. Double check with Malin.
		Pi → not there; or couldn’t find it
		data entered.

	Bagley et al. 1999. Microsatellite variation, effective population size, and population genetic structure of vermilion snapper, Rhomboplites aurorubens off the southeastern USA
		not sure if I should enter it because in Table 3 the asterisk (allelic frequencies significantly out of Hardy-Weinberg equilibrium) is present on a few of the markers.
		Should I enter the markers that have no asterisk?
		● This gets a little tricky, so good to ask. Our goal is to only enter loci that are in Hardy-Weinberg equilibrium across all populations. The asterisks in the table show that certain loci in certain populations are out of equilibrium, but that could just happen by chance. What I'm more concerned about, actually, is the statement in the Results that, "The value of f was positive for all seven loci and was significantly greater than zero at three loci (Ra1, Ra5, Ra6) based on permutation testing (0.01 < p < 0.05)." This means out of HWE across all populations. I would enter data for all loci and all populations, EXCEPT Ra1, Ra5, and Ra6.
		Data entered (except Ra1, Ra5, and Ra6)

	Batargias et al. 1999. Characterization of six polymorphic microsatellite markers in gilthead seabream, Sparus aurata
		Can’t find when it was collected or where it the individuals were collected. The paper mentions the Mediterranean Sea and Adriatic Sea. Is that too broad?
		The paper also mentions using microsatellite DNA to facilitate breeding experiments in aquaculture.
		● The paper mentions these locations as where the fish in general lives, but never says where they sampled the specific fish that were genotyped. That's not good. We can't use this paper.
		Not entered.

	Beheregaray&Sunnucks. 2001. Fine-scale genetic structure, estuarine colonization and incipient speciation in the marine silverside fish Odontesthes argentinensis.
		I couldn’t find the common name in english on fishbase. however, the paper calls it “silverside fish”.
		Some locations that were sampled were estuarine, the others were marine. Do I only include the marine individuals?
		● Note that this paper is actually Beheregaray & Sunnucks 2001. Using "silverside" as the common name is fine. You did the right thing by only entering the marine samples.
		Fixed citation.
		Data entered.

	Bailey et al. 1999. Population Ecology and Structural Dynamics of Walleye Pollock (Theragra chalcogramma)
		Paper is mainly what the title says. Table 4 mentions mtDNA sequencing however the information needed is not provided.
		Should I find the paper that is referenced (Shields and Gust 1995) and enter that information?
		● It looks like Shields & Gust 1995 is a mtDNA paper, but you're focusing on microsatellites. You could get Powers 1996, since it looks like it might have msat data.
		Powers 1996 is an “(unpubl. report)”. Couldn’t find it online.
		Data not entered.

	Bonhomme et al. 2002. Analyse comparee de la structure genetique de deux especes de poissons marins apparentees et sympatriques Dicentrarchus labrax et Dicentrarchus punctatus
		This paper is in french. I can kind of make out the information that is needed but I am not 100% confident.
		Not entered.

	Chiu et al. 2002. Polymorphic microsatellite markers for stock identification in Japanese anchovy (Engraulis japonica)
		The location of where the fish were sampled is very vague: “collected from waters near Taiwan”.
		“The observed genotypes were deviated from Hardy-Weinberg expectation at five out of six loci (P<0.05) resulting from heterozygote diversity” Does this also mean that it is out of HWE?
		Data not entered.


	Bahri-Sfar et al. 2000. Fragmentation of sea bass populations in the western and eastern Mediterranean as revealed by microsatellite polymorphism.
		Did not enter the first or second generation bred in captivity. (Table 1)
		Should I enter the samples from the coastal/continental lagoon habitat or stick with the sea habitat? (Table 1.)
		● Careful about Bhari-Sfar et al. 2000. Samples from Valencia, Sete, Annaba, and Marsala should not be entered since the data are from other studies you will enter (Naciri et al.1999, Garcia de Leon 1997). 
		● In Bhari-Sfar, did you enter coastal/lagoon populations? I can't tell from your notes.
		Samples from Valencia, Sete, Annaba, and Marsala were removed.
		Data entered (from sea locations)

	Buonaccorsi&Graves. 2000. Isolation and characterization of novel polymorphic tetra-nucleotide microsatellite markers from the blue marlin, Makaira nigricans.
		The sampling site is very, very vague: “Pacific Ocean”.
		Data not entered.

	Buonaccorsi et al. 2001. Reconciling patterns of inter-ocean molecular variance from four classes of molecular markers in blue marlin (Makaira nigricans).
		The sampling site is vague: “Pacific and Atlantic Ocean”
		Data not entered.

	Coughlan et al. 2000. Isolation and characterization of 11 microsatellite loci in Atlantic halibut (Hippoglossus hippoglossus L.)
		The sampling site is vague: “Iceland” (found in Table 1.)
		Data not entered.

	DeWoody&Avise. 2000. Microsatellite variation in marine, freshwater and anadromous fishes compared with other animals.
		The paper seems to be a meta-analysis. It does not mention any marine by name.
		Data not entered.

	Gaffney. 2000. Molecular tools for understanding population structure in Antarctic species.
		The paper goes over molecular tools. There is no He for any fish species mentioned.
		Data not entered.

	De Innocentiis et al. 2001. Allozyme and microsatellite loci provide discordant estimates of population differentiation in the endangered dusky grouper (Epinephelus marginatus) within the Mediterranean Sea.
		Some of the areas sampled are greater than 3 degrees latitude or longitude. I used the centroid location as the lat and long coordinates. Is that fine?
		For Table 2 should I only enter data for those which did not have ☨P values (“representing significant HWE departures)
		● In De Innocentiis et al. 2001, please only exclude locus GAG023. Reading the paper, you'll notice that this is what the authors did, so we'll do the same. You can enter data from the other loci and populations. 
		● In De Innocentiis et al. 2001, you note that there are some regions that span >3degrees. You shouldn't enter data from those specific regions.
		GAG023 was already excluded. Removed areas sampled that are >3 degrees.
		NW and NA locations are larger than 3 degrees. Data from these locations were removed. 
		Data entered.

	Feldheim et al. 2001. Population genetic structure of the lemon shark (Negaprion brevirostris) in the western Atlantic: DNA microsatellite variation.
		Confused about entering the He data for each of the 4 locations (Table 2). Is the He an average across all 4 sites?
		● The data in Table 2 of Feldheim et al. 2001. Molecular Ecology 10:295-303 appears to be averaged across all four sites. If those sites are within 3 degrees of each other, you can enter those data as one line per locus (so two lines total). You would NOT want to re-enter the same data on multiple lines, as it looks like you did in the spreadsheet. If I'm interpreting the paper correctly, though, those sites are spread quite far apart, and so we can't use the averaged data. If the paper reported He and pi by site, it would be OK, but I don't see those data anywhere.
		Data not entered.

	Feldheim et al. 2002. The breeding biology of lemon sharks at a tropical nursery lagoon.
		Table 2 includes the He of “five populations in the western Atlantic”. The location is too vague for those populations.
		Data not entered.

3/27/2015
Sarah Picon

	Reilly&Ward 1999. Microsatellite loci to determine population structure of the Patagonian toothfish Dissostichus eleginoides.
		The paper stated that they sampled from 2 sites 60km apart off Macquarie Island. Each degree of latitude or longitude is approximately 111km apart so the location is useable.
		I differentiated between both sites under site sampled but I used the same lat/long. point for each. Should I pick another lat/long. to differentiate between both sites?
		Did not enter cmrDe30 from site 2 because it was stated to deviate significantly from HWE.
		Data entered.

	Turner et al. 1998. Polymorphic microsatellite DNA markers in red drum (Sciaenops ocellatus)
		The paper only gives an estimate between 7-11 individuals used for each locus screening. Is using an estimate fine?
		● For Turner et al. 1998, an estimate is fine. I would use 9 (the average), and make a note in your logbook about the decision that you made.
		Used the estimate of 9 individuals sampled for each locus screening.
		Data entered.

3/28/2015
Sarah Picon

	Pardini et al. 2000. Isolation and characterization of dinucleotide microsatellite loci in the Great White Shark, Carcharodon carcharias.
		Were the 20 white shark and other elasmobranch species samples used for Table 2? (pg. 1177; top of first paragraph)
		When the paper says 5 out of 6 primers pairs tested were polymorphic, does it mean that the other one is not polymorphic (therefore monomorphic)? (pg. 1177; second paragraph)
		● It looks like the 20 white shark samples were used for Table 1, while the other species were used for Table 2.
		● Yes, the other must have been monomorphic (and so you wouldn't enter it).
		Data entered. 

	Sola et al. 1999. Genetic characterisation of Epinephelus marginatus through cytogenetic, allozyme and microsatellite analyses: preliminary results.
		No He.
		Data not entered.


3/29/2015
Stan Piotrowski

	Gleason & Burton 2012: ID of fish eggs
		-- DNA barcoding initiative to identify fish eggs and larvae down to the species level using mtDNA loci for fisheries science 
		-- one of the main goals is to minimize error when identifying different commercially important species in egg and larval surveys
		--Samples came from two sources-
			- 1998 and 1999 CalCOFI cruises 
			- 2005 cruise in the California Cowcod Conservation Area 
		No data entered for this paper, as there were no data for He or pi 

	Golani & Bernardi 2012: Differential invading potential among cryptic species of a Lessepsian bioinvader, the blotchfin dragonet Callionymus filamentosus 
		-- Lessepsian bioinvasion-- bioinvasion of the Mediterranean from the Red Sea via the Suez Canal 
		-- two mtDNA markers, the control region and CO1 
		For sampling location…. 
		-- samples collected by trawl and beach seine from 1995-2008, but location isn’t very precise.  Only lists Eilat, Israel and Jaffa, Israel for sampling locations.  Pick a random point off these places, or disqualify the paper?  
		-- for the markers entered, I just used two separate lines for each marker since data were given for both 
		check with Dr. Pinsky about sampling locations before entering more data. 
		Data for each of the two markers appear to be mixed between two locations (see phylogenetic trees)
			Probably don’t want to enter data 
		Clade data 
		-- Red Sea Clade, exclusively individuals from Red Sea (Eilat, Israel) = 
			-- 35 for the D-loop
			-- 57 for the control-region 
		--Lessepsian clade-- 
			-- From Red Sea-- 
				-- 9 for the D-loop
				-- 11 for control-region 
			-- From Mediterranean
				-- 46 for the D-loop
				-- 56 for control-region 
		No data entered for this paper-- one population was from an invasion (don’t count it) and the other population was only from one site… not enough to enter data.  

	Gold et al. 2010: population structure of carite in waters offshore of Trinidad and northern Venezuela 
	-- everything normal for entering data, except no standard error data for haplotype or nucleotide diversity 


3/30/2015
Ash Bhattacharjee

	-- 1:00 PM
	Cui et al. 2010
	- It looks like there are a couple things about this paper that make it unable to be used for this 
	project
	- It looks like the sample size is less than 4 for each species: "One T. japonicus (Tj-s) was 
	collected in South China Sea and two individuals (Tj-y1, Tj-y2) were from the Yellow Sea. For Pampus 
	sp. specimens, three individuals (Pa1-3) were collected in the Yellow Sea."
	- Seems like they only looked at the tandem repeats from the control regions, because that's what is
	presented in Table 1 (very few bp, since it's just the tandem repeats). 
	- There is no diversity data
	*I did not input this paper in the excel sheet

	Cunha et al. 2012
	- In the methods section it says that they used 2 mtDNA markers, but Table 1 which has the diversity 
	data says it is present mt CR sequence data. So I have input 1 for number of markers: control region
	- Used Figure 1 to estimate coordinates of sampling sites, based on map given. 
	- Anton Dohrn Bank was not on the map, so I used google maps for this site and still couldn't find it.
	I haven't included this in my excel sheet.
	- There was no diversity data for samples taken at Rosemary Bank, so this is not input to my excel 
	sheet
	- Great Meteor Bank has 0 h and nucleotide diversity, what does this mean? Should samples like this 
	be included in the data entry?
	- Collection date is not given anywhere in the paper. I used the accession numbers to trace it back to
	GenBank, but there was no collection data there either. Left this box empty. 
	- Source: Cunha et al. 2012 PLoS ONE 7:1-13. Is this the correct citation?

	Stop -- 2:00 PM 

Chloe Lewis

	Karl et al. 2012: 
		→ the values for the lat/log are not explicitly provided in the paper, however a map of the sampled sites is provided; accurate estimates were made using Google Maps. 
		→ of the sampled sites, only the following locations are included in the data results: BRI, NTL, BMN, FLD, BLZ, and Western Atlantic; note that the location labeled “Western Atlantic” refers to the overall genetic diversity was calculated for each of the samples collected in the western Atlantic, all western Atlantic samples combined, and overall including the African and Pacific samples.
		→ BRI refers to the combined sample sites of ADR and FNR, which are less than 3 miles apart.
		→ two mtDNA markers were sampled: control region and cytochrome oxidase I
		→ only the total base pairs are given; the paper does not list how many were from each individual sample.

	Kaoueche et al. 2013: 
		→ none of the coordinates are given, accurate estimates were made using google maps.
		→ the paper does not discuss collection years 
		→ only the total base pairs are given; the paper does not list how many were from each individual sample.

	Kitamura et al. 2010: 
		→ no data entered
		→ the paper does not meet the 3x3 degree criteria for maximum latitude/longitude average range; the pi and h values are averaged across sample site location coordinates that are more than 3 degrees apart lat/long.
		→ collection years are given in ranges so times were averaged 
		→ 8 sites of the North Atlantic defined 4 haplotypes (h=1) with a pi diversity of 0.01479 and that the Southern Atlantic had 13 sites defining 17 haplotypes (h=.816 and pi=.00893).

	Le Port and Lavery. 2012: 
		→ for the following sample locations coordinates are not given, accurate estimates were made using Google Maps: Victoria, Tasmania, Western Australia, Eastern Cape, and Western Cape (also the coordinates were taken off the coast of these sites, as they are all on the continent). 
			→ google maps could not accurately estimate site location coordinates for: Gascoyne Seamount so all data from this site were excluded.
		→ collection years are given in ranges, so the times are averaged
		→ the h and pi values are assigned to 5 polymorphic sites; these sites are comprised of all 15 sample sites.
		→ the paper provides no information on collection years.

	Lee et al. 2012: 
		→ only the total bp value is given, the bp values are not given for individual samples.
		→ pi value not given, however h values are
		→ collection years are given in ranges, so the times are averaged

3/31/3015
Sarah Picon

	Smith&McVeagh 2000. Allozyme and microsatellite DNA markers of toothfish population structure in the Southern Ocean.
		For Table 1. I am not sure how the values for He were calculated. The numbers are >1 for all He values.
		● Table 1 is for allozymes (see the legend), and so you wouldn't enter it. It is also allele frequencies, not He values.
		Only To2, To3, and To5 will be entered. The other loci are from another study.
		To3 is noted to depart significantly from HWE but it “was not significant when adjusted for multiple tests”.
		● You want to use data from Table 3. In the notes below the table, they define Hets_exp as the "number of heterozygotes expected assuming Hardy–Weinberg equilibrium". That's not a typical definition, but it means that you need to divide Hets_exp by "N fish" to get the proportion. So for Prince E Is, He = Hets_exp/N fish = 40.24/43 = 0.9358
		Data entered.

	Perez-Enriquez&Taniguchi 1999. Use of microsatellite DNA as genetic tags for the assessment of a stock enhancement program of Red Sea Bream.
		This study is collected samples of “wildfish” from areas where the aquaculture center has released their raised fish.
		“277,000 and 439,000 juveniles...were released into the wild in different locations of Kochi Prefecture. For the present study, two samples from the wild population were obtained from a location in the middle part of the Prefecture”.
		● Correct not to enter this. This population has too much influence from a captive population.
		Data not entered.

	Stevenson et al. 1998. Stock identification in Nassau Grouper, Epinephelus striatus, using microsatellite DNA analysis.
		Double checking. Does x^2=He?
		● Wow, this is written like an old paper. I couldn't find He values in any of the tables, and x^2 values are not He. Given that they grouped loci together (last paragraph on p. 734), we won't use this paper.
		Data not entered.

	Smith et al. 1998. Polymorphic microsatellite DNA in toothfish Dissostichus eleginoides from the Southern Ocean.
		Am I only supposed to enter “A075”? If so, I should download the entire article?
		● That's a conference abstract, not a paper. We'll skip it.
		Data not entered.


4/1/2015
Chloe Lewis

	Kai et al. 2011: 
		→ no data entered
		→ the paper fails to report values for pi and h

	Kai et al.2012: 
		→ no data entered
		→ the paper gives no values for h and pi; just mentions that they were estimated for each of the sampled species.
		→ two separate species were sampled in this study; no specifics on the results beyond sample size and regional locations

	Karlsson et al. 2009:	
		→ lat/lon coordinates are not explicitly given, however they can be accurately estimated using the map given. 

	Katafuchi et al. 2011: → no data entered 
		→ the paper fails to report values for pi and hi
		→ fails to report data explicitly recording sample size values at each sampling site

	Kawakami et al. 2010: → no data entered
		→ the paper fails to report any information regarding  pi and h values 

	Keskin et al. 2009: →  no data entered
		→ the paper reports no information on pi or h values
				
	Keskin et al. 2012: → no collection year info
		→ only total base pair amount for the study is given [395 (12S total), 298 (cytb total), 540 (COII total)]
		→ no pi values
		

4/2/2015
Chloe Lewis

	Durand et al. 2005: 
		→ the coordinates are presented as ranges, so the averages have been taken and reported into the spreadsheet
		→ there are neither coordinates reported for the following sample sites: Cape-2,Cape-3,Cape-4, Cape-5, IBE, and Mad, nor is there a map of the site locations offered in the paper; therefore data from these sites are not included in the spreadsheet. 
		→ the collection year for S is reported as a range, so the average value was rounded and recorded in the spreadsheet. 
		→ the base pair values are not reported in the paper

4/3/2015
Sarah Picon

	Perez-Enriquez et al. 1999. Genetic variability and pedigree tracing of a hatchery-reared stock of red sea bream (Pagrus major) used for stock enhancement, based on microsatellite DNA markers.
		Individuals used are the broodstock and hatchery-reared stock.
		Data not entered.

	Ward&Reilly 2001. Development of microsatellite loci for population studies of the pink ling, Genypterus blacodes (Teleostei: Ophidiidae).
		Are the locations sampled at too vague?
		There is no map specifying the areas sampled like the map in Ward et al. 2001.
		● Some of the locations do seem too vague (New South Wales is ~10degrees tall). EVIC and WVIC may be small enough. You can check by clicking the upper and lower bounds of the region they describe to see if it is >3degrees. Same for EW, though in this case, you'll just have to assume it's not too far from coast.
		After checking the upper and lower bounds of the regions, they are all over 3 degrees in latitude or longitude.
		Data not entered.

	Ward&Elliot 2001. Genetic population structure of species in the South East Fishery of Australia.
		The paper uses data from several studies. Should I look at the specific studies for microsatellites?
		● No need to pull the underlying studies. No need to enter this paper.
		Data not entered.

	Ward et al. 2001. Population structure of pink ling (Genypterus blacodes) from south-eastern Australian waters, inferred from allozyme and microsatellite analyses.
		Not sure how to enter the data for microsatellites in Table 4. Which location do I use?
		● They are, in effect, averaging across all of their sample areas, which is >3degrees. So you can't enter these data.
		Data not entered.


4/4/2015
Chloe Lewis

	Kim et al. 2012: 
		→ no data entered
		→ the paper mentions “genetic divergence” states that China Z had the highest value followed by China D, and then Korea. 
		→ there is no h or pi value provided by the paper for site location “China C”; this site is excluded from the excel spreadsheet.
		→ no information provided for h or pi values
		→ no collection year info provided
		→ only the total base pair amount is provided; not by site location
		→ The paper doesn't seem to define what they mean by intra-population genetic divergence

4/5/2015
Ash Bhattacharjee

	-- 7:00 PM
	Cuveliers et al. 2012
	- There are 5 sample sites at which juvelines were used (not sure if this makes a difference?). 
	- Two sites had less that 4 samples (Bel 08 & BISA), so I did not input data for these two
	- Table 1 provided all location data & names of sites
	- Table 3 contained all molecular diversity indices 
	- What would be the country name for Kattegat strait, Scheldt estuary & Wadden sea? I thought it 
	would be Atlantic Ocean which is the nearest ocean..

	Daley et al. 2012
	- Coordinates estimated using map given (Figure 1)
	- No collection date is given
	- The paper doesn't give expected heterozygosity (couldn't enter this box)

	Daley-Engel et al. 2010
	- I think this paper is averaging the diversity data over large regions, because they only give one
	value for the he and pi for Squalus mitsukurii (from this paper) but many sample sites are recorded.
	- I haven't used this paper for entry, is this correct?

	Daly-Engel et al 2012 a
	- No collection dates provided
	- No coordinates for sampling sites, so I used google maps to find this. Could be greater than 3 deg
	lat or long? 

	Daly-Engel et al 2012 b
	- Only diversity data on microsattelite, and only provides Fst values for mtDNA 
	- Couldn't use this paper because it doesn't have any he or pi for mtDNA 

	Stop --

Chloe Lewis

	Kim et al. 2010 (b): 
		→ the paper does not specify the  collection years from which the samples were taken
		→ only the total value of base pairs are given, they are not provided by site location.
		→ one locality (Gangneung) is technically located in the Korean Peninsula . This is the one sample site that the paper states was sampled in the Republic of Korea, while the other six sample sites were taken in Hokkaido, Japan. 

	Kodama et al. 2009: 
		→ no data entered
		→ no information provided for pi or h values 
		→ no map or coordinates given for sampled sites 		      

	Kon et al. 2011: 
		→ no data entered
		→ no information is provided for pi or h values
		→ site location info gives only the coordinates of Koror Island in Palau, western Pacific Ocean, but does not specify the coordinates of the 3 localities.

	Kumar et al. 2012(Auxis thazard): 
		→ all required and preferred data is recorded from this paper. 
		→  the bp value provided only gives the value of total bp, not by individual samples collected.

	Kumar et al. 2012(Euthynnus affinis):
		→ no pi values are given, however the paper does provide h values 
		→ the bp value provided only gives the value of total bp, not by individual samples collected.
		→ the paper does not explicitly say that the site locations are found in the country of India, but that they all were collected from the Indian Ocean. Therefore, under “country” column, “indian ocean” is recorded. 

	Levy-Hartman et al. 2011: 
		→ only total bp amount is given, values are not provided for individual samples.
		→ no coordinates are provided for first 7 sites (these values were found accurately  via google maps) with help from the map provided in the paper (figure 1).
		→ the pooled values from the last row of table 1 are excluded from the excel spreadsheet.


4/6/2015
Chloe Lewis

	Kim et al. 2010 (a): 
		→ no data entered
		→ the paper provides no information on pi or h values
		→ though there is a map (vague), there are no coordinates given for sample site locations.

	Kriwet et al. 2010: 
		→ no data entered
		→ the paper provides no information/values for h or pi.
		→ a map is offered as a reference, but no information is given for sample site location coordinates.

	Kwun et al. 2011: 
		→ no data entered
		→ the paper provides no values for h or pi
		→ coordinates for sample site locations are not provided

	Lakra et al. 2011: 
		→ no data entered
		→ no information for h or pi are provided
		→ there is no information on collection years and the paper offers no values for sample site coordinates; there is no map to reference.

Sarah Picon

	Shaw et al. 1999. Microsatellite DNA analysis of population structure in Atlantic herring (Clupea harengus), with direct comparison to allozyme and mtDNA RFLP analyses.
		Did not enter data from Pacific samples because the location exceeded 3 degrees longitude.
		Data entered (except for Pacific samples)

4/7/2015
Chloe Lewis

	Leray et al. 2009: 
		→clade 1 signifies the Indian Ocean, clade 2 signifies the Pacific Rim (predominantly) and South Polynesia, clade 3 signifies South Polynesia, clade 4 signifies Marquesas, and clade 5 signifies Hawaii & Johnston.
		→ no data entered
		→ it appears that each clade is a separate species (note discussion of speciation in title and abstract, plus text throughout). We only use comparisons of diversity within species, and so we can't use this paper

Sarah Picon

	Ruzzante 1998. A comparison of several measures of genetic distance and population structure with microsatellite data: bias and sampling variance.
		No information on He.
		Data not entered.

	Ruzzante et al. 2001. Stability in the historical pattern of genetic structure of Newfoundland cod (Gadus morhua) despite the catastrophic decline in population size from 1964 to 1994.
		Is it okay to use the median lat, and long, given in the paper (Table 1.)?
		●	Yes, OK to use these lat/lon values.
		Data entered.

4/9/2015
Ash Bhattacharjee

	-- 12:30 PM
	Dammannagoa et al 2011
	- They used two portions of the mtDNA: ATPase subunit 6 & 8, so I wrote it like this:
	ATPase 6, 8 

	Diaz-James et al 2010
	- "The results recorded
	by these studies, which include the entire distribution range of
	these fish throughout the oceans that lie between the latitudes of
	30–40N and 30–40S"... Based on this and Figure 4, the diversity indices were averaged over more than 
	3 degrees lat & long.
	*DId not include this paper in the data entry

	Dibattista et al. 2011
	- Sampling site coordinates obtained through estimating from Figure one and checking with google maps
	for more detailed coordinates 
	- Pacific ocean sites are not labeled on the map with coordinates, so I used google maps for those 
	sites

	Dibattista et al. 2012a
	- No diversity data included in this paper, so I couldn't enter it into the excel sheet

Stan Piotrowski

	Gold et al. 2011: Phylogenetic relationships of tropical Western Atlantic snappers in subfamily Lutjaninae inferred from mtDNA sequences 
		Red flag: “sequences were obtained from three individuals of each species.” the instructions explicitly say not to include data from papers with a sample size of less than 4 individuals…
		--although they sampled 12 different species, only 3 individuals per species were used for sequencing.. check with Dr. Pinksy

	Gold et al. 2011: Genetic Divergence and Effective Size among Lane Snapper in U.S. Waters of the Western Atlantic Ocean 
		-- authors did NOT use RFLP, used sequencing 
		mtDNA locus used was NADH-dehydrogenase subunit 4 (ND-4)
		*check with Dr. Pinsky if hypotype (nucleon) diversity is the haplotype diversity, or not 

	Gonazles-Wanguemert et al. 2011: Phylogeographical history of the white seabream Diplodus sargus (Sparidae): Implications for insularity 
		-- sampled from four island localities and five continental localities 
		--no RFLP sequencing, everything fine on that end 
		-- only one marker, the mtDNA control region (410 bp segment) 

	Gonzales-Wanguemert & Perez-Ruzafa 2012: In two waters: contemporary evolution of lagoonal and marine white seabream (Diplodus sargus) populations 
		-- sites sampled were A Coruna, Quarteria, Banyuls, and Murica (the latter of the two sites had already been sampled in the previous year) 
		-- as per the instructions, if multiple samples were from the same site, enter data with the highest sample size, and to break the tie, use the most recent sample if applicable 
		In 2011, Banyuls = 21, Murica = 20 individuals
		In 2012 (this study), Banyuls = 22, Murica = 25 
		-- Conclusion: enter data for THIS YEAR’s sample (i.e. 2012), not from 2011 
		-- deleted rows for Banyuls and Murica for the previous paper, entered it for this paper instead since the sample sizes were greater for this year 
		-- same thing for site Galicia for the previous paper… this current paper (2012) had 24 individuals from Galicia, the paper previous had 23..
		-- conclusion: use data from Galicia from 2012 paper 
		-- samples weren’t sequenced via RFLP, so everything good here 
			**** the sample sites listed as “Coastal Lagoon”... they still count as marine, yes? 

	Gonzalez-Wanguemert et al. 2010: Connectivity patterns inferred from the genetic structure of white seabream (Diplodus sargus L.) 
		-- same sites as Gonzalez-Wanguemert et al. 2011, but used cyt. b locus instead 
		-- are these data okay to enter? They’re from the same site but with a different locus 

	Gorbachev et al. 2011: Intraspecies structure of the Pacific Herring 
		No He or pi values listed, only common or variant haplotypes were reported.  For this reason, I didn’t enter any data for this paper 

	Aizdaicher & Markina 2011: Influence of changes in sea water salinity on the growth, photosynthetic pigment content, and cell size of the benthic alga… 
		-- Study species was a benthic diatom, not a fish.  For this reason, no data were entered for this paper 

	Gorbachev et al. 2012: Phylogeography of Pacific herring Clupea pallasii from Eurasian Seas 
		-- “mean” nucleotide diversity values reported…. is this okay?  The values seemed abnormally large compared to the nucleotide diversity values from other papers, or maybe it just means there is a high degree of polymorphism in the populations? 
		-- check with Dr. Pinsky to be sure, but I entered the data anyway, can always go back and take it out 
		-- First three sites were taken from Gorbachev et al. 2011 (Tatar Strait, Gulf of Patience, and Peter the Great Bay)
			-- the locations of these three sites were given in lat/lon degrees and minutes
			-- the resolution of the map they provided in the paper was only a schematic, wanted to be as accurate as possible so I went ahead and used the data provided in the paper for lat/lon locations instead of trying to find it with google maps 
		-- one of the locations was the Anadyr Estuary… this is technically marine, right? (entered data for this site, can take it out if necessary) 
		-- although two markers were reported, only the control region marker was used for the nucleotide diversity values
		-- CO1 values were used for the dendrograms (evolutionary relationships) 
		-- only nucleotide diversity values reported, no haplotype diversity values 

	Grant et al. 2010: Biogeographical contingency and the evolution of tropical anchovies (genus Cetengraulis) from temperate anchovies (genus Engraulis) 
		--although the authors had Haplotype and nucleotide diversity values, only the samples taken from the Western Atlantic were given a specific location
		--all the other sampling locations were given very roughly, all by the range of the species and arbitrary sampling locations (for example, southern range off California)
		-- for this reason, I chose to only include those data from the Western Atlantic from this paper 

	Grant et al. 2012: Limits of Bayesian skyline plot analysis of mtDNA sequences to infer historical demographies in Pacific herring (and other species)
		-- control region sequences used were much smaller than others in previous papers, but it was over 200bp so it’s okay 
		-- standard error not reported, so use formula in the instructions… standard deviation/ sqrt (N)
		Question: what’s the difference between theta pi and pi?  It doesn’t really distinguish between the two in the table, but the values are very different
			-- I put in the values from theta pi after digging around a bit in the paper 

	Griffiths et al. 2011: Levels of connectivity between longnose skate in the Mediterranean Sea and the north-eastern Atlantic Ocean 
		--stocks are genetically isolated, important for conservation of the skate 
		Problems with sampling locations-- 
		-- Two individuals caught on Rockall Bank and 12 along the coast of Norway… no specified locations for those caught off the coast of Norway (these constitute those sampled in the Northeast Atlantic, which are all listed together in the table) 
		-- Because these data were averaged over more than 3° latitude, I did NOT enter data for skates caught in the northeast Atlantic 
		-- For those individuals from the Mediterranean… they originated in waters off Mallorca and Menorca, two islands off Spain… but they aren’t very specific.  Do I include data for these areas by looking at an average distance between the two islands? 
		- check with Dr. Pinsky to be sure 
		Gubili et al. 2011: Antipodean white sharks on a Mediterranean walkabout? 
		-- collections of samples… 
			-- 2 from the eastern Mediterranean basin, Turkey
			-- 1 from Tunisia 
			-- another off Sicily 
		These sampling locations were way too spread out to be average together, so I didn’t enter any data for this particular paper 

Sarah Picon

	Garcia de Leon et al. 1997. Microsatellite polymorphism and population subdivision in natural populations of European sea bass Dicentrarchus labrax (Linnaeus, 1758).
		Data entered.

	Guillemaud et al. 2000. Interspecific Utility of Microsatellites in Fish: A Case Sudy of (CT)n and (GT)n Markers in the Shanny Lipophrys pholis (Pisces: Blenniidae) and Their Use in Other Blennioidei.
		He was averaged across 3 sampling sites. The sampling sites are over 3 degrees lat. and long.
		Data not entered.

4/10/2015
Sarah Picon

	Guillemaud et al. 2000. Microsatellite characterization in the rainbow wrasse Coris julis (Pisces: Labridae)
		Data entered.

	Heist&Gold 1999. Microsatellite DNA variation in Sandbar Sharks (Carcharhinus plumbeus) from the Gulf of Mexico.
		He was averaged across site from Chesapeake Bay and Gulf of Mexico. Both sites are over 3 degrees lat. and long. from each other.
		Data not entered.

	Heist&Gold 2000. DNA microsatellite loci and genetic structure of red snapper in the Gulf of Mexico.
		He was averaged across all 4 sites. Sites are over 3 degrees lat. and long. of each other.
		Data not entered.

	Herbinger et al. 1997. Family relationships and effective population size in a natural cohort of Atlantic cod (Gadus morhua) larvae.
		No He information.
		Data not entered.

	Jones&Avise 1997. Microsatellite analysis of maternity and the mating system in the Gulf pipefish Syngnathus scovelli, a species with male pregnancy and sex-role reversal.
		Data entered.

	Jones et al. 1998. A microsatellite assessment of sneaked fertilizations and egg thievery in the fifteenspine stickleback.
		Data entered.

4/13/2015
Chloe Lewis

	Li et al. 2011: 
		→ the paper does not offer any information on base pair values for the samples collected.
		→ the sample site Qingdao is located in the Yellow Sea and the sample site Zhoushan is located in the East China Sea 
		→ the map provided by the paper offers site location data on overwintering grounds and spawning grounds of the Japanese flounder 
		→ the H values reported in table 2 are Nei’s genetic diversity values and are not equivalent to to the h values (haplotype diversity) of the populations for each sampled site, which are reported under the results section on page 4; the h values were used in the spreadsheet.
		→ the coordinate values are explicitly provided by the paper along with a detailed map.
		
	Limborg et al. 2012: 
		→ only 4 of the 7 sample sites were typed for only mtDNA markers; 3 of the sample sites were also used to type for microsatellite markers
		→ the paper provides no information on base pair values for the samples collected. 
		→ the coordinate values are explicitly provided by the paper along with a detailed map.
		→ the paper offers no pi values, however it does provided h values 

4/14/2015
Ash Bhattacharjee

	-- 8:00 PM
	Dibattista et al. 2012b
	- Had to use google maps for coordinates because the figure provided didn't have any
	- Didn't include "Hawaiian Islands" because the range is too large to get a coordinate (probably 
	larger than 3 deg lat/long
	- Averaged year: 2008
	- Used fishbase to get the common name since paper just stated Indo-Pacific butterflyfish

	Drew & Barber 2012a
	- Only pi values available, no he(Table 1 only gives population parameter. Is that he?
	- Regardless, this paper cannot be used because the diversity indices were given for all individuals
	in all sampled locations. This means it has been averaged over a large space, more than 3 deg lat/long

	Drew & Barber 2012b
	- 2012a & b are the same paper

	Durand et al. 2012
	- No diversity data available in this paper to input
	- Table 1 provides an extensive collection of accession numbers for museum species from other collectors
	over various years.

	Eble et al 2011a
	- None of the sampling site coordinates were given (no legend/map either)
	- Many of the sample sites have been entered in my excel before, for which I have looked up and/or
	estimated coordinates previously. I used these values for the previously seen sites.
	- Collection dates given in figure 1: "Collections in Hawaii were made in June/July 2005-06,
	and elsewhere from September 2006 to June 2009"

	-- Stop 10:15 PM--

4/14/2015
Sarah Picon

	Jones et al. 1998. Microsatellite evidence for monogamy and sex-biased recombination in the Western Australian seahorse Hippocampus angustus.
		Data entered.

	Jones et al. 1999. The genetic mating system of a sex-role-reversal pipefish (Syngnathus typhle): a molecular inquiry.
		Data entered.

4/15/2015
Chloe Lewis

	Liu et al. 2010: 
		→ the sample sites were specified as being located around Taiwan; not within the boundaries of any particular country 
		→ the coordinates of the sample sites were accurately estimated using map provided in the paper 
		→ two of the sample sites did not meet the 3degree lat/long criteria, therefore they were excluded from the spreadsheet data record; these sites are as follows: TS, CW, and HN. 
		→ there are no base pair values given for each individual sample site location; the value is given for the overall section of mtDNA. 

	Liu et al. 2012:
		→ samples were collected in various countries throughout the western pacific ocean region
		→ the sample site BA (Bali, Indonesia) was immediately excluded due to the lack of h or pi values provided by the paper 
		→ the following sample sites did not meet the 3degree lat/long criteria, therefore they were excluded from the spreadsheet data record: GU, TA, AU, FI, 

	Liu et al. 2010: 
		→ the sample collection sites were taken throughout the northwestern region of the pacific ocean.
		→ the base pair value provided by the paper is for the overall samples; not individually.
		→ the coordinates were accurately estimated using the map given in the paper. 

Sarah Picon

	Jones et al. 2001. Surprising similarity of sneaking rates and genetic mating patterns in two populations of sand goby experiencing disparate sexual selection regimes.
		Data entered.

	Jones et al. 2001. How cuckoldry can decrease the opportunity for sexual selection: Data and theory from a genetic parentage analysis of the sand goby, Pomatoschistus minutus.
		Information from Table 1. was already entered in Jones et al. 2001. Surprising similarity…
		Data not entered.

4/16/2015
Sarah Picon

	Jorstad et al. 2001. Intermingling of Herring Stocks in the Barents Sea Area.
		No He information.
		Data not entered.

	Kvarnemo et al. 2000. Monogamous pair bonds and mate switching in the Western Australian seahorse Hippocampus subelongatus.
		No He information original to the study. He information used was from the Jones et al. 1998 Molecular Ecology study.
		Data not entered.

	Lemaire et al. 2000. Do discrepancies between microsatellite and allozyme variation reveal differential selection between sea and lagoon in the sea bass (Dicentrarchus labrax)?
		No He information.
		Data not entered.

	McDowell&Graves 2002. A genetic perspective on atlantic sailfish stock structure.
		Locations sampled are too vague and there is no He information.
		Data not entered.

	McGowan&Reith 1999. Polymorphic microsatellite markers for Atlantic halibut, Hippoglossus hippoglossus.
		Data entered.

4/17/2015
Sarah Picon

	McPherson et al. 2000. Microsatellite population structure in Herring at three spatial scales.
		Observed heterozygosity is only listed in Table 1. There is no information on expected heterozygosity.
		Data not entered.

	McPherson et al. 2001. Genetic diversity of coastal Northwest Atlantic herring populations: implications for management.
		Data entered.

4/19/2015
Ash Bhattacharjee

	-- 7:00 PM
	Eble et al 2011b
	- No collection date is given
	- The coordinates can be estimated from the map provided in Fig 1
	- Coordinates of longitude seem to not match up, Fig 1 says longitude is in East for Hawaiian Archi-
	pelago but google maps says West

	Lemair et al 2005
	- No coordinates given, and the names of each sample location is given in abbreviations however the 
	paper does give the full real name of any of the sites. How do I use google maps to get lat & long
	coordinates without the actual names of the sites? The figure given gives a general area, without
	an lat/long coordinates to refer to. Without coords or exact names, there is no way to tell how
	large the ranges of sampling and averaging are either.
	- All the diversity data is there

	Sotka et al 2005
	- No he values given
	- All other data provided (including coordinates)

	Teske et al. 2004
	- No diversity data of he or pi given in this paper: cannot include for data entry

	Stop -- 

4/20/2015
Chloe Lewis

	Lopez et al. 2010: 
		→ the bp value provided by the paper is assigned to all of the samples; not individual samples.
		→ the paper offers no coordinates or map location information for sample site Sin-04, therefore this site has been excluded from the data entry.
		→ the lat/long coordinates for sample site PE are outside of the 3 degree range, therefore this sample has been excluded from the data entry.
		→ the paper offers no information on collection years for the sample sites.

4/21/2015
Chloe Lewis

	Luzatto et al. 2012:
		→ no data has been entered from this paper
		→ the paper offers no information on pi or h values

	Mach et al. 2011: 
		→ 9 of the samples were collected in both 2005 and 2006, therefore the averages of these collection dates were taken and recorded.
		→ the sample size values of these same 9 sample sites consist of both “n” values of 2005 and 2006 respectively.
		→ the base pair value provided is not unique to each sample site; the value provided is the overall value.
		→ the coordinates were explicitly provided by the paper 

4/23/2015
Chloe Lewis

	Machado-Schiaffino et al. 2010: 
		→ the samples were collected in 2001,2002,2004, and 2005, therefore the average of these years was taken and recorded.
		→ 2 species were sampled for in this paper; both being North American hakes.
		→ the paper offers no information on base pair value.
		→ the coordinates were explicitly given in the paper along with a reference map.

Sarah Picon

	McPherson et al. 2001. Isolation of nine novel tetranucleotide microsatellites in Atlantic herring (Clupea harengus).
		Paper uses data from a different study (A. McPherson, unpublished).
		●	The data from this paper can (and should) be entered. The citation to the unpublished report doesn't count since it was never published. It also seems to only refer to an initial test for variability, not the more extensive test shown in Table 1.
		Data entered except for the data from the Celtic Sea (area is too broad).

	Mendonca et al. 2001. Isolation and characterization of four microsatellite loci in the tidewater goby (eucyclogobius newberryi).
		No He data given. Only the observed heterozygosity is given (Table 2).
		Data not entered.

	Miller et al. 2000. Development of tri- and tetranucleotide repeat microsatellite loci in Atlantic cod (Gadus morhua).
		Sampled area of “inshore and offshore regions of Newfoundland and Labrador” exceed 3 degrees lat. and long.
		Data not entered.

	Naciri et al. 1999. Genetic study of the Atlantic/Mediterranean transition in sea bass (Dicentrarchus labrax).
		Data entered.

	Nielsen et al. 2001. Population of origin of Atlantic cod.
		Locations sampled at are too broad: North Sea, Baltic Sea, and Northeastern Arctic Ocean.
		Data not entered.

	Nugruho&Taniguchi 1999. Isolation of greater amberjack microsatellite DNA and their application as genetic marker to species of genus seriola from Japan.
		Data entered.

	Nugruho et al. 2001. Genetic divergence of kingfish from Japan, Australia and New Zealand inferred by microsatellite DNA and mitochondrial DNA control region markers.
		Data for J-1 (Sdn-06 and Sdn-09) and A-2 (Sdn-03 and Sdn-09) were entered.
		If the lat. and long. measurements is slightly over 3 degrees (about 0.3 degrees greater) can I still enter it (for the N-1 data)?
		●	Unfortunately, we can't use it if it is >3degrees.
		Data entered for J-1 and A-2. Data not entered for N-1 data.

4/24/2015
Chloe Lewis

	Marko et al. 2010:
		→ no data from this paper has been used.
		→ the site names are not explicitly given by the authors of this paper. 
		→ this paper does not meet the 3 degree lat/lon requirement.

4/27/2015
Stan Piotrowski

	Guillermo Ceballos et al. 2012: Phlogeography of the sub-Antarctic notothenioid fish Eleginops maclovinus: evidence of population expansion 
		-- nothing out of the ordinary, entered all data normally
		-- just looked on Googlemaps for the lat and lon as opposed to putting in minutes and seconds (instructions preferred lat lon degrees)

	Gunawickrama et al. 2010: preliminary evidence for genetic heterogeneity of the goby S. bibarbatus in the Benguela ecosystem
		--authors used RFLP analysis, not sequencing 
		-- for this reason, I did not enter data for this paper 

	Gwak & Nakayama 2011: Pacific cod genetic variation and population structure 
		--authors used sequencing kit, not RFLP analysis, so everything is good there 
		-- authors did not distinguish between standard deviation or standard error.. ambiguous, so I left those values out of the data sheet
		-- inputted the haplotype and nucleotide diversity values normally 

	Habib et al. 2011: greenling Hexagrammos otakii
		-- DNA sequenced normally, no RFLP analysis 
		-- Three different mtDNA markers used → COI, COIII-ND3-ND4L, and cytochrome b
		-- Five locations were presented, although data from the locations in western Japan were pooled 
		-- these pooled data were some sites greater than 3° latitude apart, so I didn’t include data from those particular sites in the sheet 
		--The other three sites from Korea were all entered normally, but no data for standard error was entered as it was ambiguous whether or not the values were standard deviation or standard error 
		-- common name: no official one listed on fish base, but the authors used “fat greenling” and that was one of the common names used on fish base.  Went with that, but check with Dr. Pinsky to be sure 
		-- Another question…. I looked up the marker “COIII-ND3-ND4L” and it looks like a region of the mitochondrial genome, not a particular gene.  Is this still okay to use as a marker?  It didn’t look like any marker I had encountered previously so I wanted to be sure.  

	Hamasaki et al. 2007: genetic effects of hatchery fish on wild populations in red sea bream Pagrus major 
		-- no RFLP analysis, authors used sequencing 
		-- authors distinguished hatchery from wild fish using a deformity found on the nostril of the hatchery fish, but not present in wild fish
		-- I excluded data from the hatchery fish but still used data for the proposed wild fish… is this okay?  I was thinking about the potential for mixing of hatchery-released fish with wild fish and messing up the analysis 
		--I entered data for the wild fish but can take it out if need be 
		Sampling locations for the wild fish-
		-- East China Sea 
		-- Shibushi Bay 
		-- Central Kagoshima Bay 
		-- no data entered for hatchery released fish (labeled HA on the map and table with genetic diversity values)
		--SE values listed here, so no problem entering those data 
		-- averaged the control region sequences from 492-498 bp 

	Han et al. 2011: mtDNA control region Hong Kong grouper and yellow grouper 
		-- no data entered for nucleotide or haplotype diversity, so I didn’t enter any data for this particular paper 
		-- I can’t access the article online from the Wiley online library, as the Rutgers servers have been down all day due to another hacker
		--I can look more when the servers are cleared for any supplemental data not presented in this short paper 

	Han et al. 2012: phylogeography of Ammodytes personatus in Northwestern Pacific 
		-- no data entered for this paper either, as the diversity values given were from pooled individuals of two different lineages that were sampled at locations greater than 3° latitude apart
		--further, individuals in some of the northern sampling locations were from mixed lineages, both A and B, and the authors provided no way of telling them apart
		--for these reasons, I excluded this paper from the data sheet

	He et al. 2012: taxonomic status of Chinese bahaba… 
		-- two mtDNA markers used- 16S rRNA and cyt b 
		-- no diversity data reported…I will look to see if there are any supplemental data or figures available online once the Rutgers servers are back up and running 

	Hess et al. 2011: A steep genetic cline in yellowtail rockfish Sebastes flavidus… 
		--sequenced, no RFLP here 
		-- samples collected from 1993-2005 and pooled…. average these 
		-- no data entered for locations Yakutat and Iphegenia, AK, as they only had one sample each site
		-- no data entered for Crescent City, OR as there were only two samples from this site, need 4 minimum 
		-- convert SD to SE (SE = SD/ sqrt. N)

	Ho et al. 2012: population genetic structure, etc. of yellowhead jawfish
		-- paper lists two markers, the control region and COI, but makes no distinction between the two in the data table 
		-- paper also says the two loci were concatenated… googled and it said “linked together” can these not be used? 

	Hsu et al. 2011: black seabream complex.. AFLP analyses
		-- authors used AFLP analysis, not RFLP but not sequencing either… check with Dr. Pinsky to be certain that we can’t include this paper 
		-- no pi or Haplotype diversity values anyway 

	Hyde & Vetter: population genetic structure in vermilion rockfish

Chloe Lewis

	McBride et al. 2010: 
		→ none of the data from this paper were entered.
		→ the paper offers no information on h values, pi values, or base pair values.
		→ there is no information regarding coordinates of sample sites.

	McCusker and Bentzen 2010a: 
		→ none of the data from this paper were entered.
		→ the paper offers no information on h values, pi values, or base pair values.
		→ there is no information regarding coordinates of sample sites. 

4/28/2015
Chloe Lewis

	McCusker and Bentzen 2010b: 
		→ no data was entered for this paper.
		→ 5 of the 7 sample sites were sampled multiple times.
		→ 3 different species of fish were collected in this study.

4/30/2015
Chloe Lewis

	Menezes et al. 2012: 
		→ this paper provides no information on base pair values.
		→ the coordinates of sample sites were explicitly given in the pair. 

5/1/2015
Chloe Lewis

	Medonca et al. 2011: 
		→  the coordinates for sample sites are not explicitly given, however the paper provides a map from which accurate estimates were made. 
		→ no base pair values are provided from this paper. 
		→ there is no single country from which these samples were taken, therefore the oceanic region has been recorded.

	Milana et al. 2011: 
		→ no data from this paper has been entered. 
		→ no pi or h values are provided.
		→ the paper offers no informative data regarding sample site locations. 

	Miller et al. 2011: 
		→ no data from this paper has been entered.
		→ no pi or h values are provided.
		→ the paper does explicitly provide coordinate data of sample sites as well as a map.
		→ the base pair value is given.

	Mirams et al. 2011: 
		→ no data from this paper has been entered.
		→ the paper only offers a vague map displaying the entirety of the range of the study, but offers no information regarding sample sites, plots of these sites, or coordinate values.
		→ the paper does provide data for pi values

	Morgan et al. 2011: 
		→ no data from this paper has been entered. 
		→ no pi or h values provided.
		→ no information regarding coordinates of sample site locations. 

5/4/2015
Chloe Lewis

	Morgan et al. 2012: 
		→  no data from this paper has been entered.
		→ no pi or h values provided	
		→ the coordinates of sample sites are not explicitly given, however a map is provided for accurate estimation.


	Mukai et al. 2009: 
		→ a map is provided to allow for precise estimates of sampling sites; the coordinates of these sites are not explicitly given. 
		→ pi and he values are provided. 
		→ the samples were collected in the Northwestern Pacific Ocean; therefore the paper does not name any particular country from which the sampling occurred. 
		→ the base pair value given is the overall value of the sample sites; not unique to each location.
		→ data collected from the Guam location have been omitted from the data spreadsheet as it does not meet the 3-degree lat/lon requirement; the site is over 10 degrees lat away from the rest. 

5/5/2015
Chloe Lewis

	Sotka et al. 2005: 
		→ the base pair value given is the overall value of the sample sites; not unique to each location.
		→ pi values are given for each sample site; only number of haplotypes are mentioned, no haplotype diversity values. 
		→ the coordinates were provided by the paper along with a map to reference their locations relative to one another. 

	Teske et al. 2004: 
		→ no data has been entered for this paper.
		→ all of the given sample location sites are vague; only the name of the countries are mentioned, there is no precise information given as to where around these countries the sampling occurred. 
		→ due to the vast distances between most of the sites that were sampled for mtDNA, they would not meet the 3-degree lat/lon criteria. 
		→ there is no map provided and there are no coordinates explicitly given of the sample location sites. 
		→ there are no pi or he values given. 

5/6/2015
Ash Bhattacharjee

	-- 7:00 PM
	Lemair et al.
	- Used previous studies to figure out what the abbreviations were (which sites etc), and used the 
	coordinates provided in those papers or online with google maps to input data
	- No collection year provided
	- 3 abbreviations I was not able to find and did not include: MKSR, MTAN & MTAH

	Filled in Longitudes for Fishery lat mtDNA 3 Kieryn 2013-11-04.xlsx
	- Used papers cited and/or google maps 

5/7/2015
Ash Bhattacharjee

	-- 6:00 PM
	Continuing filling in longitudes for Fishery lat mtDNA 3 Kieryn 2013-11-04.xlsx
	- Noticed that the first entry for Duren et ala. 2005 > Central North is a range of 27-39 deg long
	which means its > 3 degrees*
	- For Yu et al. there are only 1 set of Q1,Q2 & Q3. Not sure which is for east china or yellow sea 
	since there is only one figure for both with only one set.

	Input longitudes for  Fishery lat mtDNA 4 Kieryn 2013-09-10.xlsx

5/14/2015
Chloe Lewis

	Muths et al. 2011: 
		→ the location of the collection sites is the southwestern region of the Indian Ocean; therefore there is no specific country. 
		→ the paper provides no pi values, however the paper does provide h values for each sample site. 
		→ the bp value is the overall value for the study; not the value of each individual sample site. 
		→  the coordinates of sample sites are not explicitly given, however a map is provided for accurate estimation; more accurate estimates were made using google maps. 

5/15/2015
Chloe Lewis

	Montanari et al. 2012: 
		→ no data entered
		→ the study used molecular data from hybrids of Chaetodon trifasciatus and C. lunulatus; mtDNA results results confirmed hybridization.
		→ this paper does not meet the requirements for data entry. 

5/18/2015
Chloe Lewis

	Lemaire Versini Bonhomme: 
		→ no data entered for this paper.
		→ the paper does not meet the 3 degree lat/lon criteria for data entry; more than half of the sample sites greatly exceed 3 degrees lat.

5/19/2015
Chloe Lewis

	Lewallen et al. 2011: 
		→  no data entered
		→ the paper provides no information or data for pi and h values
		→ there are no coordinates on the map displaying the sample site locations for this study; multiple sites do not meet the 3 degree criteria for data entry

5/20/2015
Sarah Picon

	Longitude excel sheet copied.
	Data entered.

5/21/2015
Sarah Picon

	Entering longitudes:
		Line 60 with “Clinton, CN”. Is “CN” supposed to be “CT” for Connecticut?
		I can’t find “Flatbrotten, Norway-Sweden” on google maps (begins on line 237)
		Starting at line 309, the location seems too broad. Nova Scotia, Canada was measured at 43.357279, -65.650454 (at one end) to 46.058119, -59.476138 (at the other end). So it exceeds 3 degrees longitude. Should I remove that entry?
		Check with Malin.

5/27/2015
Sarah Picon

	Entering longitudes:
		Canino et al. 2005 Mol Ecol Notes 5:908-910
			Data entered.

		Adams et al. 2006 Molecular Ecology 15:1109-23
			In the paper the location is written as Clinton, CN. Is it a typo?
			Check with Malin.

		Antoro et al. 2006 Marine Biotechnology 8:17-26
			Data entered.

		Babbucci et al. 2006 Mol Ecol Notes 6:1126-1128
			Data entered.

5/28/2015
Sarah Picon

	Entering longitudes:
		Bekkevold et al. 2005 Evolution 59:2656-2668
			Data entered.

		Berrebi et al. 2006 Mol Ecol Notes 6:724-727
			Data entered.

		Blanco et al. 2006 Mol Ecol Notes 6:767-769
			Data entered.

		Chevolot et al. 2006 J Sea Research 56:305-316
			Data entered.

		Chevolot et al. 2006 Mol Ecol 15:3693-3705
			Data entered.

		Galarza et al. 2006 Mol Ecol Notes 6:747-749
			Data entered.

5/30/2015
Sarah Picon

	Entering longitudes:
		Graves & McDowell 2006 Bull Mar Sci 79:469-482
			Data entered.

		Guo et al. 2005 Mol Ecol Notes 5:369-371
			Data entered.

		Hauser et al. 2006 Intl Pacific Halibut Commission Sci Report 81:1-28
			Can’t find it online.
			Paper received.

		Jean et al. 2006 Mol Ecol Notes 6:1269-1271
			Data entered.

		Jones & Barber 2005 Mol Ecol Notes 5:834-836
			Data entered.

		Karlsson & Mork 2005 ICES J Mar Sci 62:1588-1596
			Data entered.

		Kokita et al. 2006 Mol Ecol Notes 6:210-211
			“Paracentropogon rubripinnis” was the name written down under “spp”. However, in the paper the species is referred to as “Hypodytes rubripinnis”. Should I change the name to “Hypodytes rubripinnis”?
			Data entered.

		LeClair et al. 2006 Trans of the Am Fish Soc 135:1631-1643
			Article requested online.
			Data entered.

		Mackiewicz et al. 2005 Mar Bio 148:213-221
			Data entered.

		Miller-Sims et al. 2005 Mol Ecol Notes 5:841-843
			Data entered.

		Oretga-Villaizan et al. 2006 Fisheries Sci 72:556-567
			Data entered.

		Poulsen et al. 2006 Mol Ecol 15:321-331
			Data entered.

5/31/2015
Sarah Picon

	Entering longitudes
		Purcell et al. 2006 Proc R Soc B 273:1483-90
			Data entered.

		Renshaw et al. 2006 Mol Ecol Notes 6:1162-1164
			Data entered.

		Rogers et al. 2006 Mar Bio 149:915-924
			Data entered.

		Spies et al. 2005 Mol Ecol Notes 5:469-471
			Data entered.

		Van Houdt et al. 2006 Mol Ecol Notes 6:157-159
			Data entered.

		Wilson 2006 Mol Ecol 15:809-824
			Data entered.

6/3/2015
Sarah Picon

	Entering longitudes
		LeClair et al. 2006 Trans of the Am Fish Soc 135:1631-1643
			Article received
			Data entered.

		Hauser et al. 2006 Intl Pacific Halibut Commission Sci Report 81:1-28
			Article received.
			Data entered.

6/9/2015
Sarah Picon

	Entering longitudes
		Galarza et al. 2009 CJFAS 66:1478-1490
			Cabo de gata
			Blanes
			Porticello
			Syracuse
			Latitude entered for the above areas isn’t the best fit when compared to the location given in the map provided by the article. Should I change the latitude?
			Check with Malin.
			Data entered.

		Helyar et al. 2011 Cons Gen Res 3:173-176
			Data entered.

		Helyar et al. 2010 Cons Gen Res 2:1-4
			Data entered.

		Horne et al. 2010 Cons Gen Res 2:191-194
			Data entered.

6/13/2015
Sarah Picon

	Entering longitudes
		Knutsen et al. 2009 Cons Gen Res 1:89-92
			Data entered.

		Larmuseau et al. 2010 Heredity 105:532-542
			Data entered.

		Landinez-Garcia et al. 2009 Ciencias Marinas 35:321-331
			Data entered.

		Lawton et al. 2011 Mol Ecol 20:3584-3598
			Data entered.

		Leray et al. 2010 Evolution 64:1218-1230
			Data entered.

		Lin et al. 2011 Gen & Mol Res 10:764-768
			Data entered.

		Liu & Avise 2011 Marine Bio 158:893–901
			Data entered.

6/14/2015
Sarah Picon

	Entering longitudes
		Liu et al. 2011 Acta Oceanologica Sinica 30:76-83
			Data entered.

		López et al. 2010 Cons Gen 11:2431-2436
			Data entered.

		Ma et al. 2011 Gen & Mol Res 10:1455-1460
			Data entered.

		Larson et al. 2011 Cons Gen 12:679-690
			Data entered.

		Larsson et al. 2010 Heredity 104:40-51
			Data entered.

		Lawton et al. 2010 Cons Gen Res 2:209-213
			Data entered.

		McCraney et al. 2010 Mol Ecol 19:3315-3327
			Data entered.

		McCusker & Bentzen 2010 Mol Ecol 19:4228-4241
			Data entered.

		Qiu & Miyamoto 2011 Copeia 2:264-269
			Data entered.

		Mobley et al. 2010 J Biogeography 37:1363-1377
			Data entered.

		Dammannagoda et al. 2011 CJFAS 68:210-223
			Data entered.

		Muths & Bourjea 2011 Cons Gen Res 3:629-631
			Data entered.

		Nance et al. 2011 PLoS ONE 6:e21459
			Data entered.

6/16/2015
Sarah Picon

	Entering longitudes
		Palof et al. 2011 Mar Biol 158:779-792
			Data entered.

		Papetti et al. 2011 Cons Gen Res 3:259-262
			Data entered.

		Chopelet et al. 2009 Mol Ecol Res 9:1221-1223
			Data entered.

		Pinsky et al. 2010 Evol 64:2688-2700
			Data entered.

		Pita et al. 2011 Continental Shelf Research 31:376-387
			I’m not sure how the numbers for latitude were entered.
			Data entered.

6/17/2015
Sarah Picon

	Entering longitudes
		Plank et al. 2010 J Fish Bio 77:329-340
			Data entered.

		Putte et al. 2009 Polar Biol 32:1731-1741
			Data entered.

		Ravago-Gotanco et al. 2010 Cons Gen Res 2:225-227
			Data entered.

		Ring et al. 2009 Mol Ecol Res 9:1401-1403
			Data entered.

		Saillant et al. 2010 ICES J Mar Sci 67:1240-1250
			Data entered.

6/18/2015
Sarah Picon

	Entering longitudes
		Salas et al. 2009 Mar Biol 157:437-445
			Data entered.

		Sbrocco et al. 2011 Cons Gen Res 3:585-587
			Data entered.

		Sellas et al. 2011 Cons Gen Res 3:609-611
			Data entered.

		Sha et al. 2009 Cons Gen 10:1799-1801
			Data entered.

		Sorenson et al. 2011 Cons Gen Res 3:721-723
			Data entered.

		Stroganov et al. 2010 Russian J Mar Biol 35:490-493
			Data entered..

6/19/2015
Sarah Picon

	Entering longitudes
		Hess et al. 2001 CJFAS 68:89-104
			Data entered.

		Tseke et al. 2009 Mol Ecol Res 9:1162-1164
			Data entered.

		Umino et al. 2009 Fish Sci 4:909-919
			Data entered.

		Underwood 2009 Cons Gen Res 1:491-493
			Data entered.

		Underwood 2010 Cons Gen Res 2:261-264
			Data entered.

		Veríssimo et al. 2010 Mol Ecol 8:1651-1662
			Data entered.

		Veríssimo et al. 2011 J of Heredity 102:88-93
			Data entered.

		Wang et al. 2010 Gen & Mol Res 9:931-934
			Data entered.

		White et al. 2010 Heredity 106:690-699
			Data entered.

		Xu et al. 2009 Cons Gen 10:1181-1184
			Data entered.

6/25/2015
Sarah Picon

	Entering longitudes
		Machado-Schiaffino et al. 2011 Biol Cons 144:330-338
			Data entered.

		Nielsen et al. 2010 Cons Gen 11:999-1012
			Data entered.

		Pini et al. 2011 Coral Reefs 30:399-404
			Data entered.

		Roy et al. 2010 Cons Gen Res 2:45-49
			Data entered.

7/1/2015
Sarah Picon
	Entering longitudes
		Sekino et al. 2011 Cons Gen 12:139-159
			Data entered.

		Teske et al. 2010 Mar Biol 157:2029-2042
			Data entered.

		Tripp-Valdez et al. 2010 Fish Res 105:172-177
			Data entered.

7/2/2015
Sarah Picon

	Entering longitudes
		Roberts & Ayre 2010 Mar Ecol Prog Series 412:223-230
			Data entered.

		Schunter et al. 2011 J Exp Mar Biol & Ecol 401:126-133
			Data entered.

		--
		*fixed the references on the excel sheet because the last page number for each was increasing

		Miller-Sims et al. 2005. Mol Ecol Notes 5:424-426 
			Data entered.

		Chevolot et al. 2005. Mol Ecol Notes 5:427-429
			Data entered.

		Keeney et al. 2005. Mol Ecol 14:1911-1924
			Data entered.

		Hoffman et al. 2005. Mol Ecol 14:1367-1375
			Data entered.

		Stockley et al. 2005 Mar Bio 146:793-804
			Data entered.

		Withler et al. 2003. Env Biology of Fishes 69:345-357.
			Data entered.

		Shaw, Arkhipkin & Al-Khairulla. 2004. Mol. Ecol.13:3293-3303
			Data entered.

		Carlsson et al. 2004. Mol. Ecol.13:3345-3356.
			Data entered.

		Garoia et al. 2004. Marine Biotechnology. 6:446-452.
			Data entered.

		Miller-Sims et al. 2004. Mol Ecol Notes. 4:336-338.
			Data entered.

		McCauley et al. 2004. Mol. Ecol. Notes 4:494-496.
			Data entered.

		Hoffman et al. 2004. Mol Ecol Notes 4:342-344.
			Data entered.

		Williams et al. 2004. Mol Ecol Notes 4:525-527
			Data entered.

		Kumagai et al. 2004. Mar Biotechnol 6:221-228
			Data entered.

		Hoarau et al., 2004. Journal of Sea Research. 51:183-190.
			Data entered.

		Withler et al. 2004. Env Bio of Fishes 69:345-357.
			Data entered.

		Saillant. 2003. Transactions of the American Fisheries Society 132:1229-1235.
			Data entered.

		Appleyard, Williams & Ward. 2004. CCAMLR Science 11:21-32.
			Data entered.

		Williams et al. 2003. Mol Ecol Notes 3:547-549.
			Data entered.

		Nielsen et al. 2003 Mol Ecol 12:1497-1508.
			Data entered.

		Van Herwerden, Benzie & Davies. 2003. Journal of Fish Biology. 62:987-999.
			Data entered.

		Aboim et al. 2003. Mol Ecol Notes 3:18-20.
			Data entered.

		Heist et al. 2003. Mol Ecol Notes 3:59-61.
			Data entered.

		Knutsen et al. 2003 Mol Ecol 12:385-394.
			Data entered.

		Olsen, Merkouris & Seeb. 2002. Fish. Bull. 100:752-764.
			Data entered.

		Carreras-Carbonell, Macpherson & Pascual. 2004. Mol Ecol Notes 4, 438-439.
			Data entered.

		Williams et al. 2004. Mol Ecol Notes 4:46-48.
			Data entered.

		Nielsen et al. 2004. Mol Ecol 13, 585-595.
			Data entered.

		Pampoulie et al. 2004. Heredity 92, 434-445.
			Data entered.

7/23/2015
Sarah Picon

	Entering longitudes
		Cunningham et al. 2009 Can. J. Fish. Aquat. Sci 66:153-166
			Data entered.

		Skarstein, Westgaard & Fevolden 2007 Journal of Fish Biology 70:271-290
			Data entered.

		Wennevik et al. 2008 Hydrobiologia 606:7-25
			Data entered.

		Haney, Silliman, & Rand 2007 Heredity 98:294-302
			Data entered.

		Seyoum, Denison & Tringali 2007 7:141-143
			Data entered.


7/24/2015
Sarah Picon

	Entering longitudes
		Selkoe et al. 2006 Ecology 87:3082-3094
			Data entered.

		Pardo et al. 2007 Conserv Genet 8:1243-1245
			Data entered.

		Pampoulie et al. 2006 Can. J. Fish. Aquat. Sci. 63:2660-2675.
			Data entered.

7/25/2015
Sarah Picon
	Entering longitudes
		Larmuseau et al. 2007 Mol Ecol Notes.
			Fixed citation.

		Larmuseau et al. 2007 Mol Ecol Notes. 7:147-149.
			Data entered.

		Kim, Morishima & Arai 2007 Mol Ecol Notes 7:79-81.
			Data entered.

		Galarza et al. 2007 Conserv Genet 8:1251-1253.
			Data entered.

7/29/2015
Sarah Picon

	Entering longitudes
		Fritsch et al. 2007 Fisheries Research 83:123-132.
			Data entered.

		Bisol et al. 2007 Hydrobiologia 577:151-159.
			Data entered.

		Babbucci et al. 2007 Mol. Ecol. Notes 7:293-295.
			Fixed citation.
			Data entered.

		Thacker et al. 2008 Mar Biol. 153:375-385.
			Fixed citation.
			Data entered.

7/30/2015
Sarah Picon

	Entering longitudes
		Susana et al. 2007 Mol. Ecol. Notes 7:791-793.
			Data entered.

		Roques et al. 2007 Mol. Ecol. Notes 7:661-663.
			Data entered.

		Papetti et al. 2007 Polar Biol 30:1605-1613.
			Data entered.

		O'Leary et al. 2007 Journal of Fish Biology 70:310-335.
			Fixed citation.
			Data entered.

		Knutsen et al. 2007 Mol. Ecol. Notes 7:851-853.
			Data entered.

		Feldheim et al. 2007 Mol. Ecol. Notes 7:1366-1368.
			Data entered.

		Buston et al. 2007 Mol. Ecol. Res. 16:3671-3678.
			Data entered.

		Liu et al. 2008 Mol. Ecol. Res. 8:202-204.
			Data entered.

		Yagishita & Kobayashi 2008 Mol. Ecol. Res. 8:302-304.
			Data entered.

7/31/2015
Sarah Picon

	Entering longitudes
		Bernardi 2008 Mol. Ecol. Res. 8:1512-1514
			Data entered.

		Burridge & England 2009 Perm. Gen. Res. Note. 9:948-951.
			Fixed citation.
			Data entered.

		Christie & Eble 2009 Mol. Ecol. Res. 9:544-546.
			Data entered.

		Feldheim et al. 2009 Mol. Ecol. Res. 9:639-644.
			Data entered.

		Heist et al. 2008 Mol. Ecol. Res. 8:1518-1519.
			Data entered.

		Hepburn et al. 2009 Coral Reefs 28:277-288.
			Data entered.

		Jones et al. 2008 Mol. Ecol. Res. 8:1332-1334.
			Data entered.

		Matschiner, Hanel & Salzburger 2009 Mol. Ecol. Res. 18:2574-2587.
			Data entered.

		Miller-Sims et al. 2008: Mol. Ecol. 17:5036-5048.
			Data entered.

8/5/2015
Sarah Picon

	Entering longitudes
		Nohara et al. 2009 Mol. Ecol. 9:610-612.
			Data entered.

		Purcell et al. 2009 Journal of Fish Biology 74:403-417.
			Data entered.

8/6/2015
Sarah Picon

	Entering longitudes
		Tseng et al. 2009 Journal of Fish Biology 74:357-376.
			Data entered.

		Vella 2009 Collect. Vol. Sci. Pap. ICCAT 63:161-173.
			Data entered.

		Xing et al. 2009 Conserv Genet 10:789-791.
			Data entered.

8/8/2015
Sarah Picon

	Entering longitudes
		Vagelli, Burford & Bernardi 2009 Marine Genomics 1:129-134.
			Can’t figure out where to find the locations sampled at.
			Ask Malin.

		Maggio et al. 2009 ICES J Mar Sci 66:1883-1891.
			Data entered.

8/11/2015
Michael Weiss

	Muto et. al 2011 Fishery Bulletin 109(4):429-439
	Data not entered. The samples were all collected at a single site (intent of the authors was to eliminate geographic variability.) Additionally, the mtDNA was sequenced using AFLP.

	Nance et. al 2011 PLoS One 6(7):1-12
	Data was all good but in a supporting table which was not accessible via the link in the paper. The table with mtDNA data was accessible at the PLoS One website. 

	Nielsen et. al 2010 Conservation Genetics 11:999-1012
	Data entered. Two of the sites were acceptable while a third was not used since it was averaged over a large area. 

	Nohara et. al 2010 Fish Science 76:75-81
	Data entered

	Nordeide et. al 2011 Marine Ecology Progress Series 435:269-283
	Data not entered. This is a review paper with no data on genetic or haplotype diversity.

	Nunez et. al 2010 Revista de Biologia Marina y Oceanografia 45(1):565-573
	Data from 6 out of 7 (Paren has a sample size of 1) locations in South America were entered, to the exclusion of the 1 data point from South Africa (Different species, and thus only one sample site)

8/12/2015
Michael Weiss

	Ovenden et. al 2010 Marine and Freshwater Research 61:253-262
	Data not entered. The samples were collected from fish markets. Additionally, the genetic data was averaged over large areas.

	Ovenden et. al 2011 Marine Biology 158:1497-1509
	Data entered. Statistics for two different species of sharks. 

	Panithanarak et. al 2010 Zoological Studies 49(4):564-576
	Data entered. 

	Pedrosa-Gerasmio et. al 2012 PLoS One 7(4):1-7
	Data not entered. mtDNA was used to identify the organism, but no haplotype of genetic diversity figures were published.

	Peng et. al 2009 Biochemical Systematics and Ecology 37:626-632
	Data entered. 

	Pereira et. al 2009 Journal of Fish Biology 75:321-331
	Data entered.

	Pereyra et. al 2010 Fisheries Research 106:468-473
	Data not entered. 3 samples were collected, however only averaged data were reported, thus only one effective sample was taken (3 sites were within 3 degrees of each other).

	Persis et. al 2009 Molecular Biology Report 36:1733-1740
	Data not entered. 28 different species were sampled, thus the sampling size of each species was too small. Additionally, haplotype and genetic diversity were not published.

	Portnoy and Gold 2012 Journal of Biogeography 39:1499-1507 
	Data not entered. Haplotype and genetic diversity data was not entered. 

	Henriques et. al 2012 African Journal of Marine Science 34(4):585-591
	Data entered.

8/13/2015
Michael Weiss

	Psomadakis et. al 2012 Italian Journal of Zoology 79(1):111-119
	Data not entered. Only two fish were studied.

	Randall and DiBattista 2012 Pacific Science 66(1):97-110
	Data not entered. The specimens were collected over a broad range and no haplotype or genetic diversity statistics were published. 

	Randall and Rocha 2009 The Raffles Bulletin of Zoology 57(2):511-520
	Data not entered. No haplotype or genetic diversity statistics were published. 

	Randall et. al 2011 Pacific Science 65(2):265-275
	Data not entered. No haplotype or genetic diversity statistics were published. 

	Ravago-Gotanco and Juinio-Menez 2010 Molecular Ecology 19:4520-4534
	Data entered.

	Reese et. al 2010 Journal of Heredity 101(4):391-402
	Data not entered. Haplotype and genetic diversity data was averaged over sampling pool, with only one number reported.

	Reza et. al 2011 Fish Science 77:59-67 
	Data from one of the species (two collection sites) was entered, but the second species data was from a single collection site and thus not entered. 

	Rivera et. al 2011 Journal of Marine Biology Volume 2011 Article ID 765353
	Data not entered. The study did not generate any new mtDNA genetic data, rather the data was taken from another paper. 

	Rocha et. al 2010 Zootaxa 2422:22-30
	Data not entered. Only one sample site was used to harvest specimens for the study. 

	Roy et. al 2012 Marine and Coastal Fisheries: Dynamics, Management, and Ecosystem Service 4:391-402
	Data entered. 

8/14/2015
Michael Weiss

	Gotoh et. al 2009 Genes Genetic Systems 84:287-295
	Data entered. The samples were taken from marine lakes and lagoons. Area is very sample, subsequently all 6 sample sites are very close together.

	Saarman et. al 2010 Conservation Genetics 11:1989-2000
	Data entered. Some of the samples collected had unknown sources and were excluded from entry. 

	Saillant et. al 2012 Fisheries Management and Ecology 19:301-312
	Data not entered. There was data to enter, but the associated tables (which needed to be searched online) were unclear as to the source of each haplotype and nucleotide diversity statistic. 

8/17/2015
Michael Weiss

	Sala-Bozano et. al 2009 Molecular Ecology 18:4811-4826
	Data entered. Locations were sampled in two consecutive years, the sample with more specimens was recorded. 

	Sanna et. al 2011 Journal of the Marine Biological Association of the United Kingdom 91(6):1289-1293
	Data not entered. Sample size is too small.

	Santos et. al 2010 Philippine Journal of Science 139(1):43-50
	Data entered. 

8/18/2015
Michael Weiss

	Santos et. al 2011 Journal of Heredity 102(1):47-54
	Data not entered. Haplotype and nucleotide diversity were averaged over an area >3 degrees of latitude and longitude.
 
	Sassa and Tsukamoto 2010 Marine Ecology Progress Series 419:185-199
	Data not entered. RFLP was used to sequence mtDNA.
 
	Schinske et. al 2010 Marine Biology 157:123-134
	Data not entered. Nucleotide and haplotype diversity data are not reported.
 
	Sekino et. al 2011 Conservation Genetics 12:139-159
	Data entered.
 
	Noguchi et. al 2012 Mitochondrial DNA 23(3):223-229
	Data entered.
 
	Silva-Segundo et. al 2011 Review Fish Biology Fisheries 21:259-282
	Data not entered. Haplotype and nucleotide diversity data were averaged over a large area of sampling sites.
 
	Song et. al 2011 Journal of the World Aquaculture Society 42(4):512-521
	Data not entered. 8 of the samples were not wild, with the remaining two close enough geographically as to effectively constitute a single sample.
 
8/19/2015
Michael Weiss

	Most of the first hour was spent adding in a section for Hese and Pise.
 
	Smith and Campana 2010 Canadian Journal of Fisheries and Aquatic Science 67:1533-1548
	Data not entered. Haplotype and nucleotide diversity statistics are not presented.
 
	Straube et. al 2010 Zoologica Scipta 40(1):61-75
	Data not entered. Samples were taken from other studies and haplotype and nucleotide diversity statistics were not published.
 
	Yanagimoto et. al 2012 Marine Genomics 7:17-25
	Data not entered. Sequencing was accomplished via RFLP.
 
	Tanaka et. al 2011 Marine and Freshwater Research 62:548-556
	Data not entered. Only one sample site was reported.
 
	Teacher et. al 2012 BMC Evolutionary Biology 12:248
	Data not entered. Haplotype and nucleotide diversity statistics were not published.
	Teske et. al 2010 Marine Biology 157:2029-2042
	Data entered.
 
	Thacker and Roje 2009 Molecular Phylogenetics and Evolution 52:735-745
	Data not entered. Specimens were collected from a museum and genetic diversity statistics are not reported.
 
	Thangaraj and Lipton 2011 Indian Journal of Biotechnology 10:140-142
	Data not entered. Haplotype and nucleotide diversity data were not reported.
 
	Tringali et. al 2011 Journal of Heredity 102(4):416-432
	Data not entered. Haplotype and nucleotide diversity of the mtDNA are not reported.
 
	Tripp-Valdez et. al 2012 Journal of Applied Ichthyology 28:516-523
	Data entered. 

8/20/2015
Michael Weiss

	Tseng and Smith 2012 Marine and Freshwater Research 63:198-209
	Data not entered. Haplotype and nucleotide diversity data for mDNA was not reported.

	Tseng et. al 2011 Environmental Biology of Fishes 91:103-115
	Data not entered. Only one sample of each species was collected.

	Turan 2011 Journal of Black Sea/Mediterranean Environment 17(1):14-31
	Data not entered. Only one sample of each species was collected.

	Varela et. al 2012 Marine Biology 159:1049-1060
	Data entered. 

Sarah Picon

	Entering longitudes
		McDowell et al. 2007 Gulf and Caribbean Research 19:75-82.
			Data entered.

		Ball et al. 2007 Mar Biol 150:1321-1332.
			Fixed citation.
			Data entered.

8/21/2015
Michael Weiss

	Vasquez-Yeomans et. al 2011 Biological Invasions 13:2635-2640
	Data not entered. Only one specimen was collected for the study.

	Veilleux et. al 2011 Fisheries Research 108:74-80
	Data entered.

	Vergara-Chen et. al 2009 Molecular Phylogenetics and Evolution 53:602-607
	Data not entered. Only one sample of each species was collected.
	Vergara-Chen et. al 2009 Journal of Fish Biology 75:1101-1107
	Data not entered. This study used RFLP for sequencing. 

	Vergara-Chen et. al 2010 Marine Ecology 31:270-275
	Data not entered. Sample sites were close enough as to be deemed one sample site. 

	Verissimo et. al 2011 ICES Journal of Marine Science 68(3):555-563
	Data not entered. The study did not specify any coordinates of the sampling locations.

	Verissimo et. al 2012 Marine and Freshwater Research 63:505-512
	Data not entered. The study did not specify any coordinates of the sampling locations.

	Victor 2010 Journal of the Ocean Science Foundation 4:1-30
	Data not entered. No haplotype or nucleotide diversity data were reported.

	Victor 2010 Journal of the Ocean Science Foundation 3:1-16
	Data not entered. No haplotype or nucleotide diversity data were reported.

	Victor 2012 Journal of the Ocean Science Foundation 5:1-19
	Data not entered. No haplotype or nucleotide diversity data were reported.

	Victor and Randall 2010 Zoological Studies 49(6):865-871
	Data not entered. No haplotype or nucleotide diversity data were reported.

	Victor et. al 2009 Zootaxa 2215:24-36
	Data not entered. No haplotype or nucleotide diversity data were reported.

	Vignon and Morat 2010 Marine Ecology Progress Series 411:231-241
	Data not entered. No haplotype or nucleotide diversity data were reported.

	Vinas et. al 2010 ICES Journal of Marine Science 67:1222-1229
	Data not entered. The sample set contained information from a previous study. Additionally, the differentiation between two clades is unclear. 

	Vinas et. al 2011 Reviews in Fish Biology and Fisheries 21:527-541
	Data not entered. Haplotype and nucleotide diversities were averaged over a large area. 

	Visram et. al 2010 Marine Biology 157:1475-1487
	Data entered. Samples from Tanzania (3 different samples in the same country) were recorded individually while samples from the other three countries were recorded collectively. 

Sarah Picon

	Entering longitudes
		Bowen et al. 2006 Marine Biology 149:899-913.
			Data entered.
	
		Chevolot et al. 2007 Mar Biol 151:1275-1286.
			Fixed citation.
			Data entered.

8/24/2015
Michael Weiss

	von der Heyden and Connell 2012 African Journal of Marine Science 34(4):505-510
	Data not entered. Only one sample of each species was taken.

	von der Heyden et. al 2010 Molecular Phylogenetics and Evolution 55:1183-1188
	Data not entered. Haplotype and nucleotide were not reported for individual sampling sites.

	White et. al 2011 Marine Ecology Progress Series 434:155-164
	Data not entered. This study did not sequence mtDNA.

	Whitney et. al 2012 Journal of Biogeography 39:1144-1156
	Data entered. Only the sites with at least 4 sampled individuals were recorded.

8/25/2015
Michael Weiss

	Wilson and Veraguth 2010 Molecular Ecology 19:4535-4553
	Data not entered. Haplotype and nucleotide diversity statistics are not reported. 

	Winters et. al 2010 Marine Biology 157:1679-1691
	Data entered. Some collection sites consisted of less than 4 specimens and were excluded from data entry. 

	Woodall et. al 2011 Journal of Fish Biology 78:1738-1756
	Data entered. Some collection sites consisted of less than 4 specimens and were excluded from data entry.

8/26/2015
Michael Weiss

	Wu and Yang 2012 PLOS One 7(12):1-9 
	Data not entered. Most of the samples are captive, while the others are all from one location. 

	Wu et. al 2010 Fisheries Research 105:248-253
	Data not entered. Haplotype and nucleotide diversity statistics are averaged over a large range. 

	Xiao et. al 2009 Environmental Biology of Fishes 85:303-314
	Data entered. 

	Xiao et. al 2010 Biochemical Genetics 48:402-417
	Data entered. Some of the samples had too broad a sample collection range and were thus excluded. 

	Xiao et. al 2011 Genetics 139:187-198
	Data entered. Some of the samples had too broad a sample collection range and were thus excluded.

8/27/2015
Michael Weiss

	Yoon et. al 2012 Asian-Australian Journal of Animal Science 25(2):170-176
	Data not entered. Specimens used were the same as an earlier paper.

	Zarraonaindia et. al 2012 PLoS One 7(7):1-10
	Data not entered.
	Awaiting response from Professor Pinsky about mtDNA SNPs

	Zhang et. al 2011 Chinese Journal of Oceanology and Limnology 29(5):967-972
	Data not entered. Haplotype and nucleotide diversity statistics were averaged across different sample sites.

	Zhang et. al 2011 Biochemical Systematics and Ecology 39:627-634
	Data not entered. Sample size is too small.

	Zheng et. al 2012 Zoological Research 33(3):329-336
	Data not entered. Haplotype and nucleotide diversity statistics were averaged across different sample sites. (I think...the paper isn’t in English)

	Zhuang and Cheng 2010 Molecular Biology and Evolution 27(6):1391-403
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Ahonen and Stow 2008 Journal of Fish Biology 73:450-455
	Data not entered. This paper did not perform any population genetic studies, rather it was a novel method for collection of mtDNA from sharks. 

	Aizdaicher 2008 Russian Journal of Marine Biology 34(2):139-142
	Data not entered. This paper concerns microalgae, not marine fish.

	Anderson et. al 2008 North American Journal of Fisheries Management 28:592-606
	Data not entered. Haplotype and nucleotide diversity statistics are not reported for mtDNA.

	Anderson et. al 2009 Fisheries Bulletin 107:13-23
	Data not entered. Haplotype and nucleotide diversity statistics were averaged across different sample sites.

8/30/2015
Michael Weiss

	Anderson and Karel 2009 Marine and Coastal Fisheries: Dynamics, Management, and Ecosystem Science 1:121-132
	Data entered. 

	Bektas and Belduz 2008 Journal of Fish Biology 73:1228-1248
	Data entered using control region marker (had a larger n). 

	Bernardi 2009 Journal of Fish Biology 74:1049-1055
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

9/1/2015
Michael Weiss

	Betancur et. al 2007 Molecular Phylogenetics and Evolution 45:339-357
	Data not entered. Sample size is too small.

	Breines et. al 2008 Polar Biology 31:1245-1252
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Brendtro et. al 2008 Marine Biology 155:11-22
	Data not entered. Sampling regions are too broad. 

	Burford 2009 Journal of Evolutionary Biology 22:1471-1486
	Data not entered. Study did not involve mtDNA.

	Byrkjedal et. al 2008 Journal of Fish Biology 73:1183-1200
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Carr and Marshall 2008 Mitochondrial DNA 19(6):490-496
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Carr and Marshall 2008 Genetics 180:381-389
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Carreras-Carbonell 2008 Conservation Genetics 9:181-190
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Castro et. al 2007 Molecular Ecology 16:5183-5192
	Data not entered. Sampling regions are too broad.

	Chen et. al 2008 African Journal of Biotechnology 7(20):3554-3562
	Data entered. 

	Chiang et. al 2006 Fisheries Research 79:219-225
	Data not entered. Sampling regions are too broad.

	Chiang and Chen 2008 The Raffles Bulletin of Zoology 19:183-201
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Chow et. al 2009 Journal of Experimental Marine Biology and Ecology 369:17-21
	Data not entered. Sample size is too small.

	Comesana et. al 2008 Fisheries Research 89:122-131
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Cooper et. al 2009 Molecular Phylogenetics and Evolution 52:1-16
	Data not entered. This paper considered one sample of many cogeneric fishes.

	Corrigan and Beheregaray 2009 Molecular Phylogenetics and Evolution 52:205-216
	Data not entered. This paper considered one sample of many cogeneric fishes.

	Craig et. al 2008 Endangered Species Research Preprint 1-8
	Data entered. 16S marker was used since it had the largest n. 

9/2/2015
Michael Weiss

	da Silva Rodrigues-Filho et. al 2009 Genetics and Molecular Biology 32(2):405-413
	Data not entered. This study did not include haplotype or nucleotide diversity statistics. 

	Dammannagoda et. al 2008 Fisheries Research 90:147-157
	Data entered. Some locations were sampled more than once, but only the years with the largest n per location were recorded. 

	Kristoffersen and Magoulas 2008 Fisheries Research 91:187-195
	Data not entered. Haplotype and nucleotide diversity statistics are not reported. 

	Debes et. al 2008 Molecular Ecology 17:3873-3888
	Data entered. 

	Dodson et. al 2007 Molecular Ecology 16:5030-5043
	Data not entered. Statistics were averaged over large sampling areas. 

	Drew et. al 2008 Conservation Biology 22(4):965-975
	Data entered. 

	Eble et. al 2009 Marine Biology 156:689-698
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Farrell et. al 2009 ICES Journal of Marine Science 66:561-565
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Garcia-Varela and Gonzalez-Oliver 2008 Journal of Parasitology 94(4):959-962
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Garcia et. al 2008 Marine Biology 155:325-336
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

9/3/2015
Michael Weiss

	Golani et. al 2007 Biology Letters 3:541-545
	Data entered. Only samples with n of at least 4 and an identifiable sampling location were recorded. 

	Gomes et. al 2008 Genetics and Molecular Biology 31(1):372-376
	Data entered. Only new data published in this paper was recorded.

	Goswami et. al 2009 Hydrobiologia 621:213-221
	Data entered. 

	Gray et. al 2009 Bulletin of Marine Science 84(1):25-42
	Data not entered. Sample locations are too wide.

	Guo et. al 2007 Marine Biotechnology 9:682-688
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Hall and Cribb 2008 Zootaxa 1718:1-35
	Data not entered. Only 2 pages of this paper were in the folder. 

	Han et. al 2008 Fisheries Science 74:770-780
	Data entered. 

	Han et. al 2008 African Journal of Biotechnology 7(11):1774-1781
	Data not entered. Haplotype and nucleotide diversity statistics are averaged across whole clades. 

	Harold et. al 2008 Bulletin of Marine Science 82(1):119-136
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Herwerden et. al 2009 Marine Biology 156:1595-1607
	Data entered. The GBR statistic was not included since it is an average across a large span of ocean. 

9/6/2015
Michael Weiss

	Hickey et. al 2009 Molecular Ecology 18:680-696
	Data entered. Only samples with n of at least 4 and with a small enough sampling range were recorded. 

	Imai et. al 2009 Fisheries Science 75:343-350
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Jones et. al 2008 Polar Biology 31:1481-1497
	Data not entered. Haplotype and nucleotide diversity statistics were averaged for each species. 

	Karlsson et. al 2008 North American Journal of Fisheries Management 28:1294-1304
	Data not entered. Haplotype and nucleotide diversity statistics were averaged across sampling locations. 

	Kartavtsev et. al 2007 Marine Biology 152:757-773
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Kartavtsev et. al 2009 Genes and Genomics 31(2):183-197
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

9/7/2015
Michael Weiss

	Kartavtsev et. al 2009 Aquatic Biology 8:95-103
	Data not entered. Haplotype and nucleotide diversity statistics are not reported. 

	Kendall et. al 2007 NOAA Professional Paper NMFS 9:1-26
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Kijewska et. al 2009 Marine Biology Research 5:186-192
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Klossa-Kilia et. al 2007 Biological Journal of the Linnean Society 92:151-161
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.
	Kojima et. al 2009 Journal of Oceanography 65:187-193
	Data entered. 

	Kuhn et. al 2009 Copeia 2:320-327
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Kuriiwa et. al 2007 Molecular Phylogenetics and Evolution 45:69-80
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Lakra et. al 2009 Molecular Biology Report 36:831-839
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Lavoue et. al 2008 Ichthyology Research 55:328-334
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

9/8/2015
Michael Weiss

	Lewallen et. al 2007 Marine Biology 152:599-609
	Data not entered. Haplotype and nucleotide diversity statistics are not reported for mtDNA markers. 

	Loh et. al 2008 The Raffles Bulletin of Zoology 19:135-150
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Ma et. al 2008 Aquatic Biology 2:29-35
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Matschiner et. al 2009 Molecular Ecology 18:2574-2587
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	McDowell and Graves 2008 Canadian Journal of Fisheries and Aquatic Sciences 65:1307-1320
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Mendonca et. al 2009 Neotropical Ichthyology 7(2):213-216
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Murray et. al 2008 Marine Biology 153:1015-1022
	Data not entered. Sampling location is not specified well enough.

	Muths et. al 2009 Fisheries Research 97(3):263-269
	Data entered. 

	Near and Cheng 2008 Molecular Phylogenetics and Evolution 47:832-840
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Neethling et. al 2008 BMC Evolutionary Biology 8:325
	Data entered. 

	Palsson et. al 2009 Polar Biology 32:471-479
	Data entered. Only Polar Cod were recorded. 

	Palsson et. al 2008 Marine Biotechnology 10:270-277
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Ramon and Knope 2008 Molecular Phylogenetics and Evolution 46:475-483
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Robalo et. al 2009 Marine Biology 156:1733-1737
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Robinson et. al 2008 Marine and Freshwater Research 59:502-514
	Data entered. 

9/9/2015
Michael Weiss

	Rocha et. al 2008 BMC Evolutionary Biology 8:157
	Data entered. 

	Saito et. al 2008 Journal of Fish Biology 73:1937-1945
	Data entered. 

9/10/2015
Michael Weiss

	Santa Brigida et. al 2007 Brazilian Journal of Biology 67(4):919-924
	Data entered. Check to see if it’s ok that fish were not captured by the study, rather were purchased from the cities.

	Schultz et. al 2008 Molecular Ecology 17:5336-5384
	Data not entered. Ranges are not specific enough and samples were collected with a wide range of methods, including museums and fish markets. 

	Shigenobu et. al 2007 Fisheries Science 73:1104-1112
	Data entered. 

	Shui et. al 2009 Fish Science 75:593-600
	Data entered. 

	Smith et. al 2008 Journal of Fish Biology 73:1170-1182
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Smith et. al 2009 Journal of Fish Biology 74:1104-1128
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Ghorashi et. al 2008 African Journal of Biotechnology 7(12):2074-2080
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Theisen et. al 2008 Molecular Ecology 17:4233-4247
	Data not entered. Sample sites are listed too vaguely. 

	Tseng et. al 2009 Journal of Fish Biology 74:357-376
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Turan 2008 Turkish Journal of Zoology 32:437-442
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Tzeng et. al 2007 Journal of the Fisheries Society of Taiwan 34(3):237-245
	Data entered. 

	Tzeng 2007 Zoological Studies 46(6):656-663
	Data entered. 

	Umino et. al 2009 Fisheries Science 75:909-919
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Turan et. al 2009 Folia biologica (Krakow) 57(1-2):219-226
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Victor 2008 Journal of the Ocean Science Foundation 1:1-19
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Iwamoto et. al 2009 Journal of Oceanography 65:103-112
	Data entered.

	Wang et. al 2008 African Journal of Biotechnology 7(24):4384-4392
	Data not entered. Haplotype and nucleotide statistics were averaged across multiple sample sites. 

	Kijewska et. al 2009 ICES Journal of Marine Science 66:902-906
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Freshwater et. al 2009 Marine Biology 156:1213-1221
	Data not entered. Sample sites are too broad. 

	Xiao et. al 2008 African Journal of Biotechnology 7(21):3845-3852
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

	Yaakub et. al 2007 Coral Reefs 26:597-602
	Data not entered. Haplotype and nucleotide diversity statistics are not reported.

